<!doctype html><html lang=en-us><head><meta name=generator content="Hugo 0.125.5"><meta charset=utf-8><meta http-equiv=X-UA-Compatible content="IE=edge"><meta name=viewport content="width=device-width,initial-scale=1"><title>Target Enabling Packages (TEPs) </title><meta name=description content="ASAP Target Enabling Packages (TEPs) for initiating structure-based drug discovery programs"><meta name=author content="newpixcom"><meta property="og:url" content="https://asapdiscovery.org/outputs/target-enabling-packages/"><meta property="og:site_name" content="ASAP Discovery Consortium"><meta property="og:title" content="Target Enabling Packages (TEPs)"><meta property="og:description" content="ASAP Target Enabling Packages (TEPs) for initiating structure-based drug discovery programs"><meta property="og:locale" content="en-us"><meta property="og:type" content="article"><meta property="article:section" content="outputs"><div id=cookie-notice><span>We would like to use third party cookies and scripts to improve the
functionality of this website.</span>
<a id=cookie-notice-accept class="btn btn-primary btn-sm">Approve</a>
<a id=cookie-notice-deny class="btn btn-primary btn-sm">Deny</a>
<a href=/privacy class="btn btn-primary btn-sm">More info</a></div><script>function createCookie(e,t,n){var s,o="";n&&(s=new Date,s.setTime(s.getTime()+n*24*60*60*1e3),o="; expires="+s.toUTCString()),document.cookie=e+"="+t+o+"; path=/"}function readCookie(e){for(var t,s=e+"=",o=document.cookie.split(";"),n=0;n<o.length;n++){for(t=o[n];t.charAt(0)==" ";)t=t.substring(1,t.length);if(t.indexOf(s)==0)return t.substring(s.length,t.length)}return null}function eraseCookie(e){createCookie(e,"",-1)}if(readCookie("cookie-notice-option")=="true"){function loadScriptAsync(e,t){if(typeof t!="function")throw new Error("Not a valid callback for async script load");var n=document.createElement("script");n.onload=t,n.src=e,document.head.appendChild(n)}loadScriptAsync("https://www.googletagmanager.com/gtag/js?id=G-M3L857X90H",function(){window.dataLayer=window.dataLayer||[];function e(){dataLayer.push(arguments)}e("js",new Date),e("config","G-M3L857X90H",{anonymize_ip:!0})})}else readCookie("cookie-notice-option")!="false"&&(document.getElementById("cookie-notice").style.display="block");document.getElementById("cookie-notice-accept").addEventListener("click",function(){createCookie("cookie-notice-option","true",31),document.getElementById("cookie-notice").style.display="none",location.reload()}),document.getElementById("cookie-notice-deny").addEventListener("click",function(){createCookie("cookie-notice-option","false",31),document.getElementById("cookie-notice").style.display="none",location.reload()})</script><link rel=canonical href=https://asapdiscovery.org/outputs/target-enabling-packages/><link rel="shortcut icon" type=image/png href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAGAAAABgCAYAAADimHc4AAAABGdBTUEAALGPC/xhBQAAAAFzUkdCAK7OHOkAAAAgY0hSTQAAeiYAAICEAAD6AAAAgOgAAHUwAADqYAAAOpgAABdwnLpRPAAAAAZiS0dEAP8A/wD/oL2nkwAAAAlwSFlzAAAASAAAAEgARslrPgAAERBJREFUeNrtnWmQVNd1x3/nvd57FgaaGc2IAcQigQABEshgULRYe2ztCnbJll1SVaxU7ConVUk+pCqVL/G3VDmJHceyHSeWHCRZC5FBu20kYyMEQhL7SMAMDDD71nv3W04+PEYMMCvQm6N/VTND9+t37zv/e8+5Z7l3RFWVSoXrgm2DCPh9gHjvq3rvDePc/5cRfKXuwIVCe/rQ17Yhbe1IwA9rVsK6a3HjGfT9/ZgrF0JsBmpZOHsOYTbWI00Npe72eTBK3YELgSaS8NRLyAtvgOWgHT3ovz0FOz5ET/TCTzfBsXYA3GwO5+W34ODhUnd7VFTkDNC3dyN/+AD5+z+HlUsgm0P/83n0uS24930RI5vGGRzCjSdwh+KQjINll7rbo6LiCFBVtOUo4g/Aovng80GVD1Ysgnffh8E4DCbRl95E9xyCdA6zpRVuv6nUXR8VFUeAoIhPABd13WGzi2Tz4PMhfj9EgphLr8JYthhNJHGPtJW622Oi8myAGMiyhYhlw669EE+inT24Oz6ExfNhRh2Eg8h1SzHWrYJ1q3Avqwf9bBV0ySBrVqKH2+F7P0dnNUA6jxEKwN8+BhkgEgHDHL4ajUTRoJ9ypEAq1g8YSqI7PkKPnUCiEWT1NbBwNm48Ay1tGAtnwbQa1HZwDrdhxuqQ2PRS9/o8VC4Bw1BFBaQsx/fEqDwbMBKquIfb4PAxz9utQFQ0AWrb6POvIs+/Ao5T6u5cECrSCA9DVXETcUTB+GwGlAYOSh6lMsX/R0BAFiWHW+puXDAqmgAFsrhkpVLHf4XbAIAcgLooVORCtPIJ0Mr1AeCPgQBcpGJNcIUToODp/wpdgkIlE+A4kMmQzecR10HjCQgFwTTB7/d+VgAqjwDbhsEhGBqCRILcQL83A1pbIeD3hB8OQ910qK4Co7wXepVDgCrE49DX7/10HMjnybmOZwEcByzxUo/pDCSSUFsDM2IQjZS692OiMghwXRgchFMdkMt9+raKkBVQFD237CSfh55ej4ymRqipphwXquU9P8Eb+QMDcOLkWcIfRg6wxvt+KgUnT0EyVeonGRXlT0A6DV3dYJ0vZkXJiZKdzD06OkclsNQobwIsCzq7IJMZ/XMVMijpyQTjEgno6yu7JWt5E5DJQDI57iVZIDsZ1T5sxPP5Uj/VWShfAlQ9/T1mokVAICen40GTQTbn3bOMUL4EuK43A9yxQs1KOpulO56gJ5EgOxn97jgQT4xzz+KjfAmwLMiNoS5EONnZyQ+eeordBw7wwcGD/Nfzz9M/ODh+FbQqWPmyIqB8/QDX9V4jBSqClc9zYN8BfvXmG7R2dfHEww/juA6/3foOzmCCu++4lfkLFmCOFYpwyyt2VL5lKZmsF15Ipz0SREglEmzf+g6bX97C5fPn8eCGB5k7ezaqysGWj/nlxmfJDQ1y34P3s/JznyMYDp8tbFWoqYErrji9n6D0KF8CbBvajsHQEK7r0tV2jN9teY3Dhw+zdN1aPn/zTcRmxj69XFXpONXB1tdfp/XDPSxfuZI1t9/KjKZGZOQsml4Hc+aUTYyofAlQhVMd2O3ttO3+iHeeeQ7b72fdww9w5Ypr8AcC56sSEbKZDPt3vMf25zdRV13DjV/9Mk1XLcQwTU/ozbMgFruwPhUA5UsAkGo5zMHv/4jW3buZvuQqltxxGw1zZp89os97IsG1bU62fML+198i3dbOwvVrufLOWwk21MMVcyEUKvWjneluORKgtk1830GO/PdGTm7fwdxbb+SqO75AIBqZvAEVIT0wyIFNm+navpPmlcuZ99UNRNetQXzlof+hDAlwkkn6Nr/JiZ8+jcydRfMjD1FXHeVC0yt2Lkff3gMcf+kV/EaQ5m8/Tt0t6zGCwVI/KlBGBKjr4rSdYGjjS/Rv3UbgxrXM/NqfEWlu8gJp3d0XvH5Xn49UJk/3Lzfj7D3AzHvvovr+uzEb60u+e7I8CHAc7Hd3k/7nH5G3bUJPfI3ITZ/HiIS9zy3LI6Gvb+ok+HzQ0AAN9djxBKlXf431k18QmTWb0N88gbF4QUlXRKUnYDCObnwZ5/W3sec2YT76EL4VS5BzhWLZ0NfrZcSyEwSgVT2hRqPeimda7ac5YrVsrD+8hz79Av6BFPLQ3XDv7Ui4NIa5aARoMg19A97WopoqiNV5Qtrbgv7j95AbV6NffwiprRlfsOk09PZ6CZZ8/nyjbBgQCHpCnzEdAoFR1Yx2dMMPn0Y7OpF/+A7S3IRaNvT0IemMt1KK1XmJ/konQI+fgh/8AgYTUFuNDMTh9vVwz81g2WhnD9JUD8MqZyI4jpdcyee9l+MA4qmbgB+CQU/wE6mWeALtHUAub0BzefSZzcjO/TC9FnoHkKvnweMPe2QWCIVfjzkOvPI20tGFPv4QzGpEn3kNnnwOWdgMK65GFsyZ2j1N09sHFjmdbB8eQ1M1qDXVSE219/sbv0Oe3Yx840Hc9auRDw6hGzfBFU3I/XcVzFgXnACNJ9CWI8gjX0JuXOO999gD6CdH4MBhZMXVF9/IRQpHLQt27UGuXw4P3IkRjcCcy9HOTvR37yF33jz52TlFFN78Z3PgWLixujPvTauCmXVexUI5lBXaDm4qizbVw7AxNgyITUNSWbALt/um8ASEQojpR/rjZ97rHYATHUhdeZSKiGlihIPQ2f1p4l4dBz3Zh8yo8+xKgVBwFSTVVehVC2DjFjRvIw0z0NfexkBg2aJCNz85BPzI6mvQH/0PbPwVLF/iHe7x4X549L6CroSKsww91e2dZtJyFMwAMrMWvn4vrFoOZnmEhUml0RffhC1bwVbwKXLXDbDhS38EBIB3pEB3H5q3MOqne0cKlNshSpbtHX2TSCI1UaQ+BsFAQZssjSecy6HJDDKtunyqmC0L4kmvhLGI2bKSzH93zwH0u99DW4+XovlRoR/sQ7/7r2hre1HbLY0C9vnRnXvQfYdK0vx5UIXf74R9LUXPFZeGgPlzsFcuJb/nAJq3Lv5+FwmNJ8kfOoK9dhU01he17ZIQYNRU49ywmvTho9idXaXowlnIt58kNTCA3rQGKXDw7TxZlOqhfcsWk+ofJLX9/VJ1AfAcrvjvd5AJ+TEWLyx6+yUjwGxswI5NZ2DbuzjpzMXf8AJh9Q/Q/852mNOMOX1a0dsvGQFGVRT/tdfQf6iFXE8vDmA5St5RLEdx3MJFiVz1wjuWQvzIcQba2jBXLvXOmysySlYeIIZB4PrrSGx5g/Y9h6maNgvb0U93vJuG4jch7BfCfsF/ke6CKmRtJZ1XcjbYLriq9O4+SCJSQ2ruIvrTEPZDyFc8H7EkBLgKGQvSl88nM28Bx3btZdbKtRgjRuAwEUOiRAJCXVgIBwTjAgRjOTCUcYlnFcsZnlmCk0nTtf8g/hXLycWa6Em6+A2lNmxQGxJ8RfARi66CXIX+lNIZd8kGqghcs5zuj3aR6ew4a9jJiOuTOaUj4TKQVtwp6qVU3murL63knRFqTSDZdpTB461Er1+DEQqh6l3Tm3LpiLuk84UPEhSVAMf1hD+QdrEcRQyDyJVXMdDdRd/+veN+13agP+XSn3JxJimXjKX0JFxSeT0vdayOQ9euHaSsPKH5888Kiqt6xPUmXbJWYUkoKgGJnNKfPiNAVSXcdDnMnMHJ3e/h5POMlx9wFAbSSjI7sVByNvQklexoJxaLYKVTtO/agdk8m2CsflSDn7agL+UtCAqFohGQtZXBjHu2ClElUFNL7YpVdB1rJTM4MKHxcxQGMkp2HAdaFeLZsVWIiDB06gTdp05Ru/JafMHgmCWPqbwSzxbuRK6iEOAJZHShGX4/TWvXkXQseo+1MpkMWdZS4ll3zDLRvOPZjfE6dPLAfqyqKPUrrhu32Nc93fdCZSWLQoDtQnqczYkz5i/E39BIy87tWPkJd/0C3irKHkM1pPOeMR0VImRTKVoP7mXG0uXUzWqecHTnHSVnF2YOFIUAy1HssSynKqHqauoWXMm+XdsZ7O4av/x8+J6u57CdC8f1Rv9Ys0OA7vY2jrTsp3HZcvyh8IQV165LZRPgOT1jf26aPpoWL6Gjt5vjnxzCMA0Mwxz3hZi4oxzI7QIqY3zfNEHg4Efvk7RtGhcumlRJgMII/+HSoiiO2ERrdwXq58zDH4ux7e03kWiUiWyBCEyPCNXBM+dlCZBzoDfpYjmjf8e2bX6/7TfUNs9mWv1lk+q/4hn/QvwpmvLYqaCKz++HQIBXXtnEu7u2M6ExFgiY4DvHNXZdT/+PRbq6LrgOX//mdwiFw5S6NrkoBBgyvjgd2+aDrW+S6+3lm9/+O+ZcuWhCwRgCM6IGkRFbxQSwXOhNekG9seAPBJk7b+H5FdhjQACfUZj4UFEI8JunOz+GTFo/2s2uLZu47U/v59aHv0owHJ5wJ5LfhMYaIXxOANNV6IgryXE3zuuURr4I+I3CnM1YJAIEvynY5+gFEYP+E8fY+uPv09TUzPp7HiYQCuFOYhOGKd49z4UhEA1AKudOOW40dlsQLFCkuiirINOAaPCcESRCLjHEvheeIWA73PLIY9TMiE1qZAqekMeq6YoEhKDv0o3XoO/S3m8kikKAADUhIRI48xCuZfHJyy/Sse1t1j36OE2Ll0xaLUQC3v3GEonfhOqgXBKdbRpQGxZ8BZJU0WJBARPqIsMPIgwePMDJl19kwZ/cwuzr107aIPpMqIsYBMYZkcOEVwcvTm8LHpFVwcJlZ4oaDY0EhFi1iX2ijeNP/geXLVrCog2P4AtO7I2CR2Isapw1k8aCz4RYlUHVRZBQFRKmR40LSgJNFkUlwBCoIkvu1ZcwOo4z98ENRBoamYyPGfAJsahBbXjyWbGACTOrDGqm8B3w1M60sDCzSggUOCtWVEdMHYe+V39Nbvt7rP6rbxFdtZSko1gjTpAZeQq6t/yD6pBBdUgImlPfTRDwQX2VQTSgDGWVTP5MVm2kBw3eAIkEhJqQEA1IUQq3i0pAet9Bup/8OQ1rrmP2nbdgRAJUW0rGEnK24jheLEfwVEjA9Az3xSbJTcOzCWG/kLGUrOXlhoeJMAyP6JDfa69QBnc0FI0AZ2CQxM+eoTo2ncu+8RXM03uuvKoHUD37DHQ5/c+lVL9+0/NJRsaPRrZXimr54nCdzZH/2bOYnxyh4S8fJzSn+bxLRDwVMPySSyz88doabq8UKE5G7P29GC++RtUX7yS0akVpnrRMUXgV1D8Iz20mcMt62HBvWR0VUw4ovDRsG7l6Ptx6A5Sg9rLcUfgtSq7rbf/x+8vmnLZywtRnQN7y/mp1XY0nVMtGu3qR2iqorvI2uB1ph6EkVEVg4ezTR8d/htEwZQLcU/3wv69jPHQbXN6I09eP/suPMe+6GT53LfrDp5Hfvgv1M6F/CK5bBH/xNWiYWepnLUtMWSdoNo+e6Pz0OHnNW+jRVugfQLftRPYcQr/5Zfinv0a/9Sj6fgv68ltldVptOeECjLBA/xD6m+1o0zHo7MEYiKOWjf7hQ2TVMuRLt3nbTxsb0Nbj6CdHMTKZ08n2zzASUydAQBIp2Psxcqofo38QEinUdnDjCWR2I3J676+YBsysQ44cRQpZYFnBmDoBCjqnCePx+2HeXJy2dvTYEUy/D7MhhtHV4x07HA6hqQy652MkEvUOUfoM52HqNkAUNfC28Af8EAqgokjAj9yyFt21F/79adi1F31qE7L9Q2T9dQXf8l+pmPIMkNoozuplaHXUC2BFo7hfuAF3bjPm0kXohnvQd3ZC+68Q00Se+Aqsv77Uz1m2mLIjpq563q3/dIxYFTdvIT7T0/2ui6bSkMlBOIhEI585YOOg9MdW/j/H/wHPYXLfRc591AAAACV0RVh0ZGF0ZTpjcmVhdGUAMjAxOC0wOS0yN1QxOTo1NDo0NSswMDowMIDoSlsAAAAldEVYdGRhdGU6bW9kaWZ5ADIwMTgtMDktMjdUMTk6NTQ6NDUrMDA6MDDxtfLnAAAARnRFWHRzb2Z0d2FyZQBJbWFnZU1hZ2ljayA2LjcuOC05IDIwMTQtMDUtMTIgUTE2IGh0dHA6Ly93d3cuaW1hZ2VtYWdpY2sub3Jn3IbtAAAAABh0RVh0VGh1bWI6OkRvY3VtZW50OjpQYWdlcwAxp/+7LwAAABh0RVh0VGh1bWI6OkltYWdlOjpoZWlnaHQAMTkyDwByhQAAABd0RVh0VGh1bWI6OkltYWdlOjpXaWR0aAAxOTLTrCEIAAAAGXRFWHRUaHVtYjo6TWltZXR5cGUAaW1hZ2UvcG5nP7JWTgAAABd0RVh0VGh1bWI6Ok1UaW1lADE1MzgwNzgwODVJxYBaAAAAD3RFWHRUaHVtYjo6U2l6ZQAwQkKUoj7sAAAAVnRFWHRUaHVtYjo6VVJJAGZpbGU6Ly8vbW50bG9nL2Zhdmljb25zLzIwMTgtMDktMjcvMTMzYjc5NWY2OTU4ZjNiZTAwOTA3MzM0OGI4YTQ3NDAuaWNvLnBuZ+0QeFYAAAAASUVORK5CYII="><link href=/css/font.css rel=stylesheet type=text/css><link href=/css/kube.css rel=stylesheet type=text/css><link href=/css/kube.legenda.css rel=stylesheet type=text/css><link href=/css/highlight.css rel=stylesheet type=text/css><link href=/css/master.css rel=stylesheet type=text/css><link href=/css/kube.demo.css rel=stylesheet type=text/css><link href=/css/custom.css rel=stylesheet type=text/css><script src=/js/jquery-2.1.4.min.js type=text/javascript></script><script type=text/javascript src=/js/tocbot.min.js></script></head><body class=page-kube><header><div class=show-sm><div id=nav-toggle-box><div id=nav-toggle-brand><a href=/>ASAP Discovery Consortium</a></div><a data-component=toggleme data-target=#top href=# id=nav-toggle><i class=kube-menu></i></a></div></div><div class=hide-sm id=top><div id=top-brand><a href=/ title=home>ASAP Discovery Consortium</a></div><nav id=top-nav-main><ul><li><a href=/pipeline/>Pipeline</a></li><li><a href=/outputs/>Outputs</a></li><li><a href=https://www.choderalab.org/news/2021/10/26/asap-avidd-proposal>Concept</a></li><li><a href=/members/>Members</a></li><li><a href=/sab/>SAB</a></li><li><a href=/faq/>FAQ</a></li><li><a href=/awards/>Awards</a></li><li><a href=/forum/>Open Science Forum</a></li><li><a href=https://postera.ai/moonshot/>COVID Moonshot</a></li></ul></nav><nav id=top-nav-extra><ul></ul></nav></div></header><main><div id=main><div id=hero><h1>Target Enabling Packages (TEPs)</h1><p class=hero-lead></p><div class="message error" data-component=message>This page is still in development. Please report any issues to <a href=https://github.com/asapdiscovery/asapdiscovery.github.io/issues>our issue tracker</a>.</div></div><div id=kube-component class=content><nav id=contents><ol class=js-toc></ol></nav><script type=text/javascript>document.addEventListener("DOMContentLoaded",function(){tocbot.init({tocSelector:".js-toc",contentSelector:".content",headingSelector:"h3"})})</script><p>A <a href=https://www.thesgc.org/tep>Target Enabling Package (TEP)</a> is a complete data package needed to enable structure-based drug discovery against an antiviral target.
Pioneered by the <a href=https://www.thesgc.org/tep>Structural Genomics Consortium</a>, each TEP contains relevant protein constructs and plasmid resources for one or more viral family members, protein expression and purification protocols, crystallization conditions, structures from an <a href=https://www.diamond.ac.uk/Instruments/Mx/Fragment-Screening.html>X-ray fragment screen at the Diamond Light Source XChem facility</a>, small molecule hits, and biochemical assay protocols with at least one validated inhibitor with quantifiable activity.</p><h3 style=text-align:center id=ASAP-COV-MPRO>SARS-CoV-2 / MERS-CoV Mpro</h3><div class="row align-center"><a href="https://www.ncbi.nlm.nih.gov/protein/%25!s%28int=43740578%29"><span class="label success">Gene: 43740578 </span></a>&nbsp;
<a href=https://www.uniprot.org/uniprotkb/P0C6U8><span class="label error">Uniprot: P0C6U8 </span></a>&nbsp;
<a href="https://www.ebi.ac.uk/intenz/query?cmd=SearchEC&amp;ec=3.4.22.69"><span class="label warning">EC: 3.4.22.69</span></a></div><br><div class="row align-center"><table class=unstyled><tr><td><img title="Crystallography construct" src=/images/icons/tep-stage-completed_hub021b18cb248db28b117f458e39aeb5b_10162_e22862b00861f0d3434857db1ff02de5.png width=300 height=88><br><img title="Assay construct" src=/images/icons/tep-stage-completed_hub021b18cb248db28b117f458e39aeb5b_10162_2851d035e13cbc115749c33624801624.png width=300 height=88></td><td><img title="Biophysical assay" src=/images/icons/tep-stage-in-progress_huc9d943f35f5911a48f54b3ad3b40c43a_10037_80222a2cb4b3b4d0cafa06d6608261b8.png width=300 height=55><br><img title="Robust crystals" src=/images/icons/tep-stage-completed_hub021b18cb248db28b117f458e39aeb5b_10162_e3383ab8d75acf7a535bdc511b331b23.png width=300 height=55><br><img title="Biochemical assay" src=/images/icons/tep-stage-completed_hub021b18cb248db28b117f458e39aeb5b_10162_1b7ab3b95a075266c44f73afb56d1cd3.png width=300 height=55></td><td><img title="X-ray fragment screen and structural analysis" src=/images/icons/tep-stage-completed_hub021b18cb248db28b117f458e39aeb5b_10162_ae1969e18479b9f6892265d5fd14a303.png width=300 height=180></td><td><img title="First-round progression for TEP validation" src=/images/icons/tep-stage-completed_hub021b18cb248db28b117f458e39aeb5b_10162_5f535c81547d74d3984d598f890911a7.png width=300 height=180></td></tr></table></div><b>Contributors:</b>
<span class="label success">Aaron Keeley </span><span class="label success">Ailsa Powell </span><span class="label success">Alexandre Dias </span><span class="label success">Alice Douangamath </span><span class="label success">Alpha Lee </span><span class="label success">Andre de Godoy </span><span class="label success">Anna Carbery </span><span class="label success">Anthony Aimon </span><span class="label success">Anthony Tumber </span><span class="label success">Blake Balcomb </span><span class="label success">C. David Owen </span><span class="label success">Charlie Tomlinson </span><span class="label success">Charline Giroud </span><span class="label success">Christopher Schofield </span><span class="label success">Claire Strain-Damerell </span><span class="label success">Daren Fearon </span><span class="label success">Dong Wang </span><span class="label success">Efrat Resnick </span><span class="label success">Eleanor Williams </span><span class="label success">Gemma Davison </span><span class="label success">György M. Keserü </span><span class="label success">Haim Barr </span><span class="label success">Hanna F. Klein </span><span class="label success">James D. Firth </span><span class="label success">Jasmin Aschenbrenner </span><span class="label success">Jessica Dixon </span><span class="label success">John D. Chodera </span><span class="label success">José Brandão-Neto </span><span class="label success">Korvus Wang </span><span class="label success">Lee Harris </span><span class="label success">Lennart Brewitz </span><span class="label success">Lizbé Koekemoer </span><span class="label success">Louise Dunnett </span><span class="label success">Lucas Ferreira </span><span class="label success">Marco Mazzorano </span><span class="label success">Martin A. Walsh </span><span class="label success">Martin E. M. Noble </span><span class="label success">Mathew P. Martin </span><span class="label success">Matthew Snee </span><span class="label success">Michael Fairhead </span><span class="label success">Michael J. Waring </span><span class="label success">Nir London </span><span class="label success">Noa Lahav </span><span class="label success">Oleg Fedorov </span><span class="label success">Paul Gehrtz </span><span class="label success">Peter Marples </span><span class="label success">Peter O’Brien </span><span class="label success">Petra Lukacik </span><span class="label success">Péter Ábrányi-Balogh </span><span class="label success">Rachael Skyner </span><span class="label success">Rambabu N. Reddi </span><span class="label success">Ryan Lithgo </span><span class="label success">S. Paul Jones </span><span class="label success">Tamas Szommer </span><span class="label success">Thomas D. Downes </span><span class="label success">Tika R. Malla </span><span class="label success">Warren Thompson </span><span class="label success">Xiaomin Ni</span><br><br><div class="prettybox warning" data-component=prettybox><b>Target Nominator:</b> Martin Walsh and Ed Griffen<br><b>Target PI:</b> Frank von Delft<br><b>Therapeutic Area(s):</b> Infectious diseases<br><b>Disease Relevance:</b> The main protease (Mpro) is responsible for viral polyprotein processing and therefore a therapeutic target.<br><b>Viral family:</b> coronaviridae<br><b>Viruses:</b> [SARS-CoV-2 MERS-CoV]<br></div><div class="prettybox success" data-component=prettybox><h4>Summary of Project</h4>The coronavirus family poses a major global health concern. Notably, Severe Acute Respiratory Syndrome (SARS), Middle East Respiratory Syndrome (MERS) and Severe Acute Respiratory Syndrome coronavirus 2 (SARS-CoV-2) also known as COVID-19 cause severe illness in mammals and birds. The COVID-19 pandemic has highlighted the need to identify novel therapeutic interventions and strategies of pandemic preparedness. MERS-CoV is a zoonotic virus that was first discovered in 2012 [1](https://doi: 10.1056/NEJMoa1211721). The disease has spread rapidly with large outbreaks as recent as 2015 and 2018 <a href=https://www.who.int/en/news-room/fact-sheets/detail/middle-east-respiratory-syndrome-coronavirus-(mers-cov)>2</a>, <a href=https:>3</a>//doi: 10.1016/S0140-6736(15)60454-8), <a href=https://www.emro.who.int/health-topics/mers-cov/mers-outbreaks.html>4</a>, [5](https://doi: 10.1128/genomeA.00818-15). Currently there is no therapeutic intervention for MERS-CoV with 35% of reported cases resulting in human death <a href=https://pubmed.ncbi.nlm.nih.gov/26049252/>6</a>]. Like-wise to SARS-CoV-2, MERS-CoV produces a main protease (Mpro) which is essential for viral replication and therefore an attractive target to stop the virus [7](https://doi: 10.1074/jbc.M115.651463), [8](https://doi: 10.1016/j.virusres.2015.05.018), [9](https://doi: 10.1016/j.virusres.2015.02.025). This TEP includes the protocols used for producing recombinant MERS-CoV Mpro, assay documentation as well as crystallization protocols. Moreover, it contains information about different enzymatic and viral replication assays for orthogonal validation of small molecule inhibitors in order determine structure-activity relationships. All data are fully open and constitute a unique resource, not only for this target, but for improving methodology of fragment-to-hit and hit-to-lead progression.</div><div class="prettybox focus" data-component=prettybox><h4>Scientific Background</h4>The emergence of COVID-19 has caused a global pandemic of unexpected scale. The disease is caused by the SARS-Cov-2 virus <a href=https://doi.org/10.1016%2FS1473-3099(20)30144-4>10</a>, <a href=https://doi.org/10.1038/s41586-020-2008-3>11</a>, <a href=https://doi.org/10.1056/nejmoa2001017>12</a> and has so far claimed more than [7 million lives world-wide](November 2020, <a href=https://covid19.who.int>https://covid19.who.int</a>), despite numerous regional lockdowns in order to stop the spread of the virus. However, despite the fact that SARS-CoV-2 is already the third zoonotic coronavirus outbreak after the emergence of SARS-CoV-1 in 2002 and the Middle East Respiratory Syndrome (MERS-CoV) in 2012 <a href=https://pubmed.ncbi.nlm.nih.gov/23078800/>13</a>, <a href=https://doi.org/10.1016/s0140-6736(03)13967-0>14</a>, <a href=https://doi.org/10.1056/nejmoa1211721>15</a>, there is a notable lack of antiviral treatment options. SARS-CoV-2 is a large enveloped, positive-sense, single-stranded RNA Betacoronavirus from the order of nidoviruses. Its RNA encodes two open reading frames that, through ribosome frame-shifting, generates two polyproteins pp1a and pp1ab <a href=https://doi.org/10.1093%2Fnar%2F18.7.1825>16</a>. The genome contains a large replicase gene encompassing nonstructural proteins (Nsps), followed by structural and accessory genes. The polyproteins are processed by two viral cysteine proteases: a papain-like protease (PLpro) which cleaves three sites, releasing non-structural proteins nsp1-3 and a 3C-like protease, also referred to as the main protease (Mpro), that cleaves at 11 sites to release non-structural proteins (nsp4-16). These non-structural proteins form the replicase complex responsible for replication and transcription of the viral genome and have led to Mpro and PLPro being the primary targets for antiviral drug development <a href=https://doi.org/10.1111/febs.12936>17</a>.</div><div class="prettybox error" data-component=prettybox><h4>Conclusion</h4>This TEP provides a comprehensive sampling of the binding properties of the active site of SARS-CoV-2 main protease by small molecule fragment molecules. Together with the established enzymatic assays, it forms the bases for rational, structure-based drug development. The usefulness of the data and protocols presented in this TEP have been demonstrated by the <a href=https://covid.postera.ai/covid>COVID Moonshot consortium</a> successfully developing sub-micromolar binders, that are now in lead optimisation.
SARS-CoV-2 Mpro: * <a href=https://fragalysis-legacy.xchem.diamond.ac.uk/viewer/react/preview/target/Mpro/tas/lb18145-1>Fragalysis interactive structure viewer</a> and downloads of all structures. * Fragment screen PDB group deposition <a href=https://www.rcsb.org/groups/summary/entry/G_1002135>G_1002135</a>, <a href=https://www.rcsb.org/groups/summary/entry/G_1002144>G_1002144</a>, <a href=https://www.rcsb.org/groups/summary/entry/G_1002151>G_1002151</a>, <a href=https://www.rcsb.org/groups/summary/entry/G_1002152>G_1002152</a>, <a href=https://www.rcsb.org/groups/summary/entry/G_1002153>G_1002153</a>, <a href=https://www.rcsb.org/groups/summary/entry/G_1002155>G_1002155</a>, <a href=https://www.rcsb.org/groups/summary/entry/G_1002156>G_1002156</a>, <a href=https://www.rcsb.org/groups/summary/entry/G_1002157>G_1002157</a>, <a href=https://www.rcsb.org/groups/summary/entry/G_1002163>G_1002163</a>, <a href=https://www.rcsb.org/groups/summary/entry/G_1002272>G_1002272</a></div><h4>Resources</h4><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href><img title=plasmid width=40 src=../img/icons/plasmid.png></a></div><div id=comev2-NSP5-mpro_expression-construct class=col-11><b>Plasmid for protein expression of SARS-CoV-2 Mpro protease for assays and crystallography</b><br>Mpro from SARS-CoV-2 was cloned into pGEX-6P-1 for heterologous protein expression in Escherichia coli. The protein was purified by Ni-affinity chromatography, followed by HRV 3C protease treatment which yielded native N- and C-termini. HRV 3C protease was removed by reverse Ni-affinity chromatography, followed by a gel filtration step. The purified protein appears as a dimer in size-exclusion chromatography.<br><cite>[ 2020-02-14 ]</cite><br><button data-component=toggleme data-target=#togglebox-protocol-ASAP-COV-MPRO-1 data-text="Hide Protocol">Protocol</button><div id=togglebox-protocol-ASAP-COV-MPRO-1 class=hide><cite>Available at <a href=http://doi.org/10.17504/protocols.io.rm7vzj8p8lx1/v1>protocols.io</a></cite></div><button data-component=toggleme data-target=#togglebox-details-ASAP-COV-MPRO-1 data-text="Hide Details">Details</button><div id=togglebox-details-ASAP-COV-MPRO-1 class=hide><a href="/pdfs/PAGE22-01340 - AVIDD ASAP; MERS mPRO aka 3CL protease.pdf" download>Download experimental metadata</a>
<a href="/pdfs/PAGE23-00183 - AVIDD ASAP; MVMPROA c907 (c900 Arg mutant) Sequencing Expression and Purification.pdf" download>Download experimental metadata 2</a></div></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href=https://fragalysis.diamond.ac.uk/viewer/react/preview/target/Mpro><img title=xray-fragment-screen width=40 src=../img/icons/xray-fragment-screen.png></a></div><div id=comev2-NSP15-mpro_fragment-screen-fragalysis class=col-11><b>XChem crystallographic fragment screen of SARS-CoV-2 Mpro</b><br>We performed an exhaustive crystallographic fragment screen to probe the Mpro active site, and to identify opportunities for fragment merging or growing. Additionally, a library of mild electrophilic fragments was screened by mass spectrometry for probing the binding properties around the active site cysteine. Overall, a total of 1742 soaking and 1139 co-crystallization experiments resulted in 1877 mounted crystals. This resulted in determination of 91 protein-fragment structures.<br><a href=https://fragalysis.diamond.ac.uk/viewer/react/preview/target/Mpro>https://fragalysis.diamond.ac.uk/viewer/react/preview/target/Mpro</a>
<cite>[ 2020-03-18 ]</cite><br><button data-component=toggleme data-target=#togglebox-protocol-ASAP-COV-MPRO-2 data-text="Hide Protocol">Protocol</button><div id=togglebox-protocol-ASAP-COV-MPRO-2 class=hide><cite>Crystallization protocol <a href=http://doi.org/10.17504/protocols.io.kqdg326y7v25/v1>available at protocols.io</a>
Data were collected at the beamline I04-1 at 100 K and processed with the fully automated pipelines at Diamond Light Source [<a href=https://doi.org/10.1016/j.ymeth.2011.06.010>11</a>], which variously combine XDS [<a href=https://doi.org/10.1107/s0907444909047374>12</a>], xia2 [<a href=https://doi.org/10.1107/s0907444913015308>13</a>], autoPROC [<a href=https://doi.org/10.1107/S0907444911007773>14</a>] and DIALS [<a href=https://doi.org/10.1107%2FS2059798317017235>15</a>], and select resolution limits algorithmically; no manual curation of processing parameters was applied. Further analysis was performed through XChemExplorer [<a href=https://doi.org/10.1107%2FS2059798316020234>16</a>]: for each dataset, the version of processed data was selected by the default XChemExplorer score, and electron density maps were generated with Dimple. Ligand-binding events were identified using PanDDA [<a href=https://doi.org/10.1038/ncomms15123>18</a>] (both the released version 0.2 and a pre-release development version (<a href=https://github.com/ConorFWild/pandda)>https://github.com/ConorFWild/pandda)</a>, and ligands were modelled into PanDDAcalculated event maps using Coot [<a href=https://doi.org/10.1107/s0907444910007493>19</a>]. Restraints were calculated with ACEDRG [<a href=https://doi.org/10.1107%2FS2059798317000067>20</a>] or GRADE, structures were refined with Refmac [<a href=https://doi.org/10.1107/s0907444911001314>22</a>] and Buster. Fragments were soaked into crystals by adding dissolved compound directly to the crystallisation drops. The following libraries were screened: the DSipoised library (Enamine), a version of the poised library [<a href=https://doi.org/10.1039/C5SC03115J>24</a>]; a version of the MiniFrags library [<a href=https://doi.org/10.1016/j.drudis.2019.03.009>25</a>] assembled in-house; the FragLites library; a library of shape-diverse 3D fragments (“York3D”) [<a href=https://doi.org/10.1002/chem.202001123>27</a>]; heterocyclic electrophiles [<a href="http://xlink.rsc.org/?DOI=c8md00327k">28</a>]; and the SpotFinder library. All fragments were in 100% DMSO at varying stock concentrations, detailed at <a href=https://www.diamond.ac.uk/Instruments/Mx/>https://www.diamond.ac.uk/Instruments/Mx/</a> Fragment-Screening/Fragment-Libraries.html). In brief, 55 nl of fragment stock solutions in DMSO (DSIpoised, FragLite, PepLites, York 3D, Covalent Heterocylces and SpotFinder all at 500 mM, MiniFrags at 1 M and Cysteine covalent library at 20 mM) were transferred directly to 500 nl crystallisation drops using an ECHO liquid handler giving a final compound concentration of 2–100 mM and DMSO concentration of 10%. Drops were incubated at room temperature for ~1 h prior to mounting and flash cooling in liquid nitrogen without the addition of further cryoprotectant.</cite></div></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href=https://www.rcsb.org/groups/summary/entry/G_1002135><img title=pdb-group-deposition width=40 src=../img/icons/pdb-group-deposition.png></a></div><div id=comev2-NSP15-mpro_fragment-screen-pdb-group-deposition-G_1002135 class=col-11><b>PDB group deposition G_1002135 of X-ray fragment screen of SARS-CoV-2 Mpro</b><br>COVID-19 main protease from Coronaviridae sp. screened against the DSI-poised Fragment Library by X-ray Crystallography at the XChem facility of Diamond Light Source beamline I04-1<br><a href=https://www.rcsb.org/groups/summary/entry/G_1002135>https://www.rcsb.org/groups/summary/entry/G_1002135</a>
<cite>[ 2020-03-11 ]</cite><br></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href=https://www.rcsb.org/groups/summary/entry/G_1002144><img title=pdb-group-deposition width=40 src=../img/icons/pdb-group-deposition.png></a></div><div id=comev2-NSP15-mpro_fragment-screen-pdb-group-deposition-G_1002135 class=col-11><b>PDB group deposition G_1002144 of X-ray fragment screen of SARS-CoV-2 Mpro</b><br>SARS-CoV-2 main protease from Coronaviridae sp. screened against DSI poised (Enamine), Fraglites and Peplites (Newcastle university), Mini Frags (Astex), York 3D (York university), electrophile cysteine covalent (Weizman institute) fragment libraries by X-ray Crystallography at the XChem facility of Diamond Light Source beamline I04-1<br><a href=https://www.rcsb.org/groups/summary/entry/G_1002144>https://www.rcsb.org/groups/summary/entry/G_1002144</a>
<cite>[ 2020-04-01 ]</cite><br></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href=https://www.rcsb.org/groups/summary/entry/G_1002151><img title=pdb-group-deposition width=40 src=../img/icons/pdb-group-deposition.png></a></div><div id=comev2-NSP15-mpro_fragment-screen-pdb-group-deposition-G_1002151 class=col-11><b>PDB group deposition G_1002151 of X-ray fragment screen of SARS-CoV-2 Mpro</b><br>SARS-CoV-2 main protease from Coronaviridae sp. screened against DSI poised (Enamine), Fraglites and Peplites (Newcastle university), Mini Frags (Astex), York 3D (York university), electrophile cysteine covalent (Weizman institute) fragment libraries by X-ray Crystallography at the XChem facility of Diamond Light Source beamline I04-1<br><a href=https://www.rcsb.org/groups/summary/entry/G_1002151>https://www.rcsb.org/groups/summary/entry/G_1002151</a>
<cite>[ 2020-03-25 ]</cite><br></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href=https://www.rcsb.org/groups/summary/entry/G_1002151><img title=pdb-group-deposition width=40 src=../img/icons/pdb-group-deposition.png></a></div><div id=comev2-NSP15-mpro_fragment-screen-pdb-group-deposition-G_1002151 class=col-11><b>PDB group deposition G_1002151 of X-ray fragment screen of SARS-CoV-2 Mpro</b><br>SARS-CoV-2 main protease from Coronaviridae sp. screened against DSI poised (Enamine), Fraglites and Peplites (Newcastle university), Mini Frags (Astex), York 3D (York university), electrophile cysteine covalent (Weizman institute) fragment libraries by X-ray Crystallography at the XChem facility of Diamond Light Source beamline I04-1<br><a href=https://www.rcsb.org/groups/summary/entry/G_1002151>https://www.rcsb.org/groups/summary/entry/G_1002151</a>
<cite>[ 2020-03-25 ]</cite><br></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href=https://www.rcsb.org/groups/summary/entry/G_1002152><img title=pdb-group-deposition width=40 src=../img/icons/pdb-group-deposition.png></a></div><div id=comev2-NSP15-mpro_fragment-screen-pdb-group-deposition-G_1002135 class=col-11><b>PDB group deposition G_1002152 of X-ray fragment screen of SARS-CoV-2 Mpro</b><br>SARS-CoV-2 main protease from Coronaviridae sp. screened against DSI poised (Enamine), Fraglites and Peplites (Newcastle university), Mini Frags (Astex), York 3D (York university), electrophile cysteine covalent (Weizman institute) fragment libraries by X-ray Crystallography at the XChem facility of Diamond Light Source beamline I04-1<br><a href=https://www.rcsb.org/groups/summary/entry/G_1002152>https://www.rcsb.org/groups/summary/entry/G_1002152</a>
<cite>[ 2020-04-15 ]</cite><br></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href=https://www.rcsb.org/groups/summary/entry/G_1002153><img title=pdb-group-deposition width=40 src=../img/icons/pdb-group-deposition.png></a></div><div id=comev2-NSP15-mpro_fragment-screen-pdb-group-deposition-G_1002153 class=col-11><b>PDB group deposition G_1002153 of X-ray fragment screen of SARS-CoV-2 Mpro</b><br>SARS-CoV-2 main protease from Coronaviridae sp. screened against DSI poised (Enamine), Fraglites and Peplites (Newcastle university), Mini Frags (Astex), York 3D (York university), electrophile cysteine covalent (Weizman institute) fragment libraries by X-ray Crystallography at the XChem facility of Diamond Light Source beamline I04-1<br><a href=https://www.rcsb.org/groups/summary/entry/G_1002153>https://www.rcsb.org/groups/summary/entry/G_1002153</a>
<cite>[ 2020-04-15 ]</cite><br></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href=https://www.rcsb.org/groups/summary/entry/G_1002155><img title=pdb-group-deposition width=40 src=../img/icons/pdb-group-deposition.png></a></div><div id=comev2-NSP15-mpro_fragment-screen-pdb-group-deposition-G_1002155 class=col-11><b>PDB group deposition G_1002155 of X-ray fragment screen of SARS-CoV-2 Mpro</b><br>SARS-CoV-2 main protease from Coronaviridae sp. screened against COVID Moonshot compounds by X-ray Crystallography at the XChem facility of Diamond Light Source beamline I04-1<br><a href=https://www.rcsb.org/groups/summary/entry/G_1002155>https://www.rcsb.org/groups/summary/entry/G_1002155</a>
<cite>[ 2020-05-27 ]</cite><br></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href=https://www.rcsb.org/groups/summary/entry/G_1002156><img title=pdb-group-deposition width=40 src=../img/icons/pdb-group-deposition.png></a></div><div id=comev2-NSP15-mpro_fragment-screen-pdb-group-deposition-G_1002156 class=col-11><b>PDB group deposition G_1002156 of X-ray fragment screen of SARS-CoV-2 Mpro</b><br>SARS-CoV-2 main protease from Coronaviridae sp. screened against electrophilic heterocycle and SpotFinder fragment libraries (Hungarian Academy of Sciences) by X-ray Crystallography at the XChem facility of Diamond Light Source beamline I04-1<br><a href=https://www.rcsb.org/groups/summary/entry/G_1002156>https://www.rcsb.org/groups/summary/entry/G_1002156</a>
<cite>[ 2020-06-10 ]</cite><br></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href=https://www.rcsb.org/groups/summary/entry/G_1002157><img title=pdb-group-deposition width=40 src=../img/icons/pdb-group-deposition.png></a></div><div id=comev2-NSP15-mpro_fragment-screen-pdb-group-deposition-G_1002157 class=col-11><b>PDB group deposition G_1002157 of X-ray fragment screen of SARS-CoV-2 Mpro</b><br>SARS-CoV-2 main protease from Coronaviridae sp. screened against COVID Moonshot compounds by X-ray Crystallogra phy at the XChem facility of Diamond Light Source beamline I04-1<br><a href=https://www.rcsb.org/groups/summary/entry/G_1002157>https://www.rcsb.org/groups/summary/entry/G_1002157</a>
<cite>[ 2020-06-10 ]</cite><br></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href=https://www.rcsb.org/groups/summary/entry/G_1002163><img title=pdb-group-deposition width=40 src=../img/icons/pdb-group-deposition.png></a></div><div id=comev2-NSP15-mpro_fragment-screen-pdb-group-deposition-G_1002163 class=col-11><b>PDB group deposition G_1002163 of X-ray fragment screen of SARS-CoV-2 Mpro</b><br>SARS-CoV-2 main protease from Coronaviridae sp. screened against computationally designed covalent inhibitors by X-ray Crystallography at the XChem facility of Diamond Light Source beamline I04-1<br><a href=https://www.rcsb.org/groups/summary/entry/G_1002163>https://www.rcsb.org/groups/summary/entry/G_1002163</a>
<cite>[ 2020-12-02 ]</cite><br></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href=https://www.rcsb.org/groups/summary/entry/G_1002272><img title=pdb-group-deposition width=40 src=../img/icons/pdb-group-deposition.png></a></div><div id=comev2-NSP15-mpro_fragment-screen-pdb-group-deposition-G_1002272 class=col-11><b>PDB group deposition G_1002272 of X-ray fragment screen of SARS-CoV-2 Mpro</b><br>SARS-CoV-2 main protease screened against COVID Moonshot compounds by X-ray Crystallography at the XChem facility of Diamond Light Source beamline I04-1. 466 structures in group deposition.<br><a href=https://www.rcsb.org/groups/summary/entry/G_1002272>https://www.rcsb.org/groups/summary/entry/G_1002272</a>
<cite>[ 2023-11-08 ]</cite><br></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href><img title=assay width=40 src=../img/icons/assay.png></a></div><div class=col-11><b>FRET-based proteolytic assay of SARS-CoV-2 Mpro</b><br>Fluorescent-based assay t0 measure peptide cleavage<br><cite>[ 2020-03-18 ]</cite><br><button data-component=toggleme data-target=#togglebox-protocol-ASAP-COV-MPRO-14 data-text="Hide Protocol">Protocol</button><div id=togglebox-protocol-ASAP-COV-MPRO-14 class=hide><cite>Assays are performed in Greiner 384 low volume plates into which the compounds are dispensed using an Echo 555 acoustic dispenser keeping the DMSO concentration &lt; 1%. Mpro is pre-incubated for 15 minutes at room temperature with the inhibitor compounds before the substrate is added to initiate the reaction. Protease reaction was measured continuously in a BMG Pherastar FS with a 480/520 ex/em filter set. The final reaction concentrations are 20 mM HEPES pH 7.3, 1 mM TCEP, 50 mM NaCl, 0.01% Tween-20, 10% glycerol, 5 nM Mpro, 375 nM fluorogenic peptide substrate ([5-FAM]-AVLQSGFR-[Lys(Dabcyl)]-K-amide) in 20 μL. All data is directly uploaded to the Moonshot vault on the Collaborative Drug Discovery Inc. (CDD) website for data analysis. The readouts are converted to % inhibition and the Levenberg–Marquardt algorithm is used to fit a Hill equation to dose-response data constrained with minimum value = 0.0 and maximum value = 100 to calculate IC50 values.</cite></div></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href><img title=assay width=40 src=../img/icons/assay.png></a></div><div class=col-11><b>RapidFire Mass Spectrometry assay</b><br>Mass spectroscopy-based assay to measure peptide cleavage<br><cite>[ 2020-03-18 ]</cite><br><button data-component=toggleme data-target=#togglebox-protocol-ASAP-COV-MPRO-15 data-text="Hide Protocol">Protocol</button><div id=togglebox-protocol-ASAP-COV-MPRO-15 class=hide><cite>Assays are performed in Greiner 384 low volume plates into which the compounds are dispensed using an Echo 555 acoustic dispenser keeping the DMSO concentration &lt; 1%. A 15 μM enzyme stock solution is prepared in 20 mM HEPES, pH 7.5 and 300 mM NaCl, and subsequently diluted to a working solution of 300 nM Mpro in assay buffer (20 mM HEPES, pH 7.5 and 50 mM NaCl). Mpro is pre-incubated for 15 minutes at room temperature with the inhibitor compounds before the addition of substrate to initiate the reaction. The final reaction concentrations are 20 mM HEPES pH 7.5, 50 mM NaCl, 150 nM Mpro and 2 μM substrate (TSAVLQSGFRK-NH2) in 50 μL. The reaction is incubated for 10 min at room temperature before quenching with 10% formic acid. The reactions are analysed with MS using RapidFire (RF) 365 high-throughput sampling robot (Agilent) connected to an iFunnel Agilent 6550 accurate mass quadrupole time-of-flight (QTOF) mass spectrometer using electrospray. RapidFire integrator software (Agilent) was used to extract the m/z (+1) charge states of both the substrate (1191.67 Da) and cleaved N-terminal product TSAVLQ (617.34 Da) from the total ion chromatogram data which is used to calculate the percentage inhibition. All data is directly uploaded to the Moonshot vault for data analysis. The readouts are converted to % inhibition and the Levenberg–Marquardt algorithm is used to fit a Hill equation to dose-response data constrained with minimum value = 0.0 and maximum value = 100 to calculate IC~50~ values.</cite></div><button data-component=toggleme data-target=#togglebox-details-ASAP-COV-MPRO-15 data-text="Hide Details">Details</button><div id=togglebox-details-ASAP-COV-MPRO-15 class=hide><a href="/pdfs/PAGE23-00152 - AVIDD ASAP; MERS MPRO c900 (MVMPROA p001) crystallization and further concentration.pdf" download>Download experimental metadata</a></div></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href=https://www.rcsb.org/groups/summary/entry/G_1002153><img title=pdb-group-deposition width=40 src=../img/icons/pdb-group-deposition.png></a></div><div id=comev2-NSP15-mpro_fragment-screen-pdb-group-deposition class=col-11><b>PDB group deposition of X-ray fragment screen of SARS-CoV-2 Mpro</b><br>SARS-CoV-2 main protease from Coronaviridae sp. screened against DSI poised (Enamine), Fraglites and Peplites (Newcastle university), Mini Frags (Astex), York 3D (York university), electrophile cysteine covalent (Weizman institute) fragment libraries by X-ray Crystallography at the XChem facility of Diamond Light Source beamline I04-1 Selected structures were deposited in the PDB via a group deposition.<br><a href=https://www.rcsb.org/groups/summary/entry/G_1002153>https://www.rcsb.org/groups/summary/entry/G_1002153</a>
<cite>[ 2020-04-07 ]</cite><br></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href><img title=plasmid width=40 src=../img/icons/plasmid.png></a></div><div id=MERS-CoV-protease_plasmid-biochemical-assay class=col-11><b>Protein construct for MERS-CoV protease suitable for both biochemical assays and crystallography</b><br>Construct for enzyme assays crystallography.<br><b>SCARAB ID:</b>
<img width=10 src=images/icons/scarab.png>
MVMPROA-c001<br><cite>[ 2024-04-30 ]</cite><br><button data-component=toggleme data-target=#togglebox-protocol-ASAP-COV-MPRO-17 data-text="Hide Protocol">Protocol</button><div id=togglebox-protocol-ASAP-COV-MPRO-17 class=hide><cite><a href=http://doi.org/10.17504/protocols.io.bp2l62b11gqe/v1>Available at protocols.io</a></cite></div><button data-component=toggleme data-target=#togglebox-details-ASAP-COV-MPRO-17 data-text="Hide Details">Details</button><div id=togglebox-details-ASAP-COV-MPRO-17 class=hide><ul><li>Vector:</li><li>Entry clone accession:</li><li>Cell line:</li><li>Tags and additions:</li><li>Construct protein sequence:</li></ul></div></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href=https://www.rcsb.org/structure/8R5J><img title=pdb-deposition width=40 src=../img/icons/pdb-deposition.png></a></div><div id=comev2-NSP15-mpro_apo-pdb-structure class=col-11><b>Crystal structure of MERS-CoV main protease</b><br>Apo structure of MERS-CoV Mpro protease in the C 2 2 21 space group at 1.90A resolution.<br><a href=https://www.rcsb.org/structure/8R5J>https://www.rcsb.org/structure/8R5J</a>
<cite>[ 2023-12-06 ]</cite><br><button data-component=toggleme data-target=#togglebox-protocol-ASAP-COV-MPRO-18 data-text="Hide Protocol">Protocol</button><div id=togglebox-protocol-ASAP-COV-MPRO-18 class=hide><cite><a href=http://doi.org/10.17504/protocols.io.ewov194x7lr2/v1>Available at protocols.io</a></cite></div></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href><img title=biochemical-assay width=40 src=../img/icons/biochemical-assay.png></a></div><div id=MERS-Mpro-protease_biochemical-assay class=col-11><b>Fluorescence assay for MERS-Mpro activity measurement</b><br>This method is intended to measure the activity of viral proteases by using a specific labelled-peptide that allows the detection of the cleaved product. The substrate contains the cleavage-sequence specific to the tested protease and is labeled in C-terminal by the fluorophore Edans (ex 336 nm; em: 455 nm) and in N-ternimal by the quencher Dabcyl (ex 472 nm). In the case of a non-cleaved substrate, the proximity of Dabcyl to Edans prevents the emission and the detection of the fluorescence at 455 nm. The cleavage of the peptide by the protease allows Edans’ fluorescence emission and detection.<br><cite>[ 2023-07-13 ]</cite><br><button data-component=toggleme data-target=#togglebox-protocol-ASAP-COV-MPRO-19 data-text="Hide Protocol">Protocol</button><div id=togglebox-protocol-ASAP-COV-MPRO-19 class=hide><cite><p><a href=http://doi.org/10.17504/protocols.io.8epv5rzm4g1b/v1>Available at protocols.io</a></p><p>Plate reader: Pherastar FS, BMG Labtech (Germany), 350-460 FI optic module, the plate is read every 30 s for 2 hours and shacked during 5 s before the first reading.
Method: Fifty µL of 2x protein solution were added to each well containing the compounds to be tested previously dispensed onto the plate. The mix has been incubated for one hour at room temperature and the enzymatic reaction was initiated by the addition of 50 µL of 2x (40 µM) substrate solution using the plate reader injector. The fluorescence intensity at 460 nm was read every 30 seconds for 2 hours in kinetic mode, which include a shaking step of the plate between each measurement.
The IC50 was calculated by plotting the initial velocity against various concentrations of tested inhibitor by using a four parameter dose−response curve in Prism (v8.0) software.</p></cite></div></div></div></div></div><hr style=height:5px;border:none;color:#333;background-color:#333><br><br><br><h3 style=text-align:center id=ASAP-SARS-COV-2-NS3-HELICASE>SARS-CoV-2 nsp13 helicase</h3><div class="row align-center"><a href=https://www.ncbi.nlm.nih.gov/protein/NSP13><span class="label success">Gene: NSP13 </span></a>&nbsp;
<a href=https://www.uniprot.org/uniprotkb/P0DTD1><span class="label error">Uniprot: P0DTD1 </span></a>&nbsp;
<a href="https://www.ebi.ac.uk/intenz/query?cmd=SearchEC&amp;ec=N/A"><span class="label warning">EC: N/A</span></a></div><br><div class="row align-center"><table class=unstyled><tr><td><img title="Crystallography construct" src=/images/icons/tep-stage-completed_hub021b18cb248db28b117f458e39aeb5b_10162_e22862b00861f0d3434857db1ff02de5.png width=300 height=88><br><img title="Assay construct" src=/images/icons/tep-stage-completed_hub021b18cb248db28b117f458e39aeb5b_10162_2851d035e13cbc115749c33624801624.png width=300 height=88></td><td><img title="Biophysical assay" src=/images/icons/tep-stage-not-started_hu0ad6bc3691333ead921c5ce1ee694314_9444_3996d997df11e427d2dc236cf2e2aca0.png width=300 height=55><br><img title="Robust crystals" src=/images/icons/tep-stage-in-progress_huc9d943f35f5911a48f54b3ad3b40c43a_10037_f71a74ae374dc1fa70d31362f03e7c29.png width=300 height=55><br><img title="Biochemical assay" src=/images/icons/tep-stage-in-progress_huc9d943f35f5911a48f54b3ad3b40c43a_10037_cf36f8cbd5eddd1cc66a738f8d6c493c.png width=300 height=55></td><td><img title="X-ray fragment screen and structural analysis" src=/images/icons/tep-stage-completed_hub021b18cb248db28b117f458e39aeb5b_10162_ae1969e18479b9f6892265d5fd14a303.png width=300 height=180></td><td><img title="First-round progression for TEP validation" src=/images/icons/tep-stage-not-started_hu0ad6bc3691333ead921c5ce1ee694314_9444_31ed385b113229ad572b9427160c8c98.png width=300 height=180></td></tr></table></div><b>Contributors:</b>
<span class="label success">Opher Gileadi </span><span class="label success">Joseph Newman </span><span class="label success">Yuliana Yosaatmadja </span><span class="label success">Alice Douangamath </span><span class="label success">Daren Fearon </span><span class="label success">Frank Von Delft</span><br><br><div class="prettybox warning" data-component=prettybox><b>Target Nominator:</b> Internal Nomination<br><b>Target PI:</b> Opher Gileadi<br><b>Therapeutic Area(s):</b> Infectious diseases<br><b>Disease Relevance:</b> nsp13 is a DNA/RNA helicase that is essential for SARS-CoV-2 replication, inhibitors to nsp13 could be developed as antiviral drugs.<br><b>Viral family:</b> coronaviridae<br><b>Viruses:</b> [SARS-CoV-2]<br></div><div class="prettybox success" data-component=prettybox><h4>Summary of Project</h4>To contribute towards the development of novel anti-viral therapeutics targeting the current and future emerging coronavirus threats, the Gileadi lab at the University of Oxford, together with the XChem team at Diamond Light Source, have teamed up to perform a crystallographic fragment screen against SARS-CoV-2 nsp13 helicase. nsp13 is believed to act in concert with the replication-transcription complex (NSP7/NSP82/NSP12), possibly being involved in either disrupting downstream RNA secondary structures or template switching, and plays an essential role in the life cycle of SARS-CoV-2. This TEP includes expression clones and methods for producing the full length nsp13, and fluorescencebased activity assays suitable for compound screening. We provide a crystallisation system that produces reproducible crystals that diffract to high resolution, and have performed a crystallographic fragment screen revealing 63 fragment hits across 51 datasets. The fragment hits include several hits in pockets predicted to be of functional importance, including the nucleotide and nucleic acid binding sites, opening the way to development of novel antiviral agents.</div><div class="prettybox focus" data-component=prettybox><h4>Scientific Background</h4>SARS-CoV-2 is the causative agent of the current global coronavirus (COVID-19) pandemic, a severe respiratory disease that emerged in the Chinese city of Wuhan in late 2019 [<a href=https://doi.org/10.1016/s0140-6736(20)30154-9>1</a>]. Coronaviruses belong to the order Nidovirales which have a positive strand RNA genome that is amongst the largest know RNA genomes (approximately 30 KB in length). The main SARS-CoV-2 genome encodes two open reading frames ORF1a and 1b, that when translated produce polyproteins that are processed by proteases into 16 non-structural proteins (nsp1-16)[<a href=https://doi.org/10.1080/22221751.2020.1719902>2</a>] that collectively form the machinery for viral replication and transcription. The nsp13 helicase is believed to act in concert with the replication-transcription complex (nsp7/nsp8/nsp12)[<a href=https://doi.org/10.1016%2Fj.cell.2020.07.033>3</a>] and is a critical component for viral replication [<a href=https://doi.org/10.1093/nar/gkz409>4</a>]. NSP13 is among the most conserved of the non-structural proteins, sharing only a single amino acid (V570I) difference with SARS-CoV-1. Thus, compounds targeting SARS-CoV-2 nsp13 would likely be effective against SARS-CoV-1 and potentially other future emerging coronaviruses, making it an ideal target for the development of new antiviral therapeutics. NSP13 is a large 67 kDa protein that belongs to the helicase superfamily 1B. It utilises the energy of nucleotide triphosphate hydrolysis to catalyse the unwinding of double stranded DNA or RNA in a 5’ to 3’ direction. nsp13 contains 5 domains, a N-terminal zinc binding domain that coordinates 3 structural zinc ions, a helical “stalk” domain, a beta-barrel 1B domain and two “RecA like” helicase subdomains 1A and 2A that contain the residues responsible for nucleotide binding and hydrolysis. Possible sites of inhibition of NSP13 include the nucleotide binding pocket, the DNA/RNA binding pocket and potentially allosteric pockets that may block domain movements that are required as part of the nsp13 catalytic cycle. Initial druggability analysis indicates both nucleotide and DNA/RNA binding pockets as being druggable and amongst the most well conserved pockets in the entire SARS-CoV-2 genome.</div><div class="prettybox error" data-component=prettybox><h4>Conclusion</h4>NSP13 is an important target for the development of new antiviral compounds targeting SARS-CoV-2 and other potential emerging viral threats. We describe a robust crystallisation system for nsp13 that produces crystals that routinely diffract to high resolution and have performed an X-ray fragment screen to identify initial binders. Of particular interest are the large number of hits in the nucleotide binding site and the DNA/RNA interface which may be developed into ATP or DNA competitive inhibitors. Alternatively, several pockets were discovered at interfaces between domains that could lead to allosteric inhibitors that prevent domain movements that occur as part of the helicase catalytic cycle.
Fragment screen PDB group deposition: <a href=https://www.rcsb.org/groups/summary/entry/G_1002164>https://www.rcsb.org/groups/summary/entry/G_1002164</a></div><h4>Resources</h4><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href=https://www.addgene.org/159614/><img title=plasmid width=40 src=../img/icons/plasmid.png></a></div><div id=cosv2-NSP13-helicase_plasmid class=col-11><b>Protein expression construct and expression and purification protocols for SARS-CoV-2 nsp13 helicase</b><br>We have expressed and purified full length NSP13 (1-601) in <em>E. coli</em> cells.<br><a href=https://www.addgene.org/159614/>https://www.addgene.org/159614/</a>
<cite>[ 2020-11-01 ]</cite><br><button data-component=toggleme data-target=#togglebox-protocol-ASAP-SARS-COV-2-NS3-HELICASE-1 data-text="Hide Protocol">Protocol</button><div id=togglebox-protocol-ASAP-SARS-COV-2-NS3-HELICASE-1 class=hide><cite>Medium: Terrific Broth (TB) Merck with 4 ml of glycerol Antibiotics: Kanamycin, 50 µg/ml From the glycerol stock, bacteria were inoculated in 15 ml of 1 x TB in a 50 ml tube with Kanamycin 0.05 mg/ml and 0.034 mg/ml of chloramphenicol and grown overnight in a shaker at 37°C, 250rpm. The following day, 40 ml of the overnight culture were inoculated in 4L of TB. The bacteria grew in an incubator at 37°C, shaking 180 rpm. Once the OD reached 2-3, IPTG (300uM) was added to the media and left overnight at 18°C, shaking 180 rpm. The pellets were harvested the next following day. The pellet (45g cell mass) was re-suspended in 200 ml lysis buffer (50 mM HEPES pH 7.5, 500 mM NaCl, 5% Gol, 10 mM Imidazole, 0.5 mM TCEP) with protease inhibitors (500 Merck set III). Cells were disrupted by sonication for 15 mins, 10sec on 5sec off, spin in JA25.5 24500 RPM for 30 mins. The supernatant was incubated for 40 mins with 5ml of Ni resin (IAMC sepharose) for batch binding. The tubes containing the lysate were centrifuged at 700 x g at 4°C for 5 minutes and the supernatant discarded. Beads were loaded on a gravity flow column and washed with 40 ml lysis buffer, 25 ml wash buffer (50 mM HEPES pH 7.5, 500 mM NaCl, 5% Gol, 45 mM imidazole, 0.5 mM TCEP). A further wash with 10 ml Hi-salt buffer (50 mM HEPES pH 7.5, 1 M NaCl, 5% Gol, 0.5 mM TCEP) and again with another 10 ml of wash buffer. Proteins were eluted with addition of 15ml of elution buffer (50 mM HEPES pH 7.5, 500 mM NaCl, 5% Gol, 300 mM imidazole, 0.5 mM TCEP). Elution fraction was immediately applied to a 5ml HItrap SP column using a syringe, collecting the flow through. The SP column was washed with 10ml elution buffer and proteins were eluted with 15ml Hi-salt buffer. The NSP13 protein was found to be present in flow-through, wash and elution fractions and these fractions were pooled and treated separately from this point onward. For further purification both proteins were incubated overnight with TEV protease (1:40 mass ratio) and loaded onto gel filtration systems using a superdex 200 16/60 column equilibrated in 50 mM Hepes, 500 mM NaCl, 0.5 mM TCEP. Both pools were found to crystallise with the majority of the crystals coming from the SP flow through and wash which had greater yield although slightly dirtier. Protein was concentrated to 20 mg /ml and diluted in half with water for initial crystallization trials at 10 mg/ml using a combination of screens and nucleotide combinations. Total yield was around 6 mg. The protein was confirmed by ESI-TOF intact mass spectrometry.</cite></div><button data-component=toggleme data-target=#togglebox-details-ASAP-SARS-COV-2-NS3-HELICASE-1 data-text="Hide Details">Details</button><div id=togglebox-details-ASAP-SARS-COV-2-NS3-HELICASE-1 class=hide><ul><li>Clone Source: Synthetic Gene (Twist Biosciences)</li><li>SGC Construct ID: CVNSP13A-c000 (<a href=https://www.addgene.org/159614/>https://www.addgene.org/159614/</a>)</li><li>Vector: pNIC-ZB (GenBank: GU452710.1)</li><li>Tag: N-terminal 6HIS, Z-basic tag with TEV cleavage</li><li>Host: BL21(DE3)-R3-pRARE2</li><li>Construct protein sequence:<br>Sequence with tag: MHHHHHHSSGVDNKFNKERRRARREIRHLPNLNREQRRAFIRSLRDDPSQSANLLAEAKKLNDAQPKGTENLYFQ*SMAV
GACVLCNSQTSLRCGACIRRPFLCCKCCYDHVISTSHKLVLSVNPYVCNAPGCDVTDVTQLYLGGMSYYCKSHKPPISFPLCAN
GQVFGLYKNTCVGSDNVTDFNAIATCDWTNAGDYILANTCTERLKLFAAETLKATEETFKLSYGIATVREVLSDRELHLSWEV
GKPRPPLNRNYVFTGYRVTKNSKVQIGEYTFEKGDYGDAVVYRGTTTYKLNVGDYFVLTSHTVMPLSAPTLVPQEHYVRITGL
YPTLNISDEFSSNVANYQKVGMQKYSTLQGPPGTGKSHFAIGLALYYPSARIVYTACSHAAVDALCEKALKYLPIDKCSRIIPAR
ARVECFDKFKVNSTLEQYVFCTVNALPETTADIVVFDEISMATNYDLSVVNARLRAKHYVYIGDPAQLPAPRTLLTKGTLEPEY
FNSVCRLMKTIGPDMFLGTCRRCPAEIVDTVSALVYDNKLKAHKDKSAQCFKMFYKGVITHDVSSAINRPQIGVVREFLTRNP
AWRKAVFISPYNSQNAVASKILGLPTQTVDSSQGSEYDYVIFTQTTETAHSCNVNRFNVAITRAKVGILCIMSDRDLYDKLQFT
SLEIPRRNVATLQ<br>Sequence after tag cleavage: SMAVGACVLCNSQTSLRCGACIRRPFLCCKCCYDHVISTSHKLVLSVNPYVCNAPGCDVTDVTQLYLGGMSYYCKSHKPPISF
PLCANGQVFGLYKNTCVGSDNVTDFNAIATCDWTNAGDYILANTCTERLKLFAAETLKATEETFKLSYGIATVREVLSDRELHL
SWEVGKPRPPLNRNYVFTGYRVTKNSKVQIGEYTFEKGDYGDAVVYRGTTTYKLNVGDYFVLTSHTVMPLSAPTLVPQEHYV
RITGLYPTLNISDEFSSNVANYQKVGMQKYSTLQGPPGTGKSHFAIGLALYYPSARIVYTACSHAAVDALCEKALKYLPIDKCSR
IIPARARVECFDKFKVNSTLEQYVFCTVNALPETTADIVVFDEISMATNYDLSVVNARLRAKHYVYIGDPAQLPAPRTLLTKGTL
EPEYFNSVCRLMKTIGPDMFLGTCRRCPAEIVDTVSALVYDNKLKAHKDKSAQCFKMFYKGVITHDVSSAINRPQIGVVREFL
TRNPAWRKAVFISPYNSQNAVASKILGLPTQTVDSSQGSEYDYVIFTQTTETAHSCNVNRFNVAITRAKVGILCIMSDRDLYD
KLQFTSLEIPRRNVATLQ<br>DNA Sequence (codon-optimized for E. coli expression): ATGCACCATCATCATCATCATTCTTCTGGTGTGGATAACAAGTTCAACAAGGAGCGTCGAAGAGCTCGCCGTGAAATTC
GCCATCTGCCGAACCTGAACCGCGAACAGCGTCGCGCATTTATTCGCAGCCTGCGCGATGATCCGAGCCAGAGCGCGA
ACCTGCTGGCGGAAGCGAAGAAGCTGAACGATGCGCAGCCGAAGGGTACCGAGAACCTGTACTTCCAATCCATGGCT
GTTGGTGCATGCGTTTTGTGTAATAGTCAAACTTCTTTGCGTTGCGGCGCCTGCATCCGCCGCCCATTTCTGTGCTGTAA
ATGTTGCTACGATCATGTGATTAGCACGTCGCATAAGCTGGTGCTGTCGGTCAACCCGTATGTTTGTAATGCGCCCGGT
TGTGACGTCACGGATGTTACCCAGCTCTATCTGGGTGGTATGTCTTACTACTGCAAGTCACATAAGCCGCCGATTTCATT
TCCTCTCTGCGCGAATGGCCAGGTGTTTGGTTTGTATAAAAACACCTGCGTGGGGTCTGATAATGTAACCGATTTTAAC
GCGATTGCTACGTGTGACTGGACCAACGCCGGCGATTATATCCTTGCCAACACCTGCACTGAACGTCTTAAACTCTTCGC
GGCCGAAACCCTGAAAGCAACCGAAGAAACGTTCAAACTGAGCTATGGTATCGCGACAGTACGCGAAGTACTCAGTGA
CCGGGAGTTGCACCTGTCGTGGGAAGTCGGTAAACCTCGCCCCCCGTTAAACCGCAACTACGTGTTCACCGGCTATCGC
GTTACCAAAAACAGCAAGGTGCAAATTGGTGAGTACACCTTTGAGAAAGGCGACTATGGTGATGCCGTAGTATACCGC
GGCACCACGACTTACAAACTGAATGTGGGTGATTATTTTGTATTAACCTCACATACCGTAATGCCGTTGAGCGCCCCCAC
CCTCGTACCGCAGGAACACTATGTACGTATCACGGGTTTATACCCGACGCTGAACATCAGCGATGAGTTTAGCAGTAAC
GTTGCTAACTACCAGAAAGTGGGTATGCAAAAATACTCGACGTTACAGGGCCCGCCAGGTACTGGAAAGTCTCATTTCG
CCATCGGCTTAGCCTTATACTATCCATCAGCTCGCATTGTTTACACGGCCTGCTCTCATGCTGCAGTGGACGCCTTATGC
GAGAAGGCGTTAAAATATCTGCCGATTGATAAATGCTCCCGTATTATCCCGGCGCGGGCGCGGGTCGAGTGTTTTGAC
AAATTTAAAGTGAACAGCACACTGGAACAGTATGTTTTTTGTACGGTGAATGCTCTCCCAGAGACAACTGCCGATATCG
TCGTGTTCGACGAAATCAGCATGGCCACTAACTACGATCTGTCGGTCGTTAACGCGCGTCTCCGCGCTAAGCATTATGTT
TACATTGGCGATCCGGCCCAGCTGCCGGCTCCACGCACGCTCCTGACTAAAGGTACTCTGGAACCGGAATATTTTAATA
GCGTATGCCGTCTGATGAAAACGATTGGACCTGATATGTTCCTGGGAACTTGCCGCCGGTGTCCGGCCGAGATCGTTG
ATACGGTTTCTGCTCTGGTTTATGACAATAAATTAAAAGCTCACAAAGATAAAAGCGCGCAATGCTTCAAAATGTTTTAT
AAAGGTGTAATTACCCACGATGTGAGTAGCGCGATTAACCGCCCGCAAATTGGCGTTGTCCGTGAATTCCTGACCCGGA
ATCCTGCGTGGCGCAAAGCAGTGTTCATTAGCCCATATAATAGTCAAAACGCGGTAGCGTCCAAGATTTTGGGTCTGCC
GACACAAACCGTGGACTCCAGCCAGGGCAGTGAGTACGACTATGTGATTTTTACCCAGACCACTGAGACAGCACACTCC
TGCAACGTGAACCGCTTCAATGTAGCGATCACCCGTGCGAAGGTAGGTATTTTGTGCATCATGTCCGACCGTGATCTGT
ATGACAAATTGCAGTTTACCAGCCTGGAAATCCCTCGCCGTAACGTGGCAACCCTTCAATAACAGTAAAGGTGGATACG
GATCCGAATTCGAGCTCCGTCGACAAGCTTGCGGCCGCACTCGAGCACCACCACCACCACCACTGA</li></ul></div></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href=https://www.rcsb.org/structure/7NIO><img title=structure width=40 src=../img/icons/structure.png></a></div><div id=cosv2-NSP13-PDB_structure-7NIO class=col-11><b>X-ray structure of SARS-CoV-2 nsp13 helicase</b><br>The NSP13 crystal form contains two protomers in the P1 spacegroup and diffracts to 2.2 Å.<br><a href=https://www.rcsb.org/structure/7NIO>https://www.rcsb.org/structure/7NIO</a>
<cite>[ 2020-09-16 ]</cite><br><button data-component=toggleme data-target=#togglebox-protocol-ASAP-SARS-COV-2-NS3-HELICASE-2 data-text="Hide Protocol">Protocol</button><div id=togglebox-protocol-ASAP-SARS-COV-2-NS3-HELICASE-2 class=hide><cite>Initial diffracting crystals were found in the Morpheus screen from Molecular dimensions for the APO form of the protein at 10 mg/ml. Initial crystals grew at 20 degrees from conditions containing 20 % Ethylene Glycol, 10 % PEG 8K, 0.05 M HEPES, 0.05 M MOPS, 0.03 M sodium nitrate, 0.03 M sodium phosphate, 0.03 M ammonium sulphate. For crystal optimization seeding was performed, around 5-10 crystals were crushed with a glass probe and transferred to 25 μl of well solution. A seed bead was added, and the mixture was sonicated for around 30-60 seconds with pulsing. Final seeding was performed with a 1 in 400 dilution of seed stock. Final plates were set up with protein at 5 mg/ml (diluted 4 fold in water from 20 mg/ml stock) with a slightly reduced precipitant concentration (16 % ethylene glycol, 8 % PEG 8K, 0.05 M HEPES, 0.05 M MOPS, 0.03 M sodium nitrate, 0.03 M sodium phosphate, 0.03 M ammonium sulphate), using 300 nl drops (1:1 ratio) with 20 nl seeds (added last). Data Collection: Data were collected to 1.94Å resolution at Diamond light source beamline I04-1 and processed using XDS. Data Processing: The structure was solved by molecular replacement using the program PHASER and the structure of SARS-CoV-1 NSP13 (6JYT) as a search model. Refinement was performed using PHENIX REFINE to a final Rfactor = 20.9%, Rfree = 25.3%.</cite></div></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href=https://www.rcsb.org/groups/summary/entry/G_1002164><img title=pdb-group-deposition width=40 src=../img/icons/pdb-group-deposition.png></a></div><div id=cosv2-NSP13-PDB_fragment-screen-pdb-group-deposition class=col-11><b>PDB group deposition of X-ray fragment screen of SARS-CoV-2 nsp13 helicase</b><br>The NSP13 crystal form contains two protomers and the fragment screen revealed a total of 63 fragment hits across 51 datasets. These structures have been published in the PDB and are featured as a curated collection in the <a href=https://www.rcsb.org/groups/summary/entry/G_1002164>RCSB PDB SARS-CoV-2</a> resource.<br><a href=https://www.rcsb.org/groups/summary/entry/G_1002164>https://www.rcsb.org/groups/summary/entry/G_1002164</a>
<cite>[ 2020-09-16 ]</cite><br><button data-component=toggleme data-target=#togglebox-protocol-ASAP-SARS-COV-2-NS3-HELICASE-3 data-text="Hide Protocol">Protocol</button><div id=togglebox-protocol-ASAP-SARS-COV-2-NS3-HELICASE-3 class=hide><cite>Fragments from the DSI-Poised library were added to the crystallisation drops by acoustic dispensing using an ECHO acoustic liquid handler from a 500 mM stock concentration dissolved in DMSO to a final concentration of 10%. Soaking times varied from 1 to 3 hours. Data Collection: Data were collected at Diamond light source beamline I04-1 and processed using the XChem Explorer pipeline. Data Processing: Structures were solved by difference Fourier synthesis using the XChem Explorer pipeline. Fragment hits were identified using the PanDDA program. Refinement was performed using REFMAC or BUSTER.</cite></div></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href=https://fragalysis.diamond.ac.uk/viewer/react/preview/target/nsp13><img title=xray-fragment-screen width=40 src=../img/icons/xray-fragment-screen.png></a></div><div id=cosv2-NSP13-PDB_fragment-screen-fragalysis class=col-11><b>Fragalysis interactive exploration of X-ray fragment screen of SARS-CoV-2 nsp13 helicase</b><br>The NSP13 crystal form contains two protomers and the fragment screen revealed a total of 63 fragment hits across 51 datasets. These structures have been published in the PDB and are featured as a curated collection in the <a href=https://www.rcsb.org/groups/summary/entry/G_1002164>RCSB PDB SARS-CoV-2</a> resource.<br><a href=https://fragalysis.diamond.ac.uk/viewer/react/preview/target/nsp13>https://fragalysis.diamond.ac.uk/viewer/react/preview/target/nsp13</a>
<cite>[ 2020-09-16 ]</cite><br><button data-component=toggleme data-target=#togglebox-protocol-ASAP-SARS-COV-2-NS3-HELICASE-4 data-text="Hide Protocol">Protocol</button><div id=togglebox-protocol-ASAP-SARS-COV-2-NS3-HELICASE-4 class=hide><cite>Fragments from the DSI-Poised library were added to the crystallisation drops by acoustic dispensing using an ECHO acoustic liquid handler from a 500 mM stock concentration dissolved in DMSO to a final concentration of 10%. Soaking times varied from 1 to 3 hours. Data Collection: Data were collected at Diamond light source beamline I04-1 and processed using the XChem Explorer pipeline. Data Processing: Structures were solved by difference Fourier synthesis using the XChem Explorer pipeline. Fragment hits were identified using the PanDDA program. Refinement was performed using REFMAC or BUSTER.</cite></div></div></div></div></div><hr style=height:5px;border:none;color:#333;background-color:#333><br><br><br><h3 style=text-align:center id=ASAP-SARS-CoV-2-endoribonuclease>SARS-CoV-2 nsp15 endoribonuclease</h3><div class="row align-center"><a href="https://www.ncbi.nlm.nih.gov/protein/%25!s%28int=43740578%29"><span class="label success">Gene: 43740578 </span></a>&nbsp;
<a href=https://www.uniprot.org/uniprotkb/P0DTD1><span class="label error">Uniprot: P0DTD1 </span></a>&nbsp;
<a href="https://www.ebi.ac.uk/intenz/query?cmd=SearchEC&amp;ec=2.7.7.48"><span class="label warning">EC: 2.7.7.48</span></a></div><br><div class="row align-center"><table class=unstyled><tr><td><img title="Crystallography construct" src=/images/icons/tep-stage-completed_hub021b18cb248db28b117f458e39aeb5b_10162_e22862b00861f0d3434857db1ff02de5.png width=300 height=88><br><img title="Assay construct" src=/images/icons/tep-stage-completed_hub021b18cb248db28b117f458e39aeb5b_10162_2851d035e13cbc115749c33624801624.png width=300 height=88></td><td><img title="Biophysical assay" src=/images/icons/tep-stage-not-started_hu0ad6bc3691333ead921c5ce1ee694314_9444_3996d997df11e427d2dc236cf2e2aca0.png width=300 height=55><br><img title="Robust crystals" src=/images/icons/tep-stage-completed_hub021b18cb248db28b117f458e39aeb5b_10162_e3383ab8d75acf7a535bdc511b331b23.png width=300 height=55><br><img title="Biochemical assay" src=/images/icons/tep-stage-in-progress_huc9d943f35f5911a48f54b3ad3b40c43a_10037_cf36f8cbd5eddd1cc66a738f8d6c493c.png width=300 height=55></td><td><img title="X-ray fragment screen and structural analysis" src=/images/icons/tep-stage-completed_hub021b18cb248db28b117f458e39aeb5b_10162_ae1969e18479b9f6892265d5fd14a303.png width=300 height=180></td><td><img title="First-round progression for TEP validation" src=/images/icons/tep-stage-not-started_hu0ad6bc3691333ead921c5ce1ee694314_9444_31ed385b113229ad572b9427160c8c98.png width=300 height=180></td></tr></table></div><b>Contributors:</b>
<span class="label success">Andre Schutzer Godoy </span><span class="label success">Tobias Krojer </span><span class="label success">Aline M. Nakamura </span><span class="label success">Alice Douangamath </span><span class="label success">Gabriela D. Noske </span><span class="label success">Victor O. Gawrijuk </span><span class="label success">Rafaela S. Fernandes </span><span class="label success">Glaucius Oliva </span><span class="label success">Daren Fearon </span><span class="label success">Frank von Delft</span><br><br><div class="prettybox warning" data-component=prettybox><b>Target Nominator:</b> Andre Schutzer Godoy<br><b>Target PI:</b> Frank von Delft<br><b>Therapeutic Area(s):</b> Infectious diseases<br><b>Disease Relevance:</b> nsp15 is a nidoviral RNA uridylate‐specific endoribonuclease (NendoU) from SARS-CoV-2, likely involved in innate immune evasion<br><b>Viral family:</b> coronaviridae<br><b>Viruses:</b> [SARS-CoV-2]<br></div><div class="prettybox success" data-component=prettybox><h4>Summary of Project</h4>The non-structural protein 15 (NSP15, NendoU SARS-CoV-2) from severe acute respiratory syndrome 2 virus (SARS-CoV-2) is an uridylate-specific endoribonuclease, likely responsible in the viral immune evasion mechanism. This TEP provides a set of reagents for further interrogation of the molecular function of NSP15. We have established a purification protocol for the active protein for biochemical and structural studies. Moreover, we have crystallised the protein and performed a crystallographic fragment screen which yielded several hits. Data generated here will be used for the development of enzyme inhibitors that would illuminate the biological role of the gene product, and eventually point the way to new antiviral therapies.</div><div class="prettybox focus" data-component=prettybox><h4>Scientific Background</h4>SARS-CoV-2 is the causative agent of COVID-19. NSP15 from SARS-CoV-2 is a nidoviral RNA uridylate-specific endoribonuclease that is part of the EndoU family [<a href=https://doi.org/10.1002/pro.3873>1</a>]. Members of the family act on RNA processing being active on both single-stranded RNA and double-stranded (ds) RNA, cleaving 3’of urydilates thereby generating a 2’, 3’ cyclic phosphate and 5’-hydroxyl termini [<a href=https://doi.org/10.4161/rna.8.2.15196>2</a>]. NSP15 domains are highly conserved amongst all nidoviruses and have not been identified in other RNA virus families, thus qualifying them as genetic markers of this order [<a href=https://doi.org/10.1016/j.virusres.2006.01.017>3</a>, <a href=https://doi.org/10.1073/pnas.0403127101>4</a>]. Initially, NSP15 was thought to be required for the viral replication process, acting on viral RNA synthesis. However, studies have revealed that catalytic mutant viruses for NSP15 replicate as well as wild-type virus [<a href=https://doi.org/10.1371/journal.ppat.1006195>5</a>]. Subsequently, it was revealed that NSP15 acts by limiting the sensing of viral RNA by the host, inhibiting the innate immune response, like type I interferon response in macrophages during viral infection by evading dsRNA sensors (such as MDA5, PKR and OAS/RNaseL) [<a href=https://doi.org/10.1371/journal.ppat.1006195>5</a>, <a href=https://doi.org/10.1073/pnas.1618310114>6</a>, <a href=https://doi.org/10.1073/pnas.1921485117>7</a>]. The potential of NSP15 as a target for direct acting antiviral therapies has not been yet fully explored.</div><div class="prettybox error" data-component=prettybox><h4>Conclusion</h4>We have produced active recombinant NSP15 protein that could be crystallised and used for structure determination at high resolution. We also obtained data from the NSP15 in complex with several fragments, which should provide valuable information for the development of a new enzyme inhibitor. Furthermore, we established a stable assay that can be used for further enzyme characterization.</div><h4>Resources</h4><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href><img title=plasmid width=40 src=../img/icons/plasmid.png></a></div><div id=ccosv2-NSP15-endou_plasmid-crystallography class=col-11><b>Protein expression construct for SARS-CoV-2 nsp15 endoribonuclease suitable for X-ray crystallography</b><br>NSP15 is a346 amino acid long polypeptide of approximately 38 kDa that forms a three-domain protein. These monomers are known to self-assemble as a barrel-shaped hexamer with D3 symmetry, totalling 228 kDa. For this project, we recombinantly purified NSP15 in its monomeric form. The mechanism involved in hexamerisation in-vitro and in-situ is still under investigation. Although recombinant protein obtained was monomeric, crystal packing revealed that NSP15 formed the expected hexamer during crystallization. The active site of NSP15 [<a href=https://doi.org/10.4161/rna.8.2.15196>2</a>] was found to be in complex with citrate, which is a component of the crystallisation condition. We have produced active recombinant NSP15 protein that could be crystallised and used for structure determination at high resolution.<br><cite>[ 2020-11-01 ]</cite><br><button data-component=toggleme data-target=#togglebox-protocol-ASAP-SARS-CoV-2-endoribonuclease-1 data-text="Hide Protocol">Protocol</button><div id=togglebox-protocol-ASAP-SARS-CoV-2-endoribonuclease-1 class=hide><cite>SARS-CoV-2 cDNA sample was donated by Dr. Edilson Durigon (GenBank MT126808.1), generated using SCRIPT One-Step RT-PCR (Cellco Biotec) and random hexamer primers. NSP15 coding sequence was amplified using FastPol (Cellco Biotec) with primers 5’-CAGGGCGCCATGAGTTTAGAAAATGTGGCTTTTAATG-3’ and 5’-GACCCGACGCGGTTATTGTAATTTTGGGTAAAATGTTTCTAC-3’, and inserted pET28-LIC (donated by Dr. Arie Geerlof) using Ligation-Independent Cloning method (8). The final construct pET28-LIC -NSP15 included a N-terminal 6xHis-tag followed by a TEV cleaving site and residues 1-346 of NSP15. Buffers used are detailed hereafter: Buffer A: 50 mM TRIS-HCl pH 8.0, 300 mM NaCl, 10% glycerol, 20 mM imidazole Buffer B: 50 mM TRIS-HCl pH 8.0, 300 mM NaCl, 10% glycerol, 300 mM imidazole Buffer C: 20 mM HEPES pH 7.5, 150 mM NaCl, 5% (v/v) glycerol, 0.5 mM TCEP Transformed <em>BL21(DE3)</em> cells were grown in LB Lennox medium supplemented with kanamycin at 37^o^C until OD~600~ = 1.0, and kept for 16h at 18^o^C after induction with 0.5 mM isopropyl β-D-1-thiogalactopyranoside. Harvested cells were resuspended in Buffer A supplemented with 0.1 mg/mL lysozyme, 10 U of benzonase (Cellco Biotec) and 1.0 mM DTT, and disrupted using sonication. Cleared lysate was passed through a HisTrap HP 5 mL (GE Healthcare) equilibrated in Buffer A, and them protein was eluted with Buffer B. Excessive imidazole was removed using a Sephadex XK 26/60 column (GE Healthcare) equilibrated with Buffer A. Sample was incubated overnight at 8^o^C with 4 mM DTT and 0.1 mg/mL TEV protease, then passed through a HisTrap HP 5 mL equilibrated in Buffer A to remove non-cleaved protein and TEV protease. Final purification was performed by size exclusion chromatography using a HiLoad Superdex 75 16/60 (GE Healthcare) equilibrated with Buffer C. Purity was confirmed by SDS-PAGE 12%. Protein concentration was estimated using the theoretical extinction coefficient at 280 nm of 33,140 M^-1^.cm^-1^. The eluted protein was pooled and concentrated to 3.4 mg/mL using a 30 kDa MWCO concentrator.</cite></div><button data-component=toggleme data-target=#togglebox-details-ASAP-SARS-CoV-2-endoribonuclease-1 data-text="Hide Details">Details</button><div id=togglebox-details-ASAP-SARS-CoV-2-endoribonuclease-1 class=hide><ul><li>Vector: pET28-LIC</li><li>Entry clone accession: MT126808.1</li><li>Cell line: E. coli BL21(DE3)</li><li>Tags and additions: N-terminal, TEV protease cleavable hexahistidine tag Construct protein sequence</li><li>(underlined sequence contains vector encoded His-tag and TEV protease cleavage site*):</li><li>NendoUSARS-CoV-2</li><li>(aa 1-346)</li><li>Construct protein sequence:
<code>MKHHHHHHPMSDYDIPTTENLYFQ*GAMSLENVAFNVVNKGHFDGQQGEVPVSIINNTVYTKVDGVDVELFENKTTLPVN VAFELWAKRNIKPVPEVKILNNLGVDIAANTVIWDYKRDAPAHISTIGVCSMTDIAKKPTETICAPLTVFFDGRVDGQVDLFR NARNGVLITEGSVKGLQPSVGPKQASLNGVTLIGEAVKTQFNYYKKVDGVVQQLPETYFTQSRNLQEFKPRSQMEIDFLELA MDEFIERYKLEGYAFEHIVYGDFSHSQLGGLHLLIGLAKRFKESPFELEDFIPMDSTVKNYFITDAQTGSSKCVCSVIDLLLDDFV EIIKSQDLSVVSKVVKVTIDYTEISFMLWCKDGHVETFYPKLQ</code></li></ul></div></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href=https://www.rcsb.org/groups/summary/entry/G_1002199><img title=pdb-group-deposition width=40 src=../img/icons/pdb-group-deposition.png></a></div><div id=ccosv2-NSP15-endou_fragment-screen-pdb-group-deposition class=col-11><b>PDB group deposition of X-ray fragment screen of SARS-CoV-2 nsp15 endoribonuclease</b><br>The crystallographic fragment discovery campaign using the XChem platform revealed chemically distinct fragments that could bind to NSP15. Fragment screening hits were found all over the accessible nsp15 surface area. N.B. the asymmetric unit of nsp15 crystals contains two nsp15 protomers, but some fragments were only bound to one protomer.<br><a href=https://www.rcsb.org/groups/summary/entry/G_1002199>https://www.rcsb.org/groups/summary/entry/G_1002199</a>
<cite>[ 2020-11-01 ]</cite><br><button data-component=toggleme data-target=#togglebox-protocol-ASAP-SARS-CoV-2-endoribonuclease-2 data-text="Hide Protocol">Protocol</button><div id=togglebox-protocol-ASAP-SARS-CoV-2-endoribonuclease-2 class=hide><cite>Crystallization protocol <a href=dx.doi.org/10.17504/protocols.io.36wgqn4yogk5/v1>available at protocols.io</a> For soaking, 40 nL of each fragment compound from the OPEN-EU DRIVE fragment library (library concentration of 100 mM) was added to a crystallisation drop using an ECHO acoustic liquid handler dispenser at the Diamond light source XChem facility [<a href=https://doi.org/10.1107/s205979831700331x>9</a>]. Crystals were soaked for two hours before being harvested using XChem SHIFTER technology, and data was collected at the beamline I04-1 in automated mode. The XChemExplorer pipeline [<a href=https://doi.org/10.1107%2FS2059798316020234>10</a>] was used for structure solution with parallel molecular replacement using DIMPLE [<a href=https://www.semanticscholar.org/paper/DIMPLE-a-pipeline-for-the-rapid-generation-of-maps-Wojdyr-Keegan/c715d7fd1fae575c9f80d540f4130c3011cb447d>11</a>]. Fragments were identified using PANDDA software [<a href=https://doi.org/10.1038/ncomms15123>12</a>]. Data was modelled and refined using BUSTER and COOT [<a href=https://doi.org/10.1107/s0907444910007493>13</a>], and validated using Molprobity [<a href=https://doi.org/10.1107/s0907444909042073>14</a>]. Coordinates and structure factors for exemplary data sets with bound fragments are deposited in the RCSB Protein Data Bank.</cite></div></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href=https://fragalysis.diamond.ac.uk/viewer/react/preview/target/NSP15_B><img title=xray-fragment-screen width=40 src=../img/icons/xray-fragment-screen.png></a></div><div id=ccosv2-NSP15-endou_fragment-screen-fragalysis class=col-11><b>Fragalysis interactive exploration of X-ray fragment screen of SARS-CoV-2 nsp15 endoribonuclease</b><br>The crystallographic fragment discovery campaign using the XChem platform revealed chemically distinct fragments that could bind to NSP15. Fragment screening hits were found all over the accessible nsp15 surface area. N.B. the asymmetric unit of nsp15 crystals contains two nsp15 protomers, but some fragments were only bound to one protomer.<br><a href=https://fragalysis.diamond.ac.uk/viewer/react/preview/target/NSP15_B>https://fragalysis.diamond.ac.uk/viewer/react/preview/target/NSP15_B</a>
<cite>[ 2020-11-01 ]</cite><br><button data-component=toggleme data-target=#togglebox-protocol-ASAP-SARS-CoV-2-endoribonuclease-3 data-text="Hide Protocol">Protocol</button><div id=togglebox-protocol-ASAP-SARS-CoV-2-endoribonuclease-3 class=hide><cite>For the fragment screening campaign, 30 nL protein (3.4 mg/mL) and 30 nL well solution containing 14% (w/v) PEG 6,000 and 0.1 M tri-sodium citrate pH 5.0 were equilibrated at 20^o^C by vapor diffusion in the presence of 10 nL of seeds stocks. For soaking, 40 nL of each fragment compound from the OPEN-EU DRIVE fragment library (library concentration of 100 mM) was added to a crystallisation drop using an ECHO acoustic liquid handler dispenser at the Diamond light source XChem facility [<a href=https://doi.org/10.1107/s205979831700331x>9</a>]. Crystals were soaked for two hours before being harvested using XChem SHIFTER technology, and data was collected at the beamline I04-1 in automated mode. The XChemExplorer pipeline [<a href=https://doi.org/10.1107%2FS2059798316020234>10</a>] was used for structure solution with parallel molecular replacement using DIMPLE [<a href=https://www.semanticscholar.org/paper/DIMPLE-a-pipeline-for-the-rapid-generation-of-maps-Wojdyr-Keegan/c715d7fd1fae575c9f80d540f4130c3011cb447d>11</a>]. Fragments were identified using PANDDA software [<a href=https://doi.org/10.1038/ncomms15123>12</a>]. Data was modelled and refined using BUSTER and COOT [<a href=https://doi.org/10.1107/s0907444910007493>13</a>], and validated using Molprobity [<a href=https://doi.org/10.1107/s0907444909042073>14</a>]. Coordinates and structure factors for exemplary data sets with bound fragments are deposited in the RCSB Protein Data Bank.</cite></div></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href><img title=biochemical-assay width=40 src=../img/icons/biochemical-assay.png></a></div><div id=ccosv2-NSP15-endou_biochemical-assay class=col-11><b>Biochemical assay for SARS-CoV-2 nsp15 endoribonuclease activity using FRET</b><br><cite>[ 2020-11-01 ]</cite><br><button data-component=toggleme data-target=#togglebox-protocol-ASAP-SARS-CoV-2-endoribonuclease-4 data-text="Hide Protocol">Protocol</button><div id=togglebox-protocol-ASAP-SARS-CoV-2-endoribonuclease-4 class=hide><cite>Fluorogenic oligonucleotide substrate 5'6-FAM-dArUdAdA-6-TAMRA3’ was purchased from GenScript. Activity assays were performed in 25 mM BIS-Tris-HCl buffer 6.0 pH, 0.25 μM substrate and 100 nM NSP15mon at 30 C. Time-course reactions were directly monitored in a Stratagene Mx3005P (Agilent Technologies) using FAM filters.</cite></div></div></div></div></div><hr style=height:5px;border:none;color:#333;background-color:#333><br><br><br><h3 style=text-align:center id=ASAP-SARS-COV-2-NSP3-MAC1>SARS-CoV-2 nsp3 Mac1 macrodomain</h3><div class="row align-center"><a href=https://www.ncbi.nlm.nih.gov/protein/QIT08255.1><span class="label success">Gene: QIT08255.1 </span></a>&nbsp;
<a href=https://www.uniprot.org/uniprotkb/P0DTC1><span class="label error">Uniprot: P0DTC1 </span></a>&nbsp;
<a href="https://www.ebi.ac.uk/intenz/query?cmd=SearchEC&amp;ec=3.4.19.12"><span class="label warning">EC: 3.4.19.12</span></a></div><br><div class="row align-center"><table class=unstyled><tr><td><img title="Crystallography construct" src=/images/icons/tep-stage-completed_hub021b18cb248db28b117f458e39aeb5b_10162_e22862b00861f0d3434857db1ff02de5.png width=300 height=88><br><img title="Assay construct" src=/images/icons/tep-stage-completed_hub021b18cb248db28b117f458e39aeb5b_10162_2851d035e13cbc115749c33624801624.png width=300 height=88></td><td><img title="Biophysical assay" src=/images/icons/tep-stage-completed_hub021b18cb248db28b117f458e39aeb5b_10162_05cf9afc49ba5efb1c08e4e2e8fbaf47.png width=300 height=55><br><img title="Robust crystals" src=/images/icons/tep-stage-completed_hub021b18cb248db28b117f458e39aeb5b_10162_e3383ab8d75acf7a535bdc511b331b23.png width=300 height=55><br><img title="Biochemical assay" src=/images/icons/tep-stage-completed_hub021b18cb248db28b117f458e39aeb5b_10162_1b7ab3b95a075266c44f73afb56d1cd3.png width=300 height=55></td><td><img title="X-ray fragment screen and structural analysis" src=/images/icons/tep-stage-completed_hub021b18cb248db28b117f458e39aeb5b_10162_ae1969e18479b9f6892265d5fd14a303.png width=300 height=180></td><td><img title="First-round progression for TEP validation" src=/images/icons/tep-stage-completed_hub021b18cb248db28b117f458e39aeb5b_10162_5f535c81547d74d3984d598f890911a7.png width=300 height=180></td></tr></table></div><b>Contributors:</b>
<span class="label success">Marion Schuller </span><span class="label success">Galen J. Correy </span><span class="label success">Stefan Gahbauer </span><span class="label success">Blake Balcomb </span><span class="label success">Charlie Tomlinson </span><span class="label success">Daren Fearon </span><span class="label success">Dong Wang </span><span class="label success">Eleanor Williams </span><span class="label success">Jasmin Aschenbrenner </span><span class="label success">Jessica Dixon </span><span class="label success">Korvus Wang </span><span class="label success">Lee Harris </span><span class="label success">Lizbé Koekemoer </span><span class="label success">Michael Fairhead </span><span class="label success">Peter Marples </span><span class="label success">Ryan Lithgo </span><span class="label success">Warren Thompson </span><span class="label success">Taiasean Wu </span><span class="label success">Roberto Efraín Díaz </span><span class="label success">Iris D. Young </span><span class="label success">Luan Carvalho Martins, Dominique H. Smith </span><span class="label success">Ursula Schulze-Gahmen </span><span class="label success">Tristan W. Owens </span><span class="label success">Ishan Deshpande </span><span class="label success">Gregory E. Merz </span><span class="label success">Aye C. Thwin </span><span class="label success">Justin T. Biel </span><span class="label success">Jessica K. Peters </span><span class="label success">Michelle Moritz </span><span class="label success">Nadia Herrera </span><span class="label success">Huong T. Kratochvil </span><span class="label success">Anthony Aimon </span><span class="label success">James M. Bennett </span><span class="label success">Jose Brandao Neto </span><span class="label success">Aina E. Cohen </span><span class="label success">Alexandre Dias </span><span class="label success">Alice Douangamath </span><span class="label success">Louise Dunnett </span><span class="label success">Oleg Fedorov </span><span class="label success">Matteo P. Ferla </span><span class="label success">Martin Fuchs </span><span class="label success">Tyler J. Gorrie-Stone </span><span class="label success">James M. Holton </span><span class="label success">Michael G. Johnson, </span><span class="label success">George Meigs </span><span class="label success">Ailsa J. Powell </span><span class="label success">Johannes Gregor </span><span class="label success">Matthias Rack </span><span class="label success">Victor L Rangel </span><span class="label success">Silvia Russi </span><span class="label success">Rachael E. Skyner </span><span class="label success">Clyde A. Smith </span><span class="label success">Alexei S. Soares </span><span class="label success">Jennifer L. Wierman </span><span class="label success">Kang Zhu </span><span class="label success">Natalia Jura </span><span class="label success">Alan Ashworth </span><span class="label success">John Irwin </span><span class="label success">Michael C. Thompson </span><span class="label success">Jason E. Gestwicki </span><span class="label success">Frank von Delft </span><span class="label success">Brian K. Shoichet </span><span class="label success">James S. Fraser </span><span class="label success">Ivan Ahel</span><br><br><div class="prettybox warning" data-component=prettybox><b>Target Nominator:</b> Ivan Ahel<br><b>Target PI:</b> Frank von Delft<br><b>Therapeutic Area(s):</b> Infectious diseases<br><b>Disease Relevance:</b> Nsp3 encodes a macrodomain that is important for viral replication<br><b>Viral family:</b> coronaviridae<br><b>Viruses:</b> [SARS-CoV-2]<br></div><div class="prettybox success" data-component=prettybox><h4>Summary of Project</h4>The conserved macrodomain encoded as non-structural protein 3 (nsp3 Mac1) is employed by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) to remove host-derived ribosylation, which is a post-translational modification involved in the production of antiviral cytokines. This TEP provides early tools to develop nsp3 macrodomain inhibitors, including purification protocols of recombinant proteins, reproducible crystallisation condition suitable for X-ray crystallography fragment screening, biophysical (activity and binding) assays and over 200 fragment hits representing a wide range of chemotypes, that are a starting point for the development of more selective and potent compounds.</div><div class="prettybox focus" data-component=prettybox><h4>Scientific Background</h4>The novel severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) pandemic, which causes the disease COVID-19, has dramatically impacted the world and set the necessity for the development of efficient prophylactic and therapeutic approaches, such as antivirals. As of April 2024, <a href=https://ourworldindata.org/covid-deaths>over 7 million people have died of COVID-19 worldwide</a>. Adenosine diphosphate (ADP)-ribosylation is a highly conserved posttranslational modification related to the immune response which supresses viral replication [<a href=http://doi.org/10.1101/gad.334425.119>1</a>, <a href=https://doi.org/10.1128/mbio.01721-16>2</a>]. For instance, the “antiviral poly(ADP-ribosyl) polymerase 14 (PARP14) promotes interleukin-4-dependent transcription, resulting in the production of antiviral cytokines [<a href=http://doi.org/10.4049/jimmunol.1701117>3</a>]. In this context, viral macrodomains recognise and remove host-derived ADP-ribosylation [<a href=https://doi.org/10.1128/JVI.00705-16>4</a>]. Coronaviral conserved macrodomain (“Mac1”), encoded as part of the nonstructural protein3 (Nsp3) has been shown to be essential for viral replication in animal studies [<a href=http://doi.org/10.1098/rsob.200237>5</a>, <a href=https://doi.org/10.1371/journal.ppat.1007756>6</a>]. Moreover, macrodomain catalytic null mutations render the murine hepatitis virus nonpathogenic [<a href=https://doi.org/10.1128/jvi.02082-08>7</a>]. In this way, SARS-CoV-2 Nsp3 Mac1 is an attractive drug target for the development of antivirals.</div><div class="prettybox error" data-component=prettybox><h4>Conclusion</h4>Here we have established a purification protocol for recombinant nsp3 Mac1 from SARS-CoV-2 and established a robust crystallisation suitable for fragment screening. We have performed a substantial fragment screening campaign and identified many hits which were validated by biophysical assays established in this work. The ligands and assays established in this work will pave the way for future drug discovery endeavours targeting Nsp3 Mac1.
Fragment screen PDB group deposition: <a href=https://www.rcsb.org/groups/summary/entry/G_1002238>https://www.rcsb.org/groups/summary/entry/G_1002238</a></div><h4>Resources</h4><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href=https://www.addgene.org/169210/><img title=plasmid width=40 src=../img/icons/plasmid.png></a></div><div id=cosv2-NSP3-macro_plasmid-crystallography-P43 class=col-11><b>Full-length Nsp3 Mac1 construct bearing a N-terminal His-tag followed by a TEV-cleavage site that crystallised in the space group P4<sub>3</sub></b><br>This protein yielded highly reproducible crystals upon microseeding, which typically diffracted to 0.85 Å resolution. This crystal system was DMSO tolerant and therefore suitable for our subsequent fragment soaking.<br><a href=https://www.addgene.org/169210/>https://www.addgene.org/169210/</a>
<cite>[ 2021-06-01 ]</cite><br><button data-component=toggleme data-target=#togglebox-protocol-ASAP-SARS-COV-2-NSP3-MAC1-1 data-text="Hide Protocol">Protocol</button><div id=togglebox-protocol-ASAP-SARS-COV-2-NSP3-MAC1-1 class=hide><cite>For expression of protein used for crystallisation, the construct was transformed into the E. coli Rosetta strain BL21(DE3)-R3, and cells were grown at 37°C in LB medium (Miller) supplemented with kanamycin (50 µg/ml) and chloramphenicol (35 µg/ml). After reaching an OD~600~ of 0.5 to 0.6, the temperature was lowered to 18°C before induction of protein expression overnight by adding 0.5 mM IPTG. Harvested cells were resuspended in lysis buffer [50 mM Hepes (pH 7.5), 500 mM NaCl, 5% glycerol, 20 mM imidazole, 10 mM ßME, and complete EDTA-free protease inhibitors (Roche)] and stored at -20°C until purification. For protein purification, pellets were gently thawed in lukewarm water and lysed by high-pressure homogenisation. DNA was digested using Benzonase. Proteins were purified by immobilized metal affinity chromatography (IMAC) using Ni-Sepharose resin (GE Healthcare) and eluted stepwise in binding buffer containing 40 to 500 mM imidazole. A high-salt wash with 1 M NaCl was combined with the first elution step including 40 mM imidazole. Removal of the His6 tag was carried out by addition of recombinant TEV protease during overnight dialysis into buffer without imidazole, followed by purification on a second IMAC column, and lastly by SEC (Superdex 75, GE Healthcare) in a buffer consisting of 20 mM Hepes (pH 8.0), 250 mM NaCl, and 2 mM DTT. Macrodomain protein used for HTRF assay was not subjected to TEV cleavage and purified after the IMAC step by SEC in a buffer consisting of 25 mM Hepes (pH 7.4), 300 mM NaCl, 5% glycerol, and 0.5 mM tris(2- carboxyethyl)phosphine. Proteins were characterised by SDS-polyacrylamide gel electrophoresis, then flash- frozen in liquid nitrogen, and stored at -80°C until required.</cite></div><button data-component=toggleme data-target=#togglebox-details-ASAP-SARS-COV-2-NSP3-MAC1-1 data-text="Hide Details">Details</button><div id=togglebox-details-ASAP-SARS-COV-2-NSP3-MAC1-1 class=hide><ul><li>Vector: pNIC28-Bsa4</li><li>Entry clone accession: MT215193.1</li><li>Cell line: E. coli Rosetta strain BL21(DE3)-R3</li><li>Tags and additions: N-terminal, TEV protease cleavable hexahistidine</li><li>Construct protein sequence: <code>MHHHHHHSSGVDLGTENLYFQ*SMVNSFSGYLKLTDNVYIKNADIVEEAKKVKPTVVVNAANVYLKHGGGVAGALNKATN NAMQVESDDYIATNGPLKVGGSCVLSGHNLAKHCLHVVGPNVNKGEDIQLLKSAYENFNQHEVLLAPLLSAGIFGADPIHSL RVCVDTVRTNVYLAVFDKNLYDKLVSSFL</code></li></ul></div></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href=https://www.addgene.org/169209/><img title=plasmid width=40 src=../img/icons/plasmid.png></a></div><div id=cosv2-NSP3-macro_plasmid-crystallography-C2 class=col-11><b>Full-length Nsp3 Mac1 construct bearing a N-terminal His-tag followed by a TEV-cleavage site that crystallised in the space group C2</b><br>This construct consistently crystallised in C2 and also diffracted to a high- resolution of 0.77 Å, although was less tolerant to DMSO than the construct above-cited.<br><a href=https://www.addgene.org/169209/>https://www.addgene.org/169209/</a>
<cite>[ 2021-06-01 ]</cite><br><button data-component=toggleme data-target=#togglebox-protocol-ASAP-SARS-COV-2-NSP3-MAC1-2 data-text="Hide Protocol">Protocol</button><div id=togglebox-protocol-ASAP-SARS-COV-2-NSP3-MAC1-2 class=hide><cite>SARS-CoV-2 Nsp3 Mac1 (residues 2 to 170) was cloned into a pET22b(+) expression vector with an N-terminal His6 tag and a tobacco etch virus (TEV) protease recognition site for removal of the tag (GenScript). In addition, a short linker (Asn-Ala-Gly) was included between the TEV recognition site and the Mac1 gene. To express Mac1, plasmid DNA was transformed into BL21(DE3) Escherichia coli. After overnight growth on lysogeny broth (LB) agar supplemented with carbenicillin (100 g/ml), starter cultures (10 ml of LB) were grown at 37°C for 8 hours. Large-scale cultures (1 litre of terrific broth) were grown at 37°C until an optical density of 0.8. Cultures were cooled at 4°C for 15 min, before protein expression was induced with 1 mM isopropyl ß-d-1-thiogalactopyranoside (IPTG), and the cultures were shaken at 20°C for 12 hours. Cells were collected by centrifugation and frozen at -80°C. All purification steps were performed at 4°C using an AKTA fast protein liquid chromatography system (Cytiva). Cells were resuspended in Ni-nitrilotriacetic acid (NTA) binding buffer [50 mM tris-HCl (pH 8.0), 500 mM NaCl, 10 mM imidazole, 5% glycerol, and 2 mM ß-mercaptoethanol (ßME) supplemented with Turbo-Nuclease (5 U/ml; Sigma-Aldrich, T4330)] and lysed by sonication. Cell debris was collected by centrifugation, and the lysate was applied to a 5-ml HisTrap HP column (Cytiva, 17524802). The column was washed with 25 ml of binding buffer followed by 25 ml of 5% Ni-NTA elution buffer [50 mM tris-HCl (pH 8.0), 500 mM NaCl, 500 mM imidazole, 5% glycerol, and 2 mM ßME] and then eluted with 100% elution buffer. Eluted protein was exchanged into TEV reaction buffer [50 mM tris (pH 8.0), 100 mM NaCl, 1 mM dithiothreitol (DTT), and 1% glycerol] using a HiPrep 26/10 desalting column (Cytiva, 17508701). To cleave the His6 tag, Mac1 was diluted to 1.5 mg/ml using TEV reaction buffer and incubated with recombinant TEV protease at a 1:20 ratio (Mac1:TEV) for 16 hours at 4°C. Cleaved Mac1 was separated from the uncleaved protein and TEV protease by rerunning the sample over a HisTrap HP column (pre-equilibrated with TEV reaction buffer) and collecting the flow-through. The flow-through was supplemented with 10 mM DTT and concentrated to 2.5 ml using a 10-kDa molecular weight cutoff (MWCO) centrifugal concentrator (Amicon, UFC901024). The sample was further purified by size exclusion chromatography (SEC) using a HiLoad 16/600 Superdex 75 pg column (Cytiva, 28989333) equilibrated with SEC buffer [20 mM tris (pH 8.0), 150 mM NaCl, 5% glycerol, and 2 mM DTT]. Eluted fractions were concentrated to 15 mg/ml, flash-frozen in liquid nitrogen, and stored at -80°C. Protein used for ITC was purified in the same manner, but the SEC was run with 150 mM NaCl and 20 mM tris (pH 8.0). Protein was concentrated to 10.8 mg/ml before flash-freezing in liquid nitrogen and storage at -80°C.</cite></div><button data-component=toggleme data-target=#togglebox-details-ASAP-SARS-COV-2-NSP3-MAC1-2 data-text="Hide Details">Details</button><div id=togglebox-details-ASAP-SARS-COV-2-NSP3-MAC1-2 class=hide><ul><li>Vector: pET22b(+)</li><li>Entry clone accession: MT215193.1</li><li>Cell line: E. coli BL21(DE3)</li><li>Tags and additions: N-terminal, TEV protease cleavable hexahistidine</li><li>Construct protein sequence: <code>MHHHHHHENLYFQ*SNAGEVNSFSGYLKLTDNVYIKNADIVEEAKKVKPTVVVNAANVYLKHGGGVAGALNKATN NAMQVESDDYIATNGPLKVGGSCVLSGHNLAKHCLHVVGPNVNKGEDIQLLKSAYENFNQHEVLLAPLLSAGIFGADPIHSLRVCVD TVRTNVYLAVFDKNLYDKLVSSFLE</code></li></ul></div></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href><img title=plasmid width=40 src=../img/icons/plasmid.png></a></div><div id=cosv2-NSP3-macro_plasmid-crystallography-P121 class=col-11><b>Full-length Nsp3 Mac1 construct bearing a N-terminal His-tag followed by a TEV-cleavage site that crystallised in the space group P121</b><br>This construct encodes a SUMO1 Mac fusion which when purified crystallised in P121.<br><cite>[ 2021-06-01 ]</cite><br><button data-component=toggleme data-target=#togglebox-protocol-ASAP-SARS-COV-2-NSP3-MAC1-3 data-text="Hide Protocol">Protocol</button><div id=togglebox-protocol-ASAP-SARS-COV-2-NSP3-MAC1-3 class=hide><cite>A synthetic gene (IDT) encoding a his tag sumo1 SARS-CoV-2 Nsp3 Mac1 fusion construct was cloned into the pNIC-NOTAG vector (pET28 derivative) via golden gate. The construct was transformed into the E. coli strain BL21(DE3)-RR, and cells were grown at 37°C in AIM-TB medium (FORMEDIUM) supplemented with kanamycin (50 µg/ml). After reaching an optical density at 600 nm of around 1, the temperature was lowered to 18°C before induction of protein expression overnight. Harvested cells were resuspended in lysis buffer [10 mM Hepes (pH 7.5), 500 mM NaCl, 5% glycerol, 30 mM imidazole, 0.5 mM TCEP, 1% TX-100, 0.5 mg/mL lysozyme, 0.05 mg/mL benzonase]. Proteins were first purified by immobilized metal affinity chromatography (IMAC) using Ni-Sepharose 6 FF resin (Cytiva) the column was washed with binding buffer [10 mM Hepes (pH 7.5), 500 mM NaCl, 5% glycerol, 30 mM imidazole, 0.5 mM TCEP] and target protein eluted using same buffer containing 500 mM imidazole. The SUMO tag was removed by digestion with ULP1 protease and the mixture dialysed overnight against [10 mM Hepes (pH 7.5), 500 mM NaCl, 5% glycerol, 30 mM imidazole, 0.5 mM TCEP]. Undigested protein, SUMO tagged protein and his tagged ULP1 was removed by using Ni-Sepharose 6 FF resin (Cytiva) and the unbound fraction containing untagged protein collected. Protein was lastly purified by SEC (Superose 12 pg, Cytiva) in a buffer consisting of 20 mM Tris-HCl (pH 8), 150 mM NaCl, 5% glycerol and 1 mM TCEP. Proteins were characterised by SDS-polyacrylamide gel electrophoresis and mass spec and then flash- frozen in liquid nitrogen, and stored at -80°C until required.</cite></div><button data-component=toggleme data-target=#togglebox-details-ASAP-SARS-COV-2-NSP3-MAC1-3 data-text="Hide Details">Details</button><div id=togglebox-details-ASAP-SARS-COV-2-NSP3-MAC1-3 class=hide><ul><li>Vector: pNIC</li><li>Entry clone accession: MT215193.1</li><li>Cell line: E. coli BL21(DE3)</li><li>Tags and additions: SUMO</li><li>Construct protein sequence: MHHHHHHGSGDQEAKPSTEDLGDKKEGEYIKLKVIGQDSSEIHFKVKMTTHLKKLKESYCQRQGVPMNSLRFLFEGQRIADNHTPKELGMEEEDVIEVYQEQTGG//GEVNSFSGYLKLTDNVYIKNADIVEEAKKVKPTVVVNAANVYLKHGGGVAGALNKATNNAMQVESDDYIATNGPLKVGGSCVLSGHNLAKHCLHVVGPNVNKGEDIQLLKSAYENFNQHEVLLAPLLSAGIFGADPIHSLRVCVDTVRTNVYLAVFDKNLYDKLVSSFLEMKSEK</li></ul></div></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href=https://fragalysis.diamond.ac.uk/viewer/react/preview/target/Mac1><img title=xray-fragment-screen width=40 src=../img/icons/xray-fragment-screen.png></a></div><div id=cosv2-NSP3-macro_fragment-screen-fragalysis class=col-11><b>Fragalysis interactive exploration of X-ray fragment screen of SARS-CoV-2 nsp3 Mac1 macrodomain</b><br>We performed X-ray crystallography screening at the Diamond Light Source XChem facility and UCSF with a total of 2533 fragments, which yielded 214 unique macrodomain-binders, including 192 fragments identified in the active site.<br><a href=https://fragalysis.diamond.ac.uk/viewer/react/preview/target/Mac1>https://fragalysis.diamond.ac.uk/viewer/react/preview/target/Mac1</a>
<cite>[ 2021-06-01 ]</cite><br><button data-component=toggleme data-target=#togglebox-protocol-ASAP-SARS-COV-2-NSP3-MAC1-4 data-text="Hide Protocol">Protocol</button><div id=togglebox-protocol-ASAP-SARS-COV-2-NSP3-MAC1-4 class=hide><cite>SARS-CoV-2 nsp3 Mac1 was concentrated to a final concentration of 47 mg/ml and apo crystals were grown in crystallisation solution containing 100 mM CHES (pH 9.5) and 30% PEG-3000. Fragments were soaked into crystals as previously described by adding dissolved compounds directly to the crystallisation drops using an Echo liquid handler (final concentration, 10% DMSO); drops were incubated for approximately 1 to 3 hours before mounting and flash-freezing in liquid nitrogen. Data were collected at the I04-1 beamline at 100 K and automatically processed with Diamond Light Source&rsquo;s autoprocessing pipelines using XDS (9) and either <a href=https://doi.org/10.1107/s0907444913015308>xia2</a> or <a href=https://doi.org/10.1107/s2059798317017235>DIALS</a>. Further analysis was performed with <a href=https://doi.org/10.1107%2FS2059798316020234>XChemExplorer</a>, electron density maps were generated with DIMPLE and ligand-binding events were identified using <a href=https://doi.org/10.1038/ncomms15123>PanDDA</a>. Ligands were modeled into PanDDA-calculated event maps using Coot, restraints were calculated with <a href=http://dx.doi.org/10.1107/S2059798317000067>AceDRG</a>, and structures were refined with <a href=https://doi.org/10.1107/s0907444992010400>BUSTER</a>. Coordinates, structure factors, and PanDDA event maps for the structures discussed are deposited in the Protein Data Bank.</cite></div></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href><img title=protocol width=40 src=../img/icons/protocol.png></a></div><div id=cosv2-NSP3-macro_crystallization class=col-11><b>Crystallization of full-length Nsp3 Mac1 in P1211 spacegroup</b><br>We obtained crystal which diffracted to 1.2 Å.<br><cite>[ 2023-08-02 ]</cite><br><button data-component=toggleme data-target=#togglebox-protocol-ASAP-SARS-COV-2-NSP3-MAC1-5 data-text="Hide Protocol">Protocol</button><div id=togglebox-protocol-ASAP-SARS-COV-2-NSP3-MAC1-5 class=hide><cite>SARS-CoV-2 nsp3 Mac1 was concentrated to a final concentration of 22 mg/ml. Apo crystals were grown in 0.1 M MES, pH 6.5, and 30 % w/v PEG 4000 in 800 nl drops containing 400 nl of protein, 300 nl of crystallisation solution and 100 nl of seed stock. Compounds were soaked into crystals as previously described by adding dissolved compounds directly to the crystallisation drop using an Echo liquid handler (final concentration 10 % DMSO) and incubated for 1 h before mounting and flash-freezing in liquid nitrogen</cite></div></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href><img title=protocol width=40 src=../img/icons/protocol.png></a></div><div id=cosv2-NSP3-macro_cocrystallization class=col-11><b>Cocrystallization of full-length Nsp3 Mac1 with compounds in P1211 spacegroup</b><br>We obtained crystal which diffracted to 1.5 Å.
PDB to be released:<br><cite>[ 2023-08-02 ]</cite><br><button data-component=toggleme data-target=#togglebox-protocol-ASAP-SARS-COV-2-NSP3-MAC1-6 data-text="Hide Protocol">Protocol</button><div id=togglebox-protocol-ASAP-SARS-COV-2-NSP3-MAC1-6 class=hide><cite>40 nl of compound (5 % of drop volume) was dispensed into wells using an Echo liquid handler before adding 400 nl of protein (22 mg/ml), 300 nl of crystallisation solution and 100 nl of seed stock. Crystals formed overnight and were mounted and flash-frozen in liquid nitrogen the next day.</cite></div></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href=https://www.biorxiv.org/content/10.1101/2020.11.24.393405v1><img title=preprint width=40 src=../img/icons/preprint.png></a></div><div id=cosv2-NSP3-macro_fragment-screen-preprint class=col-11><b>Fragment Binding to the Nsp3 Macrodomain of SARS-CoV-2 Identified Through Crystallographic Screening and Computational Docking</b><br><a href=https://www.biorxiv.org/content/10.1101/2020.11.24.393405v1>https://www.biorxiv.org/content/10.1101/2020.11.24.393405v1</a>
<cite>[ 2020-11-24 ]</cite><br></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href=https://www.biorxiv.org/content/10.1101/2020.11.24.393405v1><img title=biophysical-assay width=40 src=../img/icons/biophysical-assay.png></a></div><div id=cosv2-NSP3-macro_biophysical-assay-DSF class=col-11><b>High-throughput differential scanning fluorimetry (DSF) assay</b><br>A 384-well differential scanning fluorimetry (DSF) assay suitable for high-throughput quantitative screening was established.<br><a href=https://www.biorxiv.org/content/10.1101/2020.11.24.393405v1>https://www.biorxiv.org/content/10.1101/2020.11.24.393405v1</a>
<cite>[ 2020-11-24 ]</cite><br><button data-component=toggleme data-target=#togglebox-protocol-ASAP-SARS-COV-2-NSP3-MAC1-8 data-text="Hide Protocol">Protocol</button><div id=togglebox-protocol-ASAP-SARS-COV-2-NSP3-MAC1-8 class=hide><cite>Compounds were dissolved in DMSO to a final concentration of 100 mM and placed in a 384-well Echo source plate (Labcyte, PP0200). Using a Labcyte Echo, each compound was dispensed into a 384-well storage plate (Greiner Bio-One, 781280) in five stock concentrations in twofold serial dilutions (compounds, 6.25 to 100 mM; ADPr, 0.625 to 10 mM) and a final volume of 750 nl in triplicate. A protein-free plate was created as a control. Echo dispensing instructions were created by an in-house app (<a href=https://gestwickilab.shinyapps.io/echo_layout_maker/)>https://gestwickilab.shinyapps.io/echo_layout_maker/)</a>. DSF buffer was prepared by adding 10 µl of SYPRO Orange (Thermo Fisher Scientific, S6650) to 10 ml of buffer [50 mM tris- HCl (pH 7.5), 150 mM NaCl, 1 mM EDTA, 1 mM DTT, and 0.01% Triton X-100] for a final dye concentration of 5x (10 µM) SYPRO Orange. A compound plate was resuspended by the addition of 20 µl of DSF buffer and set aside for 20 min in the dark. Purified Mac1 (P43 construct purified at UCSF) was diluted to 10 µM in DSF buffer, and 2 µl of either protein solution or protein-free buffer was added to each well of a 384-well white PCR plate (Axygen, PCR-384- LC480WNFBC) using an E1 ClipTip P125 electronic pipette. Eight microlitres of resuspended compound was transferred to each well of the protein- and buffer-containing PCR plate using an Opentrons OT-2 liquid handling system, yielding the following final conditions: 2 µM Mac1, 5x (10 µM) SYPRO Orange, 3% DMSO, 0.1 to 3 mM fragments, and 0.1 to 1 mM ADPr. The PCR plate was spun briefly in a salad spinner to remove bubbles and sealed with an optically clear film (Applied Biosystems, MicroAmp Optical Adhesive Film, 4311971). In an Analytik Jena qTOWER 384G quantitative PCR instrument, plate was continuously heated from 25° to 94°C at a rate of 1°C/min, and fluorescence was measured at each degree in the TAMRA channel (535/580 nm). Fifty- three of 54 fragments were able to be tested up to 3 mM without assay interference under these conditions. Tm values were calculated online at DSFworld (<a href=https://bio.tools/dsfworld)>https://bio.tools/dsfworld)</a>.</cite></div></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href=https://www.biorxiv.org/content/10.1101/2020.11.24.393405v1><img title=biophysical-assay width=40 src=../img/icons/biophysical-assay.png></a></div><div id=cosv2-NSP3-macro_biophysical-assay-HTRF class=col-11><b>Homogeneous time-resolved fluorescence (HTRF)-based peptide displacement assay</b><br>A 384-well Homogeneous time-resolved fluorescence (HTRF)-based peptide displacement assay suitable for high-throughput quantitative screening was established.<br><a href=https://www.biorxiv.org/content/10.1101/2020.11.24.393405v1>https://www.biorxiv.org/content/10.1101/2020.11.24.393405v1</a>
<cite>[ 2020-11-24 ]</cite><br><button data-component=toggleme data-target=#togglebox-protocol-ASAP-SARS-COV-2-NSP3-MAC1-9 data-text="Hide Protocol">Protocol</button><div id=togglebox-protocol-ASAP-SARS-COV-2-NSP3-MAC1-9 class=hide><cite>Fragment inhibitory activity on Mac1 was verified by the displacement of an ADPr-conjugated biotin peptide from the His6-tagged Mac1 using HTRF with an Eu3+-conjugated anti-His6 antibody donor and streptavidin- conjugated acceptor. Compounds were dispensed into white ProxiPlate-384 Plus (PerkinElmer) assay plates using an Echo 525 liquid handler (Labcyte). Binding assays were conducted in a final volume of 16 ul with 12.5 nM Mac1, 400 nM peptide ARTK(Bio)QTARK(Aoa-RADP)S [synthesised by Cambridge Peptides (Birmingham, UK)], 1:125 streptavidin-XL665 (Cisbio), and 1:20,000 anti-His6-Eu3+ cryptate (PerkinElmer) in assay buffer [25 mM Hepes (pH 7.0), 20 mM NaCl, 0.05% bovine serum albumin, and 0.05% Tween 20]. Assay reagents were dispensed into plates using a Multidrop combi (Thermo Fisher Scientific) and incubated at room temperature for 1 hour. Fluorescence was measured using a PHERAstar microplate reader (BMG) using the HTRF module with dual-emission protocol (A = excitation of 320 nm, emission of 665 nm and B = excitation of 320 nm, emission of 620 nm). Raw data were processed to give an HTRF ratio (channel A/B x 10,000), which was used to generate IC50 curves by nonlinear regression using GraphPad Prism v8 (GraphPad Software, CA, USA).</cite></div></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href=https://www.rcsb.org/groups/summary/entry/G_1002283><img title=pdb-group-deposition width=40 src=../img/icons/pdb-group-deposition.png></a></div><div id=cosv2-NSP3-inhibitors-pdb-group-deposition class=col-11><b>PDB group deposition of crystal structures of SARS-CoV-2 NSP3 Macrodomain in complex with ASAP inhibitors</b><br>SARS-CoV-2 nsp3 Mac1 macrodomain in complex with 90 ASAP inhibitors<br><a href=https://www.rcsb.org/groups/summary/entry/G_1002283>https://www.rcsb.org/groups/summary/entry/G_1002283</a>
<cite>[ 2024-01-23 ]</cite><br></div></div></div></div><hr style=height:5px;border:none;color:#333;background-color:#333><br><br><br><h3 style=text-align:center id=ASAP-SARS-COV-2-NPROTEIN>SARS-CoV-2 Nucleocapsid (N protein)</h3><div class="row align-center"><a href=https://www.ncbi.nlm.nih.gov/protein/TBD><span class="label success">Gene: TBD </span></a>&nbsp;
<a href=https://www.uniprot.org/uniprotkb/TBD><span class="label error">Uniprot: TBD </span></a>&nbsp;
<a href="https://www.ebi.ac.uk/intenz/query?cmd=SearchEC&amp;ec=TBD"><span class="label warning">EC: TBD</span></a></div><br><div class="row align-center"><table class=unstyled><tr><td><img title="Crystallography construct" src=/images/icons/tep-stage-completed_hub021b18cb248db28b117f458e39aeb5b_10162_e22862b00861f0d3434857db1ff02de5.png width=300 height=88><br><img title="Assay construct" src=/images/icons/tep-stage-completed_hub021b18cb248db28b117f458e39aeb5b_10162_2851d035e13cbc115749c33624801624.png width=300 height=88></td><td><img title="Biophysical assay" src=/images/icons/tep-stage-in-progress_huc9d943f35f5911a48f54b3ad3b40c43a_10037_80222a2cb4b3b4d0cafa06d6608261b8.png width=300 height=55><br><img title="Robust crystals" src=/images/icons/tep-stage-completed_hub021b18cb248db28b117f458e39aeb5b_10162_e3383ab8d75acf7a535bdc511b331b23.png width=300 height=55><br><img title="Biochemical assay" src=/images/icons/tep-stage-completed_hub021b18cb248db28b117f458e39aeb5b_10162_1b7ab3b95a075266c44f73afb56d1cd3.png width=300 height=55></td><td><img title="X-ray fragment screen and structural analysis" src=/images/icons/tep-stage-completed_hub021b18cb248db28b117f458e39aeb5b_10162_ae1969e18479b9f6892265d5fd14a303.png width=300 height=180></td><td><img title="First-round progression for TEP validation" src=/images/icons/tep-stage-in-progress_huc9d943f35f5911a48f54b3ad3b40c43a_10037_0bf41094ba248abeddae737c4c98b352.png width=300 height=180></td></tr></table></div><b>Contributors:</b>
<span class="label success">Jakub Luptak </span><span class="label success">Leo James </span><span class="label success">Alice Douangamath </span><span class="label success">Daren Fearon </span><span class="label success">Jasmin Aschenbrenner </span><span class="label success">Peter Marples</span><br><br><div class="prettybox warning" data-component=prettybox><b>Target Nominator:</b> Jakub Luptak<br><b>Target PI:</b> Frank von Delft<br><b>Therapeutic Area(s):</b> Infectious diseases<br><b>Disease Relevance:</b> Infectious diseases<br><b>Viral family:</b> coronaviridae<br><b>Viruses:</b> [SARS-CoV-2]<br></div><div class="prettybox success" data-component=prettybox><h4>Summary of Project</h4>This TEP currently contains the information of the work done by Jakub Luptak on characterizing SARS-CoV2 N-protein.</div><div class="prettybox focus" data-component=prettybox><h4>Scientific Background</h4>Nucleocapsid protein undergoes several assembly and disassembly events during the viral infectious cycle. Within virions, it protects the RNA genome. Its oligomeric structure must be dismantled to release viral RNA during cell entry, then newly synthesized protein must assemble to make new virions.</div><div class="prettybox error" data-component=prettybox><h4>Conclusion</h4>TEP still in progress.</div><h4>Resources</h4><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href><img title=plasmid width=40 src=../img/icons/plasmid.png></a></div><div id=cosv2-Nprot-Nterm_plasmid class=col-11><b>Protein expression construct for SARS-CoV-2 N-protein P1127 C-terminal domain (248-365)</b><br>High yield expression construct produces 50-100 mg in 1-2. Works with 2XTY and minimal media as well.<br><cite>[ 2022-10-01 ]</cite><br><button data-component=toggleme data-target=#togglebox-protocol-ASAP-SARS-COV-2-NPROTEIN-1 data-text="Hide Protocol">Protocol</button><div id=togglebox-protocol-ASAP-SARS-COV-2-NPROTEIN-1 class=hide><cite>Expression conditions: Plasmid was transformed into C41 competent E. coli and single colony used to inoculate an overnight starter culture (100 mL in LB). Next morning, N x 0.5 L of ZYP-5052 autoinduction media (with 1X Ampicillin) were inoculated 1:100 with the starter culture. Cultures were then incubated for 3 hours at 37 °C after which the cells were cooled down to 25C and left to grow overnight. In the morning cells were pelleted by centrifugation (25 min; 5000rpm any compatible centrifuge). Pellet was snap frozen in liquid nitrogen until processed. Lysis: Pellet was resuspended in Lysis buffer to achieve a final volume of ~150 mL. To achieve complete lysis, resuspended cells were sonicated on ice at 70% amp, 5s on 10s off for 7 minutes. Soluble protein had been recovered by centrifugation (18 000rpm in JA25.50 rotor for 45 minutes). Lysis buffer: 1 M NaCl, 10 mM Imidazole, 50 mM Tris pH 8, 2 mM DTT, cOmplete Roche Protease inhibitor tablets, 15% v/v 1XBugBuster+Benzonase IMAC: Soluble protein was applied onto a gravity flow column, which has been prepared with 10 mL of Ni-NTA Agarose (5 mL of beads) washed in Buffer B. The lysate was allowed to drip slowly through the resin and the flow through collected for analysis. Next, the column was washed in 100 mL of Buffer B and collected as Wash. Finally, the protein was eluted with Buffer E in 3 mL fractions (total ~24 ml)</cite></div><button data-component=toggleme data-target=#togglebox-details-ASAP-SARS-COV-2-NPROTEIN-1 data-text="Hide Details">Details</button><div id=togglebox-details-ASAP-SARS-COV-2-NPROTEIN-1 class=hide><p>ProtParam Output</p><ul><li>Number of amino acids: 124</li><li>Molecular weight: 14038.80</li><li>Theoretical pI: 9.60</li><li>Ext. coefficient 16960</li><li>Abs 0.1% (=1 g/l) 1.208
Sequence: MAHHHHHHMSAAEASKKPRQKRTATKAYNVTQAFGRRGPEQTQG
NFGDQELIRQGTDYKHWPQIAQFAPSASAFFGMSRIGMEVTPSGTWLTYTGAIK
LDDKDPNFKDQVILLNKHIDAYKTFP</li></ul><p>p1127-pOPTH-Covid-19-NP-Cterm Map_N-prot sequence:
AGCGCAGCAGAAGCAAGCAAAAAACCGCGTCAGAAACGTACCGCAACCAAAGCA
TATAATGTTACCCAGGCATTTGGTCGTCGTGGTCCGGAACAGACCCAGGGTAAT
TTTGGTGATCAAGAACTGATTCGTCAGGGCACCGATTATAAACATTGGCCTCAG
ATTGCACAGTTTGCACCGAGCGCAAGTGCATTTTTTGGCATGAGCCGTATTGGT
ATGGAAGTTACCCCGAGCGGCACCTGGCTGACCTATACAGGTGCAATTAAACTGG
ATGATAAAGATCCGAACTTCAAGGATCAGGTGATTCTGCTGAACAAACATATCGA
TGCCTATAAAACATTTCCG</p><p>Buffer B: 300 mM NaCl, 10 mM Imidazole, 50 mM Tris pH 8, 1 mM DTT
Buffer E: 300 mM NaCl, 400 mM Imidazole, 50 mM Tris pH 8, 1 mM DTT<br><img src=/img/tep_images/sars2_nprot_example_gel_1.png alt=image>
<img src=/img/tep_images/sars2_nprot_example_gel_2.png alt=image></p></div></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href=https://fragalysis.diamond.ac.uk/viewer/react/preview/target/Nprot><img title=xray-fragment-screen width=40 src=../img/icons/xray-fragment-screen.png></a></div><div id=cosv2-Nprot-Nterm_fragment-screen-fragalysis class=col-11><b>Fragalysis interactive exploration of X-ray fragment screen of SARS-CoV-2 N-protein</b><br><a href=https://fragalysis.diamond.ac.uk/viewer/react/preview/target/Nprot>https://fragalysis.diamond.ac.uk/viewer/react/preview/target/Nprot</a>
<cite>[ 2021-11-15 ]</cite><br><button data-component=toggleme data-target=#togglebox-protocol-ASAP-SARS-COV-2-NPROTEIN-2 data-text="Hide Protocol">Protocol</button><div id=togglebox-protocol-ASAP-SARS-COV-2-NPROTEIN-2 class=hide><cite><a href=dx.doi.org/10.17504/protocols.io.4r3l2q4b3l1y/v1>Available at protocols.io</a></cite></div></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href=https://fragalysis.diamond.ac.uk/viewer/react/projects/2706/2244><img title=biophysical-assay width=40 src=../img/icons/biophysical-assay.png></a></div><div id=cosv2-Nprot-Nterm_biophysical-assay class=col-11><b>Tryptophan quenching biophysical assay for SARS-CoV-2 N-protein binding</b><br>Final concentration of protein is 14 uM (arbitrary, once compounds get more potent this will become limiting). Note that because of the way the dilutions are done, final compound concentrations are 0.9x of the designed titration.<br><a href=https://fragalysis.diamond.ac.uk/viewer/react/projects/2706/2244>https://fragalysis.diamond.ac.uk/viewer/react/projects/2706/2244</a>
<cite>[ 2021-11-15 ]</cite><br><button data-component=toggleme data-target=#togglebox-protocol-ASAP-SARS-COV-2-NPROTEIN-3 data-text="Hide Protocol">Protocol</button><div id=togglebox-protocol-ASAP-SARS-COV-2-NPROTEIN-3 class=hide><cite><p>Steps:</p><ul><li>Prepare working stock (10X) of p1127; 140 uM in Assay buffer without DMSO * Prepare compound dilutions in a 96 well plate, need 45 uL per well. If running a duplicate with blank dilution for compound fluorescence prepare 200 uL of each concentration. Normally I was doing a 1:1.5 dilution series to get a good coverage at the higher concentrations for weak compounds. * Pipette 5 uL of 10X protein into 384 well plate for compound titrations * (if needed) Pipette 5 uL of assay buffer into 384 well plate for blank titrations * Pipette 45 uL of compound dilution on top of the 5 uL of 10X protein, including just assay buffer+DMSO * Equilibrate for 10 min * Read fluorescence at 350nm * If needed subtract compound fluorescence and fit using 4 parameter dose-response in Prism.</li></ul></cite></div><button data-component=toggleme data-target=#togglebox-details-ASAP-SARS-COV-2-NPROTEIN-3 data-text="Hide Details">Details</button><div id=togglebox-details-ASAP-SARS-COV-2-NPROTEIN-3 class=hide><p><a href="/pdfs/PAGE22-01018 - ASAP; SARS-CoV2; N-protein; protocol transfer from Jakub Luptak.pdf" download>Download experimental metadata</a>
Assay buffer: 150 mM NaCl, 50 mM Tris pH 8, 1 mM DTT (usually prepared as aliquots of 10X, frozen and thawed for use). DMSO added to 1-5% v/v depending on compounds.</p><p>Protein: p1127 (nprot) in 300 mM NaCl, 50 mM Tris pH 8, 1 mM DTT</p><p>PHERAstar reader</p><p>384 well low-binding plate (CLS3575)</p></div></div></div></div></div><hr style=height:5px;border:none;color:#333;background-color:#333><br><br><br><h3 style=text-align:center id=ASAP-EVA71-2APRO>Enterovirus A71 2A protease</h3><div class="row align-center"><a href=https://www.ncbi.nlm.nih.gov/nuccore/JN544418><span class="label success">Genetic source </span></a>&nbsp;
<a href=https://www.ncbi.nlm.nih.gov/protein/JN544418.1><span class="label success">Gene: JN544418.1 </span></a>&nbsp;
<a href=https://www.uniprot.org/uniprotkb/Q66479%20%28closest%20match%29><span class="label error">Uniprot: Q66479 (closest match) </span></a>&nbsp;
<a href="https://www.ebi.ac.uk/intenz/query?cmd=SearchEC&amp;ec=3.4.22.29"><span class="label warning">EC: 3.4.22.29</span></a></div><br><div class="row align-center"><table class=unstyled><tr><td><img title="Crystallography construct" src=/images/icons/tep-stage-completed_hub021b18cb248db28b117f458e39aeb5b_10162_e22862b00861f0d3434857db1ff02de5.png width=300 height=88><br><img title="Assay construct" src=/images/icons/tep-stage-completed_hub021b18cb248db28b117f458e39aeb5b_10162_2851d035e13cbc115749c33624801624.png width=300 height=88></td><td><img title="Biophysical assay" src=/images/icons/tep-stage-not-started_hu0ad6bc3691333ead921c5ce1ee694314_9444_3996d997df11e427d2dc236cf2e2aca0.png width=300 height=55><br><img title="Robust crystals" src=/images/icons/tep-stage-completed_hub021b18cb248db28b117f458e39aeb5b_10162_e3383ab8d75acf7a535bdc511b331b23.png width=300 height=55><br><img title="Biochemical assay" src=/images/icons/tep-stage-completed_hub021b18cb248db28b117f458e39aeb5b_10162_1b7ab3b95a075266c44f73afb56d1cd3.png width=300 height=55></td><td><img title="X-ray fragment screen and structural analysis" src=/images/icons/tep-stage-completed_hub021b18cb248db28b117f458e39aeb5b_10162_ae1969e18479b9f6892265d5fd14a303.png width=300 height=180></td><td><img title="First-round progression for TEP validation" src=/images/icons/tep-stage-in-progress_huc9d943f35f5911a48f54b3ad3b40c43a_10037_0bf41094ba248abeddae737c4c98b352.png width=300 height=180></td></tr></table></div><b>Contributors:</b>
<span class="label success">Michael Fairhead </span><span class="label success">Ryan Lithgo </span><span class="label success">Andre de Godoy </span><span class="label success">Blake Balcomb </span><span class="label success">Charlie Tomlinson </span><span class="label success">Charline Giroud </span><span class="label success">Daren Fearon </span><span class="label success">Dong Wang </span><span class="label success">Eleanor Williams </span><span class="label success">Haim Barr </span><span class="label success">Jasmin Aschenbrenner </span><span class="label success">Melissa Marx </span><span class="label success">Jessica Dixon </span><span class="label success">Korvus Wang </span><span class="label success">Lee Harris </span><span class="label success">Lizbé Koekemoer </span><span class="label success">Lucas Ferreira </span><span class="label success">Noa Lahav </span><span class="label success">Oleg Fedorov </span><span class="label success">Peter Marples </span><span class="label success">Tamas Szommer </span><span class="label success">Warren Thompson </span><span class="label success">Xiaomin Ni</span><br><br><div class="prettybox warning" data-component=prettybox><b>Target Nominator:</b> Alpha Lee<br><b>Target PI:</b> Frank von Delft<br><b>Therapeutic Area(s):</b> Infectious diseases<br><b>Disease Relevance:</b> 3C proteases are a clinically validated target.<br><b>Viral family:</b> picronaviridae<br><b>Viruses:</b> enterovirus A71<br></div><div class="prettybox success" data-component=prettybox><h4>Summary of Project</h4>This TEP is being compiled to provide early tools to develop 2A protease inhibitors, including purification protocols of recombinant proteins, and reproducible crystallisation condition suitable for X-ray crystallography fragment screening.</div><div class="prettybox focus" data-component=prettybox><h4>Scientific Background</h4>The first cleavage in the enterovirus polyprotein is made by 2A protease&rsquo;s cleaving itself from the capsids. This enables capsid protein folding.  When inhibited, capsid protein folding is deranged. 2A also cleaves many host proteins, notably eIF4G. Not only does this inhibit the translation of capped (host) transcripts, but a fragment of eIF4G specifically activates viral translation.  There are several other host targets as well.</div><div class="prettybox error" data-component=prettybox><h4>Conclusion</h4>This is a summary of the work we have done on the Enterovirus A71 2A protease to date.</div><h4>Resources</h4><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href><img title=plasmid width=40 src=../img/icons/plasmid.png></a></div><div id=eva71-2A-protease_plasmid-crystallography class=col-11><b>Protein construct for EV-A71 2A protease suitable for crystallography (A71EV2A-c012)</b><br>His-tagged protein expression construction for EV-A71 2A protease<br><cite>[ 2022-10-24 ]</cite><br><button data-component=toggleme data-target=#togglebox-protocol-ASAP-EVA71-2APRO-1 data-text="Hide Protocol">Protocol</button><div id=togglebox-protocol-ASAP-EVA71-2APRO-1 class=hide><cite>The construct was transformed into the <em>E. coli</em> strain BL21(DE3)-RR, and cells were grown at 37°C in TB medium supplemented with kanamycin (50 µg/ml). After reaching an optical density at 600 nm of around 1, the temperature was lowered to 18°C before induction of protein expression overnight by adding 0.5 mM IPTG. Harvested cells were resuspended in lysis buffer [10 mM Hepes (pH 7.5), 500 mM NaCl, 5% glycerol, 20 mM imidazole]. Proteins were first purified by immobilized metal affinity chromatography (IMAC) using Ni-Sepharose 6 FF resin (Cytiva) the column was washed with binding buffer [10 mM Hepes (pH 7.5), 500 mM NaCl, 5% glycerol, 20 mM imidazole, 1.0 mM TCEP] and target protein eluted using same buffer containing 300 mM imidazole. The SUMO tag was removed by digestion with SENP1 protease and the mixture dialysed overnight against [10 mM Hepes (pH 7.5), 500 mM NaCl, 5% glycerol, 30 mM imidazole, 0.5 mM TCEP]. Undigested protein, SUMO tagged protein and his tagged SENP1 was removed by using Ni-Sepharose 6 FF resin (Cytiva) and the unbound fraction containing untagged protein collected. Protein was lastly purified by SEC (Superose 12 pg, Cytiva) in a buffer consisting of 25 mM Hepes (pH 7.5), 500 mM NaCl, 5% glycerol and 0.5 mM TCEP. Proteins were characterised by SDS-polyacrylamide gel electrophoresis andd mass spec and then flash- frozen in liquid nitrogen, and stored at -80°C until required.</cite></div><button data-component=toggleme data-target=#togglebox-details-ASAP-EVA71-2APRO-1 data-text="Hide Details">Details</button><div id=togglebox-details-ASAP-EVA71-2APRO-1 class=hide><p><a href="/pdfs/PAGE23-00246 - AVIDD ASAP; A71EV2A trunc08 A71EV2A-c012 Expression and Purification.pdf" download>Download experimental metadata 1</a>
<a href="/pdfs/PAGE23-00312 - AVIDD ASAP; A71EV2A trunc08 A71EV2A-c012 p003 crystallization & co-crystallization with telaprevir.pdf" download>Download experimental metadata 2</a>
<a href="/pdfs/PAGE23-00345 - AVIDD ASAP; MVMPROA, ZVNS2B & A71EV2A xtal mounting,screening.pdf" download>Download experimental metadata 3</a></p><ul><li>Vector: pNIC-NHStIIT</li><li>Entry clone accession:</li><li>Cell line:</li><li>Tags and additions: SUMO-tag</li><li>Construct protein sequence:
MHHHHHHGSGDQEAKPSTEDLGDKKEGEYIKLKVIGQDSSEIHFKVKMTTHLKKLKESYCQRQGVPMNSLRFLFEGQRIADNHTPKELGMEEEDVIEVYQEQTGG//SGAIYVGNYRVVNRHLATHNDWANLVWEDSSRDLLVSSTTAQGCDTIARCDCQTGVYYCSSRRKHYPVSFSKPSLIFVEASEYYPARYQSHLMLAVGHSEPGDCGGILRCQHGVVGIVSTGGNGLVGFADVRDLLWLDEEAMEQ</li></ul></div></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href=https://www.rcsb.org/structure/8POA><img title=pdb-deposition width=40 src=../img/icons/pdb-deposition.png></a></div><div id=eva71-2A-protease_structure-apo class=col-11><b>Structure of Coxsackievirus A16 (G-10) 2A protease</b><br>Apo structure, a small number of mutations away from EV-A71 2A protease<br><a href=https://www.rcsb.org/structure/8POA>https://www.rcsb.org/structure/8POA</a>
<cite>[ 2023-08-02 ]</cite><br><button data-component=toggleme data-target=#togglebox-protocol-ASAP-EVA71-2APRO-2 data-text="Hide Protocol">Protocol</button><div id=togglebox-protocol-ASAP-EVA71-2APRO-2 class=hide><cite>Plate format: Swiss CI 3 lens Screen: 0.2M Ammonium Formate and 20% PEG 3350 (Condition H01 of ShotGun 1 screen) Buffer: 25mM HEPEs, pH 7.5, 150mM NaCl, 0.5mM TCEP, 5% Glycerol Concentration: 12.5 mg/ml Drop ratio: 2:1 ratio of protein: reservoir solution Incubation: 20°C in Formulatrix for 3 days Imaging Schedule: first images after 12hrs and then following a Fibonacci sequence of days for further image collections</cite></div></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href=https://fragalysis.diamond.ac.uk/viewer/react/preview/target/A71EV2A/tas/lb32627-66><img title=xray-fragment-screen width=40 src=../img/icons/xray-fragment-screen.png></a></div><div id=eva71-2A-protease_fragment-screen-fragalysis class=col-11><b>Fragalysis interactive view of crystallographic fragment screening of Coxsackievirus A16 (G-10) 2A protease</b><br><p>Crystallographic fragment screening of Coxsackievirus A16 (G-10) 2A protease.</p><p>This structure is just a few mutations away from EV-A71 2A protease.</p><br><a href=https://fragalysis.diamond.ac.uk/viewer/react/preview/target/A71EV2A/tas/lb32627-66>https://fragalysis.diamond.ac.uk/viewer/react/preview/target/A71EV2A/tas/lb32627-66</a>
<cite>[ 2024-04-04 ]</cite><br></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href=https://www.rcsb.org/groups/summary/entry/G_1002288><img title=pdb-group-deposition width=40 src=../img/icons/pdb-group-deposition.png></a></div><div id=eva71-2A-protease_fragment-screen-pdb-group-deposition class=col-11><b>Group deposition for crystallographic fragment screening of Coxsackievirus A16 (G-10) 2A protease</b><br><p>Crystallographic fragment screening of Coxsackievirus A16 (G-10) 2A protease.</p><p>This structure is just a few mutations away from EV-A71 2A protease.</p><br><a href=https://www.rcsb.org/groups/summary/entry/G_1002288>https://www.rcsb.org/groups/summary/entry/G_1002288</a>
<cite>[ 2024-04-04 ]</cite><br></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href><img title=biochemical-assay width=40 src=../img/icons/biochemical-assay.png></a></div><div id=eva71-3C-protease_biochemical-assay class=col-11><b>EV-A71 2A protease activity biochemical assay using fluorescence intensity measurements</b><br>This method is intended to measure the activity of viral proteases by using a specific labelled-peptide that allows the detection of the cleaved product. The substrate contains the cleavage-sequence specific to the tested protease and is labeled in C-terminal by the fluorophore Edans (ex 336 nm; em: 455 nm) and in N-ternimal by the quencher Dabcyl (abs 472 nm). In the case of a non-cleaved substrate, the proximity of Dabcyl to Edans prevents the emission and the detection of the fluorescence at 455 nm. The cleavage of the peptide by the protease allows Edans’ fluorescence emission and detection.<br><cite>[ 2023-04-24 ]</cite><br><button data-component=toggleme data-target=#togglebox-protocol-ASAP-EVA71-2APRO-5 data-text="Hide Protocol">Protocol</button><div id=togglebox-protocol-ASAP-EVA71-2APRO-5 class=hide><cite>Assay buffer: 50 mM Tris pH 7.0, 150 mM NaCl, 10% glycerol and 0.5 mM DTT (optional).
Incubation: 1 hour at room temperature.
EV-A71 2A: protein stocks were stored at -80C and used as 2x solution (20 µM, 10 µM final assay concentration) in assay buffer.
Substrate: Dabcyl-TAITTLGKFGQE-Edans (LifeTein, USA) prepared as a stock solution at 5 mM in DMSO and used at 2x solution (40 µM, 20 µM final concentration assay concentration) in assay buffer.
Positive control: Telaprevir (Pubchem CID 3010818), 50 µM top final assay concentration.
Plates: 384 well plate, white, f-bottom (Greiner #781074).
Liquid handler: Echo® acoustic liquid handler (Beckman Coulter, USA).
Plate reader: Pherastar FS, BMG Labtech (Germany), 350-460 FI optic module, the plate is read every 30 s for 2 hours and shacked during 5 s before the first reading.
Method: Fifty µL of 2x protein solution were added to each well containing the compounds to be tested previously dispensed onto the plate. The mix has been incubated for one hour at room temperature and the enzymatic reaction was initiated by the addition of 50 µL of 2x (40 µM) substrate solution using the plate reader injector. The fluorescence intensity at 460 nm was read every 30 seconds for 2 hours in kinetic mode, which include a shaking step of the plate between each measurement.
The IC&lt;50> was calculated by plotting the initial velocity against various concentrations of tested inhibitor by using a four parameter dose−response curve in Prism (v8.0) software.</cite></div></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href=https://doi.org/10.1101/2024.04.29.591684><img title=preprint width=40 src=../img/icons/preprint.png></a></div><div id=eva71-2A-protease_fragment-screen-preprint class=col-11><b>Crystallographic Fragment Screen of Coxsackievirus A16 2A Protease identifies new opportunities for the development of broad-spectrum anti-enterovirals</b><br>Preprint describing fragment screen of closely related protein to EV-A71 2A protease<br><a href=https://doi.org/10.1101/2024.04.29.591684>https://doi.org/10.1101/2024.04.29.591684</a>
<cite>[ 2024-04-29 ]</cite><br></div></div></div></div><hr style=height:5px;border:none;color:#333;background-color:#333><br><br><br><h3 style=text-align:center id=ASAP-EVA71-3CLPRO>Enterovirus A71 3C protease</h3><div class="row align-center"><a href=https://www.ncbi.nlm.nih.gov/protein/JN544418.1><span class="label success">Gene: JN544418.1 </span></a>&nbsp;
<a href=https://www.uniprot.org/uniprotkb/Q66479><span class="label error">Uniprot: Q66479 </span></a>&nbsp;
<a href="https://www.ebi.ac.uk/intenz/query?cmd=SearchEC&amp;ec=3.4.22.28"><span class="label warning">EC: 3.4.22.28</span></a></div><br><div class="row align-center"><table class=unstyled><tr><td><img title="Crystallography construct" src=/images/icons/tep-stage-completed_hub021b18cb248db28b117f458e39aeb5b_10162_e22862b00861f0d3434857db1ff02de5.png width=300 height=88><br><img title="Assay construct" src=/images/icons/tep-stage-completed_hub021b18cb248db28b117f458e39aeb5b_10162_2851d035e13cbc115749c33624801624.png width=300 height=88></td><td><img title="Biophysical assay" src=/images/icons/tep-stage-in-progress_huc9d943f35f5911a48f54b3ad3b40c43a_10037_80222a2cb4b3b4d0cafa06d6608261b8.png width=300 height=55><br><img title="Robust crystals" src=/images/icons/tep-stage-in-progress_huc9d943f35f5911a48f54b3ad3b40c43a_10037_f71a74ae374dc1fa70d31362f03e7c29.png width=300 height=55><br><img title="Biochemical assay" src=/images/icons/tep-stage-completed_hub021b18cb248db28b117f458e39aeb5b_10162_1b7ab3b95a075266c44f73afb56d1cd3.png width=300 height=55></td><td><img title="X-ray fragment screen and structural analysis" src=/images/icons/tep-stage-not-started_hu0ad6bc3691333ead921c5ce1ee694314_9444_00692520805a7a4b5cb143ae16450ab1.png width=300 height=180></td><td><img title="First-round progression for TEP validation" src=/images/icons/tep-stage-in-progress_huc9d943f35f5911a48f54b3ad3b40c43a_10037_0bf41094ba248abeddae737c4c98b352.png width=300 height=180></td></tr></table></div><b>Contributors:</b>
<span class="label success">Michael Fairhead </span><span class="label success">Ryan Lithgo </span><span class="label success">Andre de Godoy </span><span class="label success">Blake Balcomb </span><span class="label success">Charlie Tomlinson </span><span class="label success">Charline Giroud </span><span class="label success">Daren Fearon </span><span class="label success">Dong Wang </span><span class="label success">Eleanor Williams </span><span class="label success">Haim Barr </span><span class="label success">Jasmin Aschenbrenner </span><span class="label success">Jessica Dixon </span><span class="label success">Korvus Wang </span><span class="label success">Lee Harris </span><span class="label success">Lizbé Koekemoer </span><span class="label success">Lucas Ferreira </span><span class="label success">Noa Lahav </span><span class="label success">Oleg Fedorov </span><span class="label success">Peter Marples </span><span class="label success">Tamas Szommer </span><span class="label success">Warren Thompson </span><span class="label success">Xiaomin Ni</span><br><br><div class="prettybox warning" data-component=prettybox><b>Target Nominator:</b> Alpha Lee<br><b>Target PI:</b> Frank von Delft<br><b>Therapeutic Area(s):</b> Infectious diseases<br><b>Disease Relevance:</b> 3C proteases are a clinically validated target.<br><b>Viral family:</b> picronaviridae<br><b>Viruses:</b> enterovirus A71<br></div><div class="prettybox success" data-component=prettybox><h4>Summary of Project</h4>This TEP provides early tools for structure-based discovery of enterovirus A71 (EV-A71) 3C protease inhibitors, including protocols of recombinant protein production, biochemical assays and reproducible crystallisation condition suitable for X-ray crystallography fragment screening.</div><div class="prettybox focus" data-component=prettybox><h4>Scientific Background</h4>While enterovirus A71 (EV-A71) typically causes respiratory illness, which can be mild or severe, it can also cause <a href=https://www.cdc.gov/acute-flaccid-myelitis/index.html>acute flaccid myelitis (AFM)</a>, a serious neurological condition which mostly affects children that causes the muscles and reflexes in the body to become weak (via <a href=https://www.cdc.gov/non-polio-enterovirus/about/ev-d68.html>CDC</a>). Enterovirus 3C protease is required to cleave the nonstructural region of the viral polyprotein and various host proteins to facilitate viral infection; thus, inhibition can reinstate the antiviral functions of some of these targets. The protease has a typical chymotrypsin fold with a Cys-His-Glu catalytic triad.</div><h4>Resources</h4><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href><img title=plasmid width=40 src=../img/icons/plasmid.png></a></div><div id=eva71-3C-protease_plasmid-biochemical-assay class=col-11><b>Protein construct for EV-A71 3C protease suitable for biochemical assays of protease activity and crystallography</b><br>Construct for enzyme assays crystallography.<br><b>SCARAB ID:</b>
<img width=10 src=images/icons/scarab.png>
A71EV3CPROA-c001<br><cite>[ 2022-10-24 ]</cite><br><button data-component=toggleme data-target=#togglebox-protocol-ASAP-EVA71-3CLPRO-1 data-text="Hide Protocol">Protocol</button><div id=togglebox-protocol-ASAP-EVA71-3CLPRO-1 class=hide><cite>Available at protocols.io <a href=dx.doi.org/10.17504/protocols.io.dm6gpzrkdlzp/v1>1L small scale expression</a>, <a href=dx.doi.org/10.17504/protocols.io.n92ld8yd7v5b/v1>>6L large scale expression</a></cite></div><button data-component=toggleme data-target=#togglebox-details-ASAP-EVA71-3CLPRO-1 data-text="Hide Details">Details</button><div id=togglebox-details-ASAP-EVA71-3CLPRO-1 class=hide><ul><li>Vector: pNIC-NHStIIT</li><li>Entry clone accession:</li><li>Cell line:</li><li>Tags and additions: His-tag</li><li>Construct protein sequence:
MGPSLDFALSLLRRNIRQVQTDQGHFTMLGVRDRLAVLPRHSQPGKTIWVEHKLINILDAVELVDEQGVNLELTLVTLDTNEKFRDITKFIPENISAASDATLVINTEHMPSMFVPVGDVVQYGFLNLSGKPTHRTMMYNFPTKAGQCGGVVTSVGKVIGIHIGGNGRQGFCAGLKRSYFASEQLEHHHHHH</li></ul></div></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href><img title=plasmid width=40 src=../img/icons/plasmid.png></a></div><div id=eva71-3C-protease_plasmid-crystallography class=col-11><b>Protein construct of inactive mutant of EV-A71 3C protease suitbale for crystallography</b><br>Construct with Avi-tag suitable for biophysical assays requiring protein immobilization<br><b>SCARAB ID:</b>
<img width=10 src=images/icons/scarab.png>
A71EV3CPROA-c002<br><cite>[ 2022-10-24 ]</cite><br><button data-component=toggleme data-target=#togglebox-protocol-ASAP-EVA71-3CLPRO-2 data-text="Hide Protocol">Protocol</button><div id=togglebox-protocol-ASAP-EVA71-3CLPRO-2 class=hide><cite>The construct was transformed into the <em>E. coli</em> strain BL21(DE3)-RR, and cells were grown at 37°C in TB medium supplemented with kanamycin (50 µg/ml). After reaching an optical density at 600 nm of around 1, , the temperature was lowered to 18°C. After 1 hour 0.5 mM D-biotin was added and then after a further 1 hour induction of protein expression was induced by adding 0.5 mM IPTG and growth continued overnight. Harvested cells were resuspended in lysis buffer [10 mM Hepes (pH 7.5), 500 mM NaCl, 5% glycerol, 30 mM imidazole, 0.5 mM TCEP, 1% TX-100, 0.5 mg/mL lysozyme, 0.05 mg/mL benzonase]. Proteins were first purified by immobilized metal affinity chromatography (IMAC) using Ni-Sepharose 6 FF resin (Cytiva) the column was washed with binding buffer [10 mM Hepes (pH 7.5), 500 mM NaCl, 5% glycerol, 30 mM imidazole, 0.5 mM TCEP] and target protein eluted using same buffer containing 500 mM imidazole. Protein was lastly purified by SEC (Superose 12 pg,Cytiva) in a buffer consisting of 10 mM Hepes (pH 7.5), 500 mM NaCl, 5% glycerol and 0.5 mM TCEP. Proteins were characterised by SDS-polyacrylamide gel electrophoresis andd mass spec and then flash- frozen in liquid nitrogen, and stored at -80°C until required. For protein crystallization protein was exchanged into a buffer of 10 mM Bicine pH 8.5, 500 mM NaCl, 5 % Glycerol, 0.5 mM TCEP to allow a higher final protein concentration.</cite></div><button data-component=toggleme data-target=#togglebox-details-ASAP-EVA71-3CLPRO-2 data-text="Hide Details">Details</button><div id=togglebox-details-ASAP-EVA71-3CLPRO-2 class=hide><ul><li>Vector: pNIC-NHStIIT</li><li>Entry clone accession:</li><li>Cell line:</li><li>Tags and additions: His-tag</li><li>Construct protein sequence:
MGPSLDFALSLLRRNIRQVQTDQGHFTMLGVRDRLAVLPRHSQPGKTIWVEHKLINILDAVELVDEQGVNLELTLVTLDTNEKFRDITKFIPENISAASDATLVINTEHMPSMFVPVGDVVQYGFLNLSGKPTHRTMMYNFPTKAGQAGGVVTSVGKVIGIHIGGNGRQGFCAGLKRSYFASEQLEHHHHHH</li></ul></div></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href><img title=plasmid width=40 src=../img/icons/plasmid.png></a></div><div id=eva71-3C-protease_plasmid-biophysical class=col-11><b>Protein construct of EV-A71 3C protease with Avi-tag suitable for biophysical assays</b><br>Construct encoding an inactive version of the protein for crystallization controle.<br><b>SCARAB ID:</b>
<img width=10 src=images/icons/scarab.png>
A71EV3CPROA-c003<br><cite>[ 2022-10-24 ]</cite><br><button data-component=toggleme data-target=#togglebox-protocol-ASAP-EVA71-3CLPRO-3 data-text="Hide Protocol">Protocol</button><div id=togglebox-protocol-ASAP-EVA71-3CLPRO-3 class=hide><cite>The construct was transformed into the E. coli strain BL21(DE3)-RR-BirA (pCDF-BirA), and cells were grown at 37°C in TB medium supplemented with kanamycin (50 µg/ml) and streptomycin (50 µg/ml). After reaching an optical density at 600 nm of around 1, the temperature was lowered to 18°C. After 30 minutes 0.5 mM D-biotin was added and then after a further 30 minutes induction of protein expression was induced by adding 0.5 mM IPTG and growth continued overnight. Harvested cells were resuspended in lysis buffer [10 mM Hepes (pH 7.5), 500 mM NaCl, 5% glycerol, 30 mM imidazole, 0.5 mM TCEP, 1% TX-100, 0.5 mg/mL lysozyme, 0.05 mg/mL benzonase, 0.5 mM D-Biotin]. Proteins were first purified by immobilized metal affinity chromatography (IMAC) using Ni-Sepharose 6 FF resin (Cytiva) the column was washed with binding buffer [10 mM Hepes (pH 7.5), 500 mM NaCl, 5% glycerol, 30 mM imidazole, 0.5 mM TCEP] and target protein eluted using same buffer containing 500 mM imidazole. Protein was lastly purified by SEC (Superose 12 pg,Cytiva) in a buffer consisting of 10 mM Hepes (pH 7.5), 500 mM NaCl, 5% glycerol and 0.5 mM TCEP. Proteins were characterised by SDS-polyacrylamide gel electrophoresis andd mass spec and then flash-frozen in liquid nitrogen,and stored at -80°C until required.</cite></div><button data-component=toggleme data-target=#togglebox-details-ASAP-EVA71-3CLPRO-3 data-text="Hide Details">Details</button><div id=togglebox-details-ASAP-EVA71-3CLPRO-3 class=hide><ul><li>Vector: pNIC-NHStIIT</li><li>Entry clone accession:</li><li>Cell line:</li><li>Tags and additions: His-Avi-tag</li><li>Construct protein sequence:
MGPSLDFALSLLRRNIRQVQTDQGHFTMLGVRDRLAVLPRHSQPGKTIWVEHKLINILDAVELVDEQGVNLELTLVTLDTNEKFRDITKFIPENISAASDATLVINTEHMPSMFVPVGDVVQYGFLNLSGKPTHRTMMYNFPTKAGQCGGVVTSVGKVIGIHIGGNGRQGFCAGLKRSYFASEQLEHHHHHHGLNDIFEAQKIEWHE</li></ul></div></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href><img title=biochemical-assay width=40 src=../img/icons/biochemical-assay.png></a></div><div id=eva71-3C-protease_biochemical-assay class=col-11><b>EV-A71 3C protease activity biochemical assay using fluorescence intensity measurements</b><br>This method is intended to measure the activity of viral proteases by using a specific labeled peptide that allows the detection of the cleaved product. The substrate contains the cleavage-sequence specific to the tested protease and is labeled in C-terminal by the fluorophore Edans (abs 336 nm; em 490 nm) and in N-terminal by the quencher Dabcyl (abs 472 nm). In the case of a non-cleaved substrate, the proximity of Dabcyl to Edans prevents the emission and the detection of the fluorescence at 490 nm. The cleavage of the peptide by the protease allows Edans’ fluorescence emission and detection.<br><cite>[ 2023-04-24 ]</cite><br><button data-component=toggleme data-target=#togglebox-protocol-ASAP-EVA71-3CLPRO-4 data-text="Hide Protocol">Protocol</button><div id=togglebox-protocol-ASAP-EVA71-3CLPRO-4 class=hide><cite>Available at <a href=dx.doi.org/10.17504/protocols.io.261ge54jyg47/v1>protocols.io</a></cite></div></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href=https://www.rcsb.org/structure/8CNY><img title=crystallization-protocol width=40 src=../img/icons/crystallization-protocol.png></a></div><div id=eva71-3C-protease_crystallization-protocol class=col-11><b>Crystallization protocol for producing soakable crystals suitable for fragment screening</b><br><a href=https://www.rcsb.org/structure/8CNY>https://www.rcsb.org/structure/8CNY</a>
<cite>[ 2023-02-28 ]</cite><br><button data-component=toggleme data-target=#togglebox-protocol-ASAP-EVA71-3CLPRO-5 data-text="Hide Protocol">Protocol</button><div id=togglebox-protocol-ASAP-EVA71-3CLPRO-5 class=hide><cite>Available at <a href=http://dx.doi.org/10.17504/protocols.io.yxmvm3e89l3p/v1>protocols.io</a></cite></div><button data-component=toggleme data-target=#togglebox-details-ASAP-EVA71-3CLPRO-5 data-text="Hide Details">Details</button><div id=togglebox-details-ASAP-EVA71-3CLPRO-5 class=hide><a href="/pdfs/PAGE23-00333 - AVIDD ASAP; A71EV3CPROA & D68EV3CPROA & A71EV2A Avitag insertion and C2A mutations.pdf" download>Download experimental metadata 1</a> <a href="/pdfs/PAGE22-01976 - AVIDD ASAP; EVD68 & EVA71 3C protease expression and purification.pdf" download>Download experimental metadata 2</a> <a href="/pdfs/TEP EV-A71 3C PAGE23-00676.pdf" download>Download experimental metadata 3</a></div></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href=https://www.rcsb.org/structure/8CNY><img title=pdb-deposition width=40 src=../img/icons/pdb-deposition.png></a></div><div id=eva71-3C-protease_apo-structure class=col-11><b>Structure of Enterovirus A71 3C protease</b><br>Apo structure of EV-A71 3C protease<br><a href=https://www.rcsb.org/structure/8CNY>https://www.rcsb.org/structure/8CNY</a>
<cite>[ 2023-04-05 ]</cite><br></div></div></div></div><hr style=height:5px;border:none;color:#333;background-color:#333><br><br><br><h3 style=text-align:center id=ASAP-EVD68-2APRO>Enterovirus D68 2A protease</h3><div class="row align-center"><a href=https://www.ncbi.nlm.nih.gov/nuccore/KM881710><span class="label success">Genetic source </span></a>&nbsp;
<a href=https://www.ncbi.nlm.nih.gov/protein/KM881710.2><span class="label success">Gene: KM881710.2 </span></a>&nbsp;
<a href=https://www.uniprot.org/uniprotkb/Q68T42><span class="label error">Uniprot: Q68T42 </span></a>&nbsp;
<a href="https://www.ebi.ac.uk/intenz/query?cmd=SearchEC&amp;ec=3.4.22.29"><span class="label warning">EC: 3.4.22.29</span></a></div><br><div class="row align-center"><table class=unstyled><tr><td><img title="Crystallography construct" src=/images/icons/tep-stage-not-started_hu0ad6bc3691333ead921c5ce1ee694314_9444_c8505e58db16a9e7e563ff075d12e98d.png width=300 height=88><br><img title="Assay construct" src=/images/icons/tep-stage-not-started_hu0ad6bc3691333ead921c5ce1ee694314_9444_6b34b3352fefa968495b3f939ea766b4.png width=300 height=88></td><td><img title="Biophysical assay" src=/images/icons/tep-stage-not-started_hu0ad6bc3691333ead921c5ce1ee694314_9444_3996d997df11e427d2dc236cf2e2aca0.png width=300 height=55><br><img title="Robust crystals" src=/images/icons/tep-stage-not-started_hu0ad6bc3691333ead921c5ce1ee694314_9444_4cabaeabe4dbf3d7edcc15afff41b6c8.png width=300 height=55><br><img title="Biochemical assay" src=/images/icons/tep-stage-not-started_hu0ad6bc3691333ead921c5ce1ee694314_9444_89318d210a91282b415bc8fc832fc6f1.png width=300 height=55></td><td><img title="X-ray fragment screen and structural analysis" src=/images/icons/tep-stage-not-started_hu0ad6bc3691333ead921c5ce1ee694314_9444_00692520805a7a4b5cb143ae16450ab1.png width=300 height=180></td><td><img title="First-round progression for TEP validation" src=/images/icons/tep-stage-not-started_hu0ad6bc3691333ead921c5ce1ee694314_9444_31ed385b113229ad572b9427160c8c98.png width=300 height=180></td></tr></table></div><b>Contributors:</b>
<span class="label success">Michael Fairhead </span><span class="label success">Lizbé Koekemoer </span><span class="label success">Charline Giroud</span><br><br><div class="prettybox warning" data-component=prettybox><b>Target Nominator:</b> Alpha Lee<br><b>Target PI:</b> Frank von Delft<br><b>Therapeutic Area(s):</b> Infectious diseases<br><b>Disease Relevance:</b> 3C proteases are a clinically validated target.<br><b>Viral family:</b> picronaviridae<br><b>Viruses:</b> enterovirus D68<br></div><div class="prettybox success" data-component=prettybox><h4>Summary of Project</h4>The conserved 2A protease of enterovirus D68 is required to cleave the nonstructural region of the viral polyprotein. This TEP aims to provide early tools for structure-based discovery of 3C protease inhibitors, including purification protocols of recombinant proteins, and reproducible crystallisation condition suitable for X-ray crystallography fragment screening.</div><div class="prettybox focus" data-component=prettybox><h4>Scientific Background</h4>While enterovirus D68 (EV-D68) typically causes respiratory illness, which can be mild or severe, it can also cause acute flaccid myelitis (AFM), a serious neurological condition which mostly affects children that causes the muscles and reflexes in the body to become weak (via CDC).</div><h4>Resources</h4><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href><img title=biochemical-assay width=40 src=../img/icons/biochemical-assay.png></a></div><div id=evd68-3A-protease_biochemical-assay class=col-11><b>EV-D68 2A protease activity biochemical assay using fluorescence intensity measurements</b><br>This method is intended to measure the activity of viral proteases by using a specific labelled-peptide that allows the detection of the cleaved product. The substrate contains the cleavage-sequence specific to the tested protease and is labeled in C-terminal by the fluorophore Edans (ex 336 nm; em: 455 nm) and in N-ternimal by the quencher Dabcyl (abs 472 nm). In the case of a non-cleaved substrate, the proximity of Dabcyl to Edans prevents the emission and the detection of the fluorescence at 455 nm. The cleavage of the peptide by the protease allows Edans’ fluorescence emission and detection.<br><cite>[ 2023-04-24 ]</cite><br><button data-component=toggleme data-target=#togglebox-protocol-ASAP-EVD68-2APRO-1 data-text="Hide Protocol">Protocol</button><div id=togglebox-protocol-ASAP-EVD68-2APRO-1 class=hide><cite>Assay buffer: 50 mM Tris pH 7.0, 150 mM NaCl, 10% glycerol and 0.5 mM DTT (optional).
Incubation: 1 hour at room temperature.
EV-A71 2A: protein stocks were stored at -80C and used as 2x solution (20 µM, 10 µM final assay concentration) in assay buffer.
Substrate: Dabcyl-TAITTLGKFGQE-Edans (LifeTein, USA) prepared as a stock solution at 5 mM in DMSO and used at 2x solution (40 µM, 20 µM final concentration assay concentration) in assay buffer.
Positive control: Telaprevir (Pubchem CID 3010818), 50 µM top final assay concentration.
Plates: 384 well plate, white, f-bottom (Greiner #781074).
Liquid handler: Echo® acoustic liquid handler (Beckman Coulter, USA).
Plate reader: Pherastar FS, BMG Labtech (Germany), 350-460 FI optic module, the plate is read every 30 s for 2 hours and shacked during 5 s before the first reading.
Method: Fifty µL of 2x protein solution were added to each well containing the compounds to be tested previously dispensed onto the plate. The mix has been incubated for one hour at room temperature and the enzymatic reaction was initiated by the addition of 50 µL of 2x (40 µM) substrate solution using the plate reader injector. The fluorescence intensity at 460 nm was read every 30 seconds for 2 hours in kinetic mode, which include a shaking step of the plate between each measurement.
The IC&lt;50> was calculated by plotting the initial velocity against various concentrations of tested inhibitor by using a four parameter dose−response curve in Prism (v8.0) software.</cite></div></div></div></div></div><hr style=height:5px;border:none;color:#333;background-color:#333><br><br><br><h3 style=text-align:center id=ASAP-EVD68-3CLPRO>Enterovirus D68 3C protease</h3><div class="row align-center"><a href=https://www.ncbi.nlm.nih.gov/protein/KM881710.2><span class="label success">Gene: KM881710.2 </span></a>&nbsp;
<a href=https://www.uniprot.org/uniprotkb/Q68T42%20%28closest%20match%29><span class="label error">Uniprot: Q68T42 (closest match) </span></a>&nbsp;
<a href="https://www.ebi.ac.uk/intenz/query?cmd=SearchEC&amp;ec=3.4.22.28"><span class="label warning">EC: 3.4.22.28</span></a></div><br><div class="row align-center"><table class=unstyled><tr><td><img title="Crystallography construct" src=/images/icons/tep-stage-completed_hub021b18cb248db28b117f458e39aeb5b_10162_e22862b00861f0d3434857db1ff02de5.png width=300 height=88><br><img title="Assay construct" src=/images/icons/tep-stage-completed_hub021b18cb248db28b117f458e39aeb5b_10162_2851d035e13cbc115749c33624801624.png width=300 height=88></td><td><img title="Biophysical assay" src=/images/icons/tep-stage-in-progress_huc9d943f35f5911a48f54b3ad3b40c43a_10037_80222a2cb4b3b4d0cafa06d6608261b8.png width=300 height=55><br><img title="Robust crystals" src=/images/icons/tep-stage-completed_hub021b18cb248db28b117f458e39aeb5b_10162_e3383ab8d75acf7a535bdc511b331b23.png width=300 height=55><br><img title="Biochemical assay" src=/images/icons/tep-stage-completed_hub021b18cb248db28b117f458e39aeb5b_10162_1b7ab3b95a075266c44f73afb56d1cd3.png width=300 height=55></td><td><img title="X-ray fragment screen and structural analysis" src=/images/icons/tep-stage-completed_hub021b18cb248db28b117f458e39aeb5b_10162_ae1969e18479b9f6892265d5fd14a303.png width=300 height=180></td><td><img title="First-round progression for TEP validation" src=/images/icons/tep-stage-in-progress_huc9d943f35f5911a48f54b3ad3b40c43a_10037_0bf41094ba248abeddae737c4c98b352.png width=300 height=180></td></tr></table></div><b>Contributors:</b>
<span class="label success">Ryan Lithgo </span><span class="label success">Annette von Delft </span><span class="label success">Blake Balcomb </span><span class="label success">Charlie Tomlinson </span><span class="label success">Charline Giroud </span><span class="label success">Daren Fearon </span><span class="label success">Dong Wang </span><span class="label success">Eleanor Williams </span><span class="label success">Jasmin Aschenbrenner </span><span class="label success">Jessica Dixon </span><span class="label success">Korvus Wang </span><span class="label success">Lee Harris </span><span class="label success">Lizbé Koekemoer </span><span class="label success">Lucas Ferreira </span><span class="label success">Michael Fairhead </span><span class="label success">Oleg Fedorov </span><span class="label success">Peter Marples </span><span class="label success">Tamas Szommer </span><span class="label success">Warren Thompson </span><span class="label success">Xiaomin Ni</span><br><br><div class="prettybox warning" data-component=prettybox><b>Target Nominator:</b> Alpha Lee<br><b>Target PI:</b> Frank von Delft<br><b>Therapeutic Area(s):</b> Infectious diseases<br><b>Disease Relevance:</b> 3C proteases are a clinically validated target.<br><b>Viral family:</b> picornaviridae<br><b>Viruses:</b> enterovirus D68<br></div><div class="prettybox success" data-component=prettybox><h4>Summary of Project</h4>The conserved 3C protease of enterovirus D68 is required to cleave the nonstructural region of the viral polyprotein. This TEP aims to provide early tools for structure-based discovery of 3C protease inhibitors, including purification protocols of recombinant proteins, and reproducible crystallisation condition suitable for X-ray crystallography fragment screening.</div><div class="prettybox focus" data-component=prettybox><h4>Scientific Background</h4>While enterovirus D68 (EV-D68) typically causes respiratory illness, which can be mild or severe, it can also cause <a href=https://www.cdc.gov/acute-flaccid-myelitis/index.html>acute flaccid myelitis (AFM)</a>, a serious neurological condition which mostly affects children that causes the muscles and reflexes in the body to become weak (via <a href=https://www.cdc.gov/non-polio-enterovirus/about/ev-d68.html>CDC</a>). Enterovirus 3C protease is required to cleave the nonstructural region of the viral polyprotein. 3C also cleaves many host proteins to facilitate viral infection; thus, inhibition can reinstate the antiviral functions of some of these targets. The protease has a typical chymotrypsin fold with a Cys-His-Glu catalytic triad.</div><div class="prettybox error" data-component=prettybox><h4>Conclusion</h4>Here we have established a purification protocol for recombinant 3C protease from enterovirus D68 and established a robust crystallisation suitable for fragment screening. We have performed a substantial fragment screening campaign and identified many hits which were validated by biophysical assays established in this work. The ligands and assays established in this work will pave the way for future drug discovery endeavours targeting enterovirus D68 3C protease.</div><h4>Resources</h4><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href><img title=plasmid width=40 src=../img/icons/plasmid.png></a></div><div id=evd68-3C-protease_plasmid-crystallography class=col-11><b>EV-D68 3C protease construct suitable for crystallography in space group P21<sub>21</sub>21.</b><br>EV-D68 bearing a C-terminal His-tag that crystallised in the space group P2<sub>1</sub>2<sub>1</sub>2<sub>1</sub>. Enterovirus D68 3C protease construct bearing a C-terminal His-tag that crystallised in the space group P2<sub>1</sub>2<sub>1</sub>2<sub>1</sub>. This protein yielded highly reproducible crystals upon microseeding, which typically diffracted to 1.7 Å resolution. This crystal system was DMSO tolerant and therefore suitable for our subsequent fragment soaking.<br><b>SCARAB ID:</b>
<img width=10 src=images/icons/scarab.png>
D68EV3CPROA-c001<br><cite>[ 2022-03-24 ]</cite><br><button data-component=toggleme data-target=#togglebox-protocol-ASAP-EVD68-3CLPRO-1 data-text="Hide Protocol">Protocol</button><div id=togglebox-protocol-ASAP-EVD68-3CLPRO-1 class=hide><cite>Available at <a href=dx.doi.org/10.17504/protocols.io.dm6gpzrkdlzp/v1>protocols.io</a></cite></div><button data-component=toggleme data-target=#togglebox-details-ASAP-EVD68-3CLPRO-1 data-text="Hide Details">Details</button><div id=togglebox-details-ASAP-EVD68-3CLPRO-1 class=hide><p><a href="/pdfs/PAGE22-01976 - AVIDD ASAP; EVD68 & EVA71 3C protease expression and purification.pdf" download>Download experimental metadata</a></p><ul><li>Vector: pNIC</li><li>Entry clone accession:</li><li>Cell line: E. coli Rosetta strain BL21(DE3)-RR</li><li>Tags and additions: C-terminal, non-cleavable hexahistidine</li><li>Construct protein sequence: MGPGFDFAQAIMKKNTVIARTEKGEFTMLGVYDRVAVIPTHASVGEIIYINDVETRVLDACALRDLTDTNLEITIVKLDRNQKFRDIRHFLPRCEDDYNDAVLSVHTSKFPNMYIPVGQVTNYGFLNLGGTPTHRILMYNFPTRAGQCGGVVTTTGKVIGIHVGGNGAQGFAAMLLHSYFTDTQKHHHHHH</li></ul></div></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href><img title=plasmid width=40 src=../img/icons/plasmid.png></a></div><div id=evd68-3C-protease_plasmid-biophysics class=col-11><b>EV-D68 3C protease construct suitable for biophysical assays that require immobilization (Avi-tag)</b><br>Assay construct with Avi-tag for biophysics<br><b>SCARAB ID:</b>
<img width=10 src=images/icons/scarab.png>
D68EV3CPROA-c002<br><cite>[ 2022-03-24 ]</cite><br><button data-component=toggleme data-target=#togglebox-protocol-ASAP-EVD68-3CLPRO-2 data-text="Hide Protocol">Protocol</button><div id=togglebox-protocol-ASAP-EVD68-3CLPRO-2 class=hide><cite>The construct was transformed into the E. coli strain BL21(DE3)-RR-BirA (pCDF-BirA), and cells were grown at 37°C in TB medium supplemented with kanamycin (50 µg/ml) and streptomycin (50 µg/ml). After reaching an optical density at 600 nm of around 1, the temperature was lowered to 18°C. After 30 minutes 0.5 mM D-biotin was added and then after a further 30 minutes induction of protein expression was induced by adding 0.5 mM IPTG and growth continued overnight. Harvested cells were resuspended in lysis buffer [10 mM HEPES (pH 7.5), 500 mM NaCl, 5% glycerol, 30 mM imidazole, 0.5 mM TCEP, 1% TX-100, 0.5 mg/mL lysozyme, 0.05 mg/mL benzonase, 0.5 mM D-Biotin]. Proteins were first purified by immobilized metal affinity chromatography (IMAC) using Ni-Sepharose 6 FF resin (Cytiva) the column was washed with binding buffer [10 mM HEPES (pH 7.5), 500 mM NaCl, 5% glycerol, 30 mM imidazole, 0.5 mM TCEP] and target protein eluted using same buffer containing 500 mM imidazole. Protein was lastly purified by SEC (Superose 12 pg,Cytiva) in a buffer consisting of 10 mM HEPES (pH 7.5), 500 mM NaCl, 5% glycerol and 0.5 mM TCEP. Proteins were characterised by SDS-polyacrylamide gel electrophoresis and mass spec and then flash- frozen in liquid nitrogen,and stored at -80°C until required.</cite></div><button data-component=toggleme data-target=#togglebox-details-ASAP-EVD68-3CLPRO-2 data-text="Hide Details">Details</button><div id=togglebox-details-ASAP-EVD68-3CLPRO-2 class=hide><ul><li>Vector: pNIC</li><li>Entry clone accession:</li><li>Cell line: E. coli Rosetta strain BL21(DE3)-RR</li><li>Tags and additions: C-terminal, non-cleavable hexahistidine</li><li>Construct protein sequence: MGPGFDFAQAIMKKNTVIARTEKGEFTMLGVYDRVAVIPTHASVGEIIYINDVETRVLDACALRDLTDTNLEITIVKLDRNQKFRDIRHFLPRCEDDYNDAVLSVHTSKFPNMYIPVGQVTNYGFLNLGGTPTHRILMYNFPTRAGQCGGVVTTTGKVIGIHVGGNGAQGFAAMLLHSYFTDTQKHHHHHHGLNDIFEAQKIEWHE</li></ul></div></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href><img title=assay width=40 src=../img/icons/assay.png></a></div><div id=evd68-3C-protease_protocol-biochemical class=col-11><b>Biochemical assay protocol for EV-D68 3C protease activity measurements using fluorescence detection</b><br>This method is intended to measure the activity of viral proteases by using a specific labelled-peptide that allows the detection of the cleaved product. The substrate contains the cleavage-sequence specific to the tested protease and is labeled in C-terminal by the fluorophore Edans (abs 336 nm; em 490 nm) and in N-terminal by the quencher Dabcyl (abs 472 nm). In the case of a non-cleaved substrate, the proximity of Dabcyl to Edans prevents the emission and the detection of the fluorescence at 490 nm. The cleavage of the peptide by the protease allows Edans’ fluorescence emission and detection.<br><cite>[ 2023-02-28 ]</cite><br><button data-component=toggleme data-target=#togglebox-protocol-ASAP-EVD68-3CLPRO-3 data-text="Hide Protocol">Protocol</button><div id=togglebox-protocol-ASAP-EVD68-3CLPRO-3 class=hide><cite>Available at <a href=dx.doi.org/10.17504/protocols.io.261ge54jyg47/v1>protocols.io</a>
<img src=../img/outputs/EV-D68-3C-protease/dose_response.png alt="Example assay data"></cite></div><button data-component=toggleme data-target=#togglebox-details-ASAP-EVD68-3CLPRO-3 data-text="Hide Details">Details</button><div id=togglebox-details-ASAP-EVD68-3CLPRO-3 class=hide><a href="/pdfs/PAGE23-00660 - EV-D68 3C pro vs GC376 IC50.pdf" download>Download experimental metadata 1</a>
<a href="/pdfs/TEP EV-D68 3C PAGE23-00660.pdf" download>Download experimental metadata 2</a></div></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href><img title=crystallization-protocol width=40 src=../img/icons/crystallization-protocol.png></a></div><div id=evd68-3C-protease_crystallization-protocol class=col-11><b>Crystallization protocol for EV-D68 3C protease suitable for fragment screening</b><br>Optimised conditions to reproduce crystals for fragment screening.<br><cite>[ 2023-02-28 ]</cite><br><button data-component=toggleme data-target=#togglebox-protocol-ASAP-EVD68-3CLPRO-4 data-text="Hide Protocol">Protocol</button><div id=togglebox-protocol-ASAP-EVD68-3CLPRO-4 class=hide><cite>Available at <a href=http://dx.doi.org/10.17504/protocols.io.n92ldm6jnl5b/v1>protocols.io</a></cite></div></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href=https://www.rcsb.org/structure/8CNX><img title=pdb-deposition width=40 src=../img/icons/pdb-deposition.png></a></div><div id=evd68-3C-protease_apo-structure class=col-11><b>Structure of Enterovirus D68 3C protease</b><br>Apo structure of EV-D68 3C protease<br><a href=https://www.rcsb.org/structure/8CNX>https://www.rcsb.org/structure/8CNX</a>
<cite>[ 2023-04-05 ]</cite><br></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href=https://fragalysis-legacy.xchem.diamond.ac.uk/viewer/react/preview/target/D68EV3CPROA/tas/lb18145-1><img title=xray-fragment-screen width=40 src=../img/icons/xray-fragment-screen.png></a></div><div id=evd68-3C-protease_fragment-screening-fragalysis class=col-11><b>Fragalysis interactive exploration of X-ray fragment screen of EV-D68 3C protease</b><br>We performed X-ray crystallography screening at XChem/Diamond Light Source with a total of 1231 fragments.<br><a href=https://fragalysis-legacy.xchem.diamond.ac.uk/viewer/react/preview/target/D68EV3CPROA/tas/lb18145-1>https://fragalysis-legacy.xchem.diamond.ac.uk/viewer/react/preview/target/D68EV3CPROA/tas/lb18145-1</a>
<cite>[ 2023-08-24 ]</cite><br><button data-component=toggleme data-target=#togglebox-protocol-ASAP-EVD68-3CLPRO-6 data-text="Hide Protocol">Protocol</button><div id=togglebox-protocol-ASAP-EVD68-3CLPRO-6 class=hide><cite>Fragments were soaked into crystals as previously described by adding dissolved compounds directly to the crystallisation drops using an Echo liquid handler (final concentration, 10% DMSO); drops were incubated for approximately 1 to 3 hours before mounting and flash-freezing in liquid nitrogen. Data were collected at the I04-1 beamline at 100 K and automatically processed with Diamond Light Source&rsquo;s autoprocessing pipelines using XDS (9) and either <a href=https://doi.org/10.1107/s0907444913015308>xia2</a> or <a href=https://doi.org/10.1107/s2059798317017235>DIALS</a>. Further analysis was performed with <a href=https://doi.org/10.1107%2FS2059798316020234>XChemExplorer</a>, electron density maps were generated with DIMPLE and ligand-binding events were identified using <a href=https://doi.org/10.1038/ncomms15123>PanDDA</a>. Ligands were modeled into PanDDA-calculated event maps using Coot, restraints were calculated with <a href=http://dx.doi.org/10.1107/S2059798317000067>AceDRG</a>, and structures were refined with <a href=https://doi.org/10.1107/s0907444992010400>BUSTER</a>.</cite></div></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href=https://www.rcsb.org/groups/summary/entry/G_1002271><img title=pdb-group-deposition width=40 src=../img/icons/pdb-group-deposition.png></a></div><div id=evd68-3C-protease_fragment-screen-pdb-group-deposition class=col-11><b>PDB group deposition of X-ray fragment screen of EV-D68 3C protease</b><br>We performed X-ray crystallography screening at XChem/Diamond Light Source with a total of 1231 fragments, generating 104 total structures of novel protein:ligand complexes.<br><a href=https://www.rcsb.org/groups/summary/entry/G_1002271>https://www.rcsb.org/groups/summary/entry/G_1002271</a>
<cite>[ 2023-08-24 ]</cite><br><button data-component=toggleme data-target=#togglebox-protocol-ASAP-EVD68-3CLPRO-7 data-text="Hide Protocol">Protocol</button><div id=togglebox-protocol-ASAP-EVD68-3CLPRO-7 class=hide><cite>Fragments were soaked into crystals as previously described by adding dissolved compounds directly to the crystallisation drops using an Echo liquid handler (final concentration, 10% DMSO); drops were incubated for approximately 1 to 3 hours before mounting and flash-freezing in liquid nitrogen. Data were collected at the I04-1 beamline at 100 K and automatically processed with Diamond Light Source&rsquo;s autoprocessing pipelines using XDS (9) and either <a href=https://doi.org/10.1107/s0907444913015308>xia2</a> or <a href=https://doi.org/10.1107/s2059798317017235>DIALS</a>. Further analysis was performed with <a href=https://doi.org/10.1107%2FS2059798316020234>XChemExplorer</a>, electron density maps were generated with DIMPLE and ligand-binding events were identified using <a href=https://doi.org/10.1038/ncomms15123>PanDDA</a>. Ligands were modeled into PanDDA-calculated event maps using Coot, restraints were calculated with <a href=http://dx.doi.org/10.1107/S2059798317000067>AceDRG</a>, and structures were refined with <a href=https://doi.org/10.1107/s0907444992010400>BUSTER</a>. PDB codes: 7GNV, 7GNW, 7GNX, 7GNY, 7GNZ, 7GO0, 7GO1, 7GO2, 7GO3, 7GO4, 7GO5, 7GO6, 7GO7, 7GO8, 7GO9, 7GOA, 7GOB, 7GOC, 7GOD, 7GOE, 7GOF, 7GOG, 7GOH, 7GOI, 7GOJ, 7GOK, 7GOL, 7GOM, 7GON, 7GOO, 7GOP, 7GOQ, 7GOR, 7GOS, 7GOT, 7GOU, 7GOV, 7GOW, 7GOX, 7GOY, 7GOZ, 7GP0, 7GP1, 7GP2, 7GP3, 7GP4, 7GP5, 7GP6, 7GP7, 7GP8, 7GP9, 7GPA, 7GPC, 7GPD, 7GPE, 7GPF, 7GPG, 7GPH, 7GPI, 7GPJ, 7GPK, 7GPL, 7GPM, 7GPN, 7GPO, 7GPP, 7GPQ, 7GPR, 7GPS, 7GPT, 7GPU, 7GPV, 7GPW, 7GPX, 7GPY, 7GPZ, 7GQ0, 7GQ1, 7GQ2, 7GQ3, 7GQ4, 7GQ5, 7GQ6, 7GQ7, 7GQ8, 7GQ9, 7GQA, 7GQB, 7GQC, 7GQD, 7GQE, 7GQF, 7GQG, 7GQH, 7GQI, 7GQJ, 7GQK, 7GQL, 7GQM, 7GQN, 7GQO, 7GQP, 7GQQ, 7GQR</cite></div></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href=https://doi.org/10.1101/2024.04.29.591650><img title=preprint width=40 src=../img/icons/preprint.png></a></div><div id=evd68-3C-protease_xray-fragment-screen-preprint class=col-11><b>Crystallographic fragment screen of Enterovirus D68 3C protease and iterative design of lead-like compounds using structure-guided expansions</b><br>Preprint describing X-ray fragment screen against EV-D68 3C protease<br><a href=https://doi.org/10.1101/2024.04.29.591650>https://doi.org/10.1101/2024.04.29.591650</a>
<cite>[ 2024-05-01 ]</cite><br></div></div></div></div><hr style=height:5px;border:none;color:#333;background-color:#333><br><br><br><h3 style=text-align:center id=ASAP-DENV-NS2B-NS3>Dengue NS2B-NS3 protease</h3><div class="row align-center"><a href=https://www.ncbi.nlm.nih.gov/protein/U87411.1><span class="label success">Gene: U87411.1 </span></a>&nbsp;
<a href=https://www.uniprot.org/uniprotkb/P14340><span class="label error">Uniprot: P14340 </span></a>&nbsp;
<a href="https://www.ebi.ac.uk/intenz/query?cmd=SearchEC&amp;ec=3.4.21.91"><span class="label warning">EC: 3.4.21.91</span></a></div><br><div class="row align-center"><table class=unstyled><tr><td><img title="Crystallography construct" src=/images/icons/tep-stage-completed_hub021b18cb248db28b117f458e39aeb5b_10162_e22862b00861f0d3434857db1ff02de5.png width=300 height=88><br><img title="Assay construct" src=/images/icons/tep-stage-completed_hub021b18cb248db28b117f458e39aeb5b_10162_2851d035e13cbc115749c33624801624.png width=300 height=88></td><td><img title="Biophysical assay" src=/images/icons/tep-stage-in-progress_huc9d943f35f5911a48f54b3ad3b40c43a_10037_80222a2cb4b3b4d0cafa06d6608261b8.png width=300 height=55><br><img title="Robust crystals" src=/images/icons/tep-stage-in-progress_huc9d943f35f5911a48f54b3ad3b40c43a_10037_f71a74ae374dc1fa70d31362f03e7c29.png width=300 height=55><br><img title="Biochemical assay" src=/images/icons/tep-stage-completed_hub021b18cb248db28b117f458e39aeb5b_10162_1b7ab3b95a075266c44f73afb56d1cd3.png width=300 height=55></td><td><img title="X-ray fragment screen and structural analysis" src=/images/icons/tep-stage-not-started_hu0ad6bc3691333ead921c5ce1ee694314_9444_00692520805a7a4b5cb143ae16450ab1.png width=300 height=180></td><td><img title="First-round progression for TEP validation" src=/images/icons/tep-stage-in-progress_huc9d943f35f5911a48f54b3ad3b40c43a_10037_0bf41094ba248abeddae737c4c98b352.png width=300 height=180></td></tr></table></div><b>Contributors:</b>
<span class="label success">Michael Fairhead </span><span class="label success">Noa Lahav </span><span class="label success">Lizbé Koekemoer </span><span class="label success">Andre de Godoy </span><span class="label success">Blake Balcomb </span><span class="label success">Charlie Tomlinson </span><span class="label success">Charline Giroud </span><span class="label success">Daren Fearon </span><span class="label success">Dong Wang </span><span class="label success">Eleanor Williams </span><span class="label success">Haim Barr </span><span class="label success">Jasmin Aschenbrenner </span><span class="label success">Jessica Dixon </span><span class="label success">Korvus Wang </span><span class="label success">Lee Harris </span><span class="label success">Lucas Ferreira </span><span class="label success">Oleg Fedorov </span><span class="label success">Peter Marples </span><span class="label success">Ryan Lithgo </span><span class="label success">Tamas Szommer </span><span class="label success">Warren Thompson </span><span class="label success">Xiaomin Ni</span><br><br><div class="prettybox warning" data-component=prettybox><b>Target Nominator:</b> Karla Kirkegaard<br><b>Target PI:</b> Frank von Delft<br><b>Therapeutic Area(s):</b> Infectious diseases<br><b>Disease Relevance:</b> NS3 encodes protease that is important for processing the viral polyprotein<br><b>Viral family:</b> flaviviridae<br><b>Viruses:</b> [Dengue]<br></div><div class="prettybox success" data-component=prettybox><h4>Summary of Project</h4>This TEP is being compiled to provide early tools to develop NS3 protease inhibitors, including purification protocols of recombinant proteins, and reproducible crystallisation condition suitable for X-ray crystallography fragment screening.</div><div class="prettybox focus" data-component=prettybox><h4>Scientific Background</h4>NS2B/3 performs all cleavages on the cytoplasmic side of the polyprotein.</div><div class="prettybox error" data-component=prettybox><h4>Conclusion</h4>This is a summary of the work we have done on the NS2b-NS3 protease from Dengue viruses to date.</div><h4>Resources</h4><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href><img title=plasmid width=40 src=../img/icons/plasmid.png></a></div><div id=flade-NS2B3-protease_plasmid-crystallography class=col-11><b>Protein expression construct for DENV-2 NS2B-NS3 protease suitable for crystallography.</b><br>Construct expressing the NS2B cofactor fused to the NS3 protease from Dengue virus serotype 2 used for FRET assay.
Generated using golden gate cloning and inverse PCR for construct boundary exploration, based on PDB 5GPI<br><cite>[ 2022-10-24 ]</cite><br><button data-component=toggleme data-target=#togglebox-protocol-ASAP-DENV-NS2B-NS3-1 data-text="Hide Protocol">Protocol</button><div id=togglebox-protocol-ASAP-DENV-NS2B-NS3-1 class=hide><cite>The construct was transformed into the E. coli strain BL21(DE3)-RR, and cells were grown at 37°C in AIM-TB medium supplemented with kanamycin (50 µg/ml). After reaching an optical density at 600 nm of around 1, the temperature was lowered to 18°C before induction of protein expression overnight. Harvested cells were resuspended in lysis buffer [10 mM Hepes (pH 7.5), 500 mM NaCl, 5% glycerol, 30 mM imidazole, 0.5 mM TCEP, 1% TX-100, 0.5 mg/mL lysozyme, 0.05 mg/mL benzonase]. Proteins were first purified by immobilized metal affinity chromatography (IMAC) using Ni-Sepharose 6 FF resin (Cytiva) the column was washed with binding buffer [10 mM Hepes (pH 7.5), 500 mM NaCl, 5% glycerol, 30 mM imidazole, 0.5 mM TCEP] and target protein eluted using same buffer containing 500 mM imidazole. The his twin strep tag was removed by digestion with TEV protease and the mixture dialysed overnight against [10 mM Hepes (pH 7.5), 500 mM NaCl, 5% glycerol, 30 mM imidazole, 0.5 mM TCEP]. Undigested protein, his twin strep tagged protein and his tagged TEV was removed by using Ni-Sepharose 6 FF resin (Cytiva) and the unbound fraction containing untagged protein collected. Protein was lastly purified by SEC (Superose 12 pg, Cytiva) in a buffer consisting of 10 mM Hepes (pH 7.5), 500 mM NaCl, 5% glycerol and 0.5 mM TCEP. Proteins were characterised by SDS-polyacrylamide gel electrophoresis and mass spec and then flash- frozen in liquid nitrogen, and stored at -80°C until required.</cite></div><button data-component=toggleme data-target=#togglebox-details-ASAP-DENV-NS2B-NS3-1 data-text="Hide Details">Details</button><div id=togglebox-details-ASAP-DENV-NS2B-NS3-1 class=hide><ul><li>Vector: pNIC-NHStIIT</li><li>Entry clone accession:</li><li>Cell line: <em>E. coli</em> Rosetta 2 strain BL21(DE3)</li><li>Tags and additions: N-terminal, TEV protease cleavable hexahistidine</li><li>Construct protein sequence:
NS2B3: SMADLELERAADVKWEDQAEISGSSPILSITISEDGSMSIKNEEEEQTLGGGGSGGGGAGVLWDVPSPPPMGKAELEDGAYRIKQKGILGYSQIGAGVYKEGTFHTMWHVTRGAVLMHKGKRIEPSWADVKKDLISYGGGWKLEGEWKEGEEVQVLALEPGKNPRAVQTKPGLFKTNAGTIGAVSLDFSPGTSGSPIIDKKGKVVGLYGNGVVTRSGAYVSAIAQTEKSIEDNPEIEDDIFRK</li></ul></div></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href><img title=plasmid width=40 src=../img/icons/plasmid.png></a></div><div id=flade-NS2B3-protease_plasmid-inactive class=col-11><b>Protein expression construct for inactive mutant of DENV-2 NS2B-NS3 protease</b><br>His to Ala mutation used to inactivate catalytic activity.
Generated using golden gate cloning and inverse PCR for construct boundary exploration, based on PDB 5GPI<br><cite>[ 2022-10-24 ]</cite><br><button data-component=toggleme data-target=#togglebox-protocol-ASAP-DENV-NS2B-NS3-2 data-text="Hide Protocol">Protocol</button><div id=togglebox-protocol-ASAP-DENV-NS2B-NS3-2 class=hide><cite>The construct was transformed into the E. coli strain BL21(DE3)-RR, and cells were grown at 37°C in AIM-TB medium supplemented with kanamycin (50 µg/ml). After reaching an optical density at 600 nm of around 1, the temperature was lowered to 18°C before induction of protein expression overnight. Harvested cells were resuspended in lysis buffer [10 mM Hepes (pH 7.5), 500 mM NaCl, 5% glycerol, 30 mM imidazole, 0.5 mM TCEP, 1% TX-100, 0.5 mg/mL lysozyme, 0.05 mg/mL benzonase]. Proteins were first purified by immobilized metal affinity chromatography (IMAC) using Ni-Sepharose 6 FF resin (Cytiva) the column was washed with binding buffer [10 mM Hepes (pH 7.5), 500 mM NaCl, 5% glycerol, 30 mM imidazole, 0.5 mM TCEP] and target protein eluted using same buffer containing 500 mM imidazole. The his twin strep tag was removed by digestion with TEV protease and the mixture dialysed overnight against [10 mM Hepes (pH 7.5), 500 mM NaCl, 5% glycerol, 30 mM imidazole, 0.5 mM TCEP]. Undigested protein, his twin strep tagged protein and his tagged TEV was removed by using Ni-Sepharose 6 FF resin (Cytiva) and the unbound fraction containing untagged protein collected. Protein was lastly purified by SEC (Superose 12 pg, Cytiva) in a buffer consisting of 10 mM Hepes (pH 7.5), 500 mM NaCl, 5% glycerol and 0.5 mM TCEP. Proteins were characterised by SDS-polyacrylamide gel electrophoresis and mass spec and then flash- frozen in liquid nitrogen, and stored at -80°C until required.</cite></div><button data-component=toggleme data-target=#togglebox-details-ASAP-DENV-NS2B-NS3-2 data-text="Hide Details">Details</button><div id=togglebox-details-ASAP-DENV-NS2B-NS3-2 class=hide><ul><li>Vector: pNIC-NHStIIT</li><li>Entry clone accession:</li><li>Cell line: <em>E. coli</em> Rosetta 2 strain BL21(DE3)</li><li>Tags and additions: N-terminal, TEV protease cleavable hexahistidine</li><li>Construct protein sequence:
NS2B3: SMADLELERAADVKWEDQAEISGSSPILSITISEDGSMSIKNEEEEQTLGGGGSGGGGAGVLWDVPSPPPMGKAELEDGAYRIKQKGILGYSQIGAGVYKEGTFHTMWAVTRGAVLMHKGKRIEPSWADVKKDLISYGGGWKLEGEWKEGEEVQVLALEPGKNPRAVQTKPGLFKTNAGTIGAVSLDFSPGTSGSPIIDKKGKVVGLYGNGVVTRSGAYVSAIAQTEKSIEDNPEIEDDIFRK</li></ul></div></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href><img title=biochemical-assay width=40 src=../img/icons/biochemical-assay.png></a></div><div class=col-11><b>DENV-2 NS2B-NS3 protease activity biochemical assay</b><br><cite>[ 2022-11-01 ]</cite><br><button data-component=toggleme data-target=#togglebox-protocol-ASAP-DENV-NS2B-NS3-3 data-text="Hide Protocol">Protocol</button><div id=togglebox-protocol-ASAP-DENV-NS2B-NS3-3 class=hide><cite><ul><li>Thaw enzymes on ice and centrifuged 5 min, 4 C, 15000 rpm.</li><li>Prepare 0.2 µM of 2X enzyme in volume of 10 µl per well.</li><li>Prepare 10 µM of 2X substrate in volume of 10 µl per well.</li><li>Dispense 10 µl buffer to positive control wells (wells 1,23 - no enzyme, no compound).</li><li>Dispense 10 µl enzyme to wells 2-22,24 (wells 2,24 negative control: enzyme + substrate, no compound)</li><li>Briefly spin down plate and incubate for 2 hours in RT.</li><li>Dispense 10 µl substrate to full plate.</li><li>Briefly spin down plate and incubate for 30 min in RT.</li><li>Read in Pherastar plate reader (BMG) in Ex: 360nm, Em: 470nm, Gain 300.</li></ul></cite></div><button data-component=toggleme data-target=#togglebox-details-ASAP-DENV-NS2B-NS3-3 data-text="Hide Details">Details</button><div id=togglebox-details-ASAP-DENV-NS2B-NS3-3 class=hide><p>Enzyme: DV2-NS2BgsgNS3</p><p>Substrate:</p><ul><li>Bz-Nle-Lys-Lys-Arg-AMC</li><li>CAS # 863975-32-0</li><li>Cayman (#27710)</li><li>5 mg dissolved in 600 µL of DMSO and aliquoted to 50 ul &ndash;> stored at -80C as 10mM</li></ul><p>Buffer: Enzymes storage buffer 10 mM HEPES pH 7.3, 500 mM NaCl, 5% glycerol, 0.5 mM TCEP.</p><p>Assay Buffer (final conc. In plate):</p><ul><li>10 mM HEPES pH 7.3</li><li>50 mM NaCl</li><li>5 % glycerol</li><li>1 mM TCEP</li><li>0.05 % Igepal</li></ul><p>Glycerol, Igepal and TCEP are added to the assay buffer before each run.</p><p>Design:</p><ul><li>10 µL of 10 µM substrate + 10 µL of 0.2 µM enzyme in assay buffer</li><li>(final assay volume: 20 µl).</li><li>Inhibitor compounds were pre-dispensed (from DMSO stock) in duplicates to 384 small volume, black plate (Greiner-784076) and stored in -20 until use (control wells - DMSO only, no compounds).</li><li>Dispensing performed using the GNF dispenser</li></ul><p>Sequence: SMADLELERAADVKWEDQAEISGSSPILSITISEDGSMSIKNE
EEEQTLGGGGSGGGGAGVLWDVPSPPPMGKAELEDGAYRIKQKGILGYSQIGA
GVYKEGTFHTMWHVTRGAVLMHKGKRIEPSWADVKKDLISYGGGWKLEGEWKE
GEEVQVLALEPGKNPRAVQTKPGLFKTNAGTIGAVSLDFSPGTSGSPIIDKKG
KVVGLYGNGVVTRSGAYVSAIAQTEKSIEDNPEIEDDIFRK</p><p>MW: 25849.03, Conc (uM): 217, Extinction Coefficient: 41940</p><p>Example Plate Layout:</p><table><thead><tr><th>Compounds conc. (µM)</th><th></th><th>1</th><th style=text-align:center>2</th><th>3</th><th>….</th><th>22</th><th>23</th><th>24</th></tr></thead><tbody><tr><td>100</td><td>A</td><td>5 µM substrate +buffer</td><td style=text-align:center>5 µM substrate +0.1 µM enzyme</td><td>5 µM substrate +0.1 µM enzyme +compound</td><td>5 µM substrate +0.1 µM enzyme +compound</td><td>5 µM substrate +0.1 µM enzyme +compound</td><td>5 µM substrate +buffer</td><td>5 µM substrate +0.1 µM enzyme</td></tr><tr><td>50</td><td>B</td><td>5 µM substrate +buffer</td><td style=text-align:center>5 µM substrate +0.1 µM enzyme</td><td>5 µM substrate +0.1 µM enzyme +compound</td><td>5 µM substrate +0.1 µM enzyme +compound</td><td>5 µM substrate +0.1 µM enzyme +compound</td><td>5 µM substrate +buffer</td><td>5 µM substrate +0.1 µM enzyme</td></tr><tr><td>25</td><td>C</td><td>5 µM substrate +buffer</td><td style=text-align:center>5 µM substrate +0.1 µM enzyme</td><td>5 µM substrate +0.1 µM enzyme +compound</td><td>5 µM substrate +0.1 µM enzyme +compound</td><td>5 µM substrate +0.1 µM enzyme +compound</td><td>5 µM substrate +buffer</td><td>5 µM substrate +0.1 µM enzyme</td></tr><tr><td></td><td>D</td><td>5 µM substrate +buffer</td><td style=text-align:center>5 µM substrate +0.1 µM enzyme</td><td>5 µM substrate +0.1 µM enzyme +compound</td><td>5 µM substrate +0.1 µM enzyme +compound</td><td>5 µM substrate +0.1 µM enzyme +compound</td><td>5 µM substrate +buffer</td><td>5 µM substrate +0.1 µM enzyme</td></tr><tr><td></td><td>E</td><td>5 µM substrate +buffer</td><td style=text-align:center>5 µM substrate +0.1 µM enzyme</td><td>5 µM substrate +0.1 µM enzyme +compound</td><td>5 µM substrate +0.1 µM enzyme +compound</td><td>5 µM substrate +0.1 µM enzyme +compound</td><td>5 µM substrate +buffer</td><td>5 µM substrate +0.1 µM enzyme</td></tr><tr><td></td><td></td><td>= positive control</td><td style=text-align:center>= negative control</td><td>= contains compound</td><td>= contains compound</td><td>= contains compound</td><td>= positive control</td><td>= negative control</td></tr></tbody></table></div></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href><img title=crystallization-protocol width=40 src=../img/icons/crystallization-protocol.png></a></div><div id=flade-NS2B3-protease_crystallization-protocol-1 class=col-11><b>Crystallization protocol of DENV-2 NS2B-NS3 protease</b><br><cite>[ 2022-10-19 ]</cite><br><button data-component=toggleme data-target=#togglebox-protocol-ASAP-DENV-NS2B-NS3-4 data-text="Hide Protocol">Protocol</button><div id=togglebox-protocol-ASAP-DENV-NS2B-NS3-4 class=hide><cite>Plate format: Swiss CI 3 lens<br>Screen: 20% PEG6000, 10% ethylene glycol, 0.01M zinc chloride (Ligand Friendly Screen, Molecular Dimensions)<br>Buffer: 25mM HEPEs, pH 7.5, 150mM NaCl, 0.5mM TCEP, 5% Glycerol<br>Concentration: 49 mg/ml<br>Reservoir volume: 30 µl<br>Protein needed: 40 µl per full plate<br>Drop ratio: 1:1 ratio of protein: reservoir solution<br>75nl of NS2B3: 75nl reservoir solution<br>Final volume: 150 nl<br>Incubation temperature: 20°C in Formulatrix<br>Imaging Schedule: first images after 12hrs and then following a Fibonacci sequence of days for further image collections.<br>Size: Platelets</cite></div></div></div></div></div><hr style=height:5px;border:none;color:#333;background-color:#333><br><br><br><h3 style=text-align:center id=ASAP-WNV-NS2B-NS3>West Nile Virus NS2B-NS3 protease</h3><div class="row align-center"><a href=https://www.ncbi.nlm.nih.gov/protein/AF404756.1><span class="label success">Gene: AF404756.1 </span></a>&nbsp;
<a href=https://www.uniprot.org/uniprotkb/Q9Q6P4><span class="label error">Uniprot: Q9Q6P4 </span></a>&nbsp;
<a href="https://www.ebi.ac.uk/intenz/query?cmd=SearchEC&amp;ec=3.4.21.91"><span class="label warning">EC: 3.4.21.91</span></a></div><br><div class="row align-center"><table class=unstyled><tr><td><img title="Crystallography construct" src=/images/icons/tep-stage-completed_hub021b18cb248db28b117f458e39aeb5b_10162_e22862b00861f0d3434857db1ff02de5.png width=300 height=88><br><img title="Assay construct" src=/images/icons/tep-stage-completed_hub021b18cb248db28b117f458e39aeb5b_10162_2851d035e13cbc115749c33624801624.png width=300 height=88></td><td><img title="Biophysical assay" src=/images/icons/tep-stage-not-started_hu0ad6bc3691333ead921c5ce1ee694314_9444_3996d997df11e427d2dc236cf2e2aca0.png width=300 height=55><br><img title="Robust crystals" src=/images/icons/tep-stage-in-progress_huc9d943f35f5911a48f54b3ad3b40c43a_10037_f71a74ae374dc1fa70d31362f03e7c29.png width=300 height=55><br><img title="Biochemical assay" src=/images/icons/tep-stage-completed_hub021b18cb248db28b117f458e39aeb5b_10162_1b7ab3b95a075266c44f73afb56d1cd3.png width=300 height=55></td><td><img title="X-ray fragment screen and structural analysis" src=/images/icons/tep-stage-not-started_hu0ad6bc3691333ead921c5ce1ee694314_9444_00692520805a7a4b5cb143ae16450ab1.png width=300 height=180></td><td><img title="First-round progression for TEP validation" src=/images/icons/tep-stage-not-started_hu0ad6bc3691333ead921c5ce1ee694314_9444_31ed385b113229ad572b9427160c8c98.png width=300 height=180></td></tr></table></div><b>Contributors:</b>
<span class="label success">Michael Fairhead </span><span class="label success">Noa Lahav </span><span class="label success">Lizbé Koekemoer </span><span class="label success">Andre de Godoy </span><span class="label success">Annette von Delft </span><span class="label success">Blake Balcomb </span><span class="label success">Beth MacLean </span><span class="label success">Charlie Tomlinson </span><span class="label success">Charline Giroud </span><span class="label success">Daren Fearon </span><span class="label success">Dong Wang </span><span class="label success">Eleanor Williams </span><span class="label success">Jasmin Aschenbrenner </span><span class="label success">Jessica Dixon </span><span class="label success">Korvus Wang </span><span class="label success">Lee Harris </span><span class="label success">Lucas Ferreira </span><span class="label success">Oleg Fedorov </span><span class="label success">Peter Marples </span><span class="label success">Ryan Lithgo </span><span class="label success">Tamas Szommer </span><span class="label success">Warren Thompson </span><span class="label success">Xiaomin Ni</span><br><br><div class="prettybox warning" data-component=prettybox><b>Target Nominator:</b> Karla Kirkegaard<br><b>Target PI:</b> Frank von Delft<br><b>Therapeutic Area(s):</b> Infectious diseases<br><b>Disease Relevance:</b> NS3 encodes protease that is important for processing the viral polyprotein<br><b>Viral family:</b> flaviviridae<br><b>Viruses:</b> [West Nile Virus]<br></div><div class="prettybox success" data-component=prettybox><h4>Summary of Project</h4>This TEP is being compiled to provide early tools to develop NS3 protease inhibitors, including purification protocols of recombinant proteins,
and reproducible crystallisation condition suitable for X-ray crystallography fragment screening.</div><div class="prettybox focus" data-component=prettybox><h4>Scientific Background</h4>NS2B/3 performs all cleavages on the cytoplasmic side of the polyprotein.</div><div class="prettybox error" data-component=prettybox><h4>Conclusion</h4>This is a summary of the work we have done on the NS2b-NS3 protease from West Nile viruses to date.</div><h4>Resources</h4><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href><img title=plasmid width=40 src=../img/icons/plasmid.png></a></div><div id=flawn-NS2B3-protease_plasmid-biochemical class=col-11><b>Protein expression construct for WNV NS2B-NS3 protease suitable for biochemical assays (construct QQ01WNVNS2B-c003)</b><br>Construct expressing a non self-cleaving version of the NS2B cofactor fused to the NS3 protease
from West nile virus used for FRET assay and initial crystallization trials.
Generated using golden gate cloning and inverse PCR for construct boundary exploration, based on PDBID: 5GPI [https://www.rcsb.org/structure/5GPI]<br><cite>[ 2022-10-24 ]</cite><br><button data-component=toggleme data-target=#togglebox-protocol-ASAP-WNV-NS2B-NS3-1 data-text="Hide Protocol">Protocol</button><div id=togglebox-protocol-ASAP-WNV-NS2B-NS3-1 class=hide><cite>The construct was transformed into the E. coli strain BL21(DE3)-RR, and cells were grown at 37°C in AIM-TB medium supplemented with kanamycin (50 µg/ml). After reaching an optical density at 600 nm of around 1, the temperature was lowered to 18°C before induction of protein expression overnight. Harvested cells were resuspended in lysis buffer [10 mM Hepes (pH 7.5), 500 mM NaCl, 5% glycerol, 30 mM imidazole, 0.5 mM TCEP, 1% TX-100, 0.5 mg/mL lysozyme, 0.05 mg/mL benzonase]. Proteins were first purified by immobilized metal affinity chromatography (IMAC) using Ni-Sepharose 6 FF resin (Cytiva) the column was washed with binding buffer [10 mM Hepes (pH 7.5), 500 mM NaCl, 5% glycerol, 30 mM imidazole, 0.5 mM TCEP] and target protein eluted using same buffer containing 500 mM imidazole. The his twin strep tag was removed by digestion with TEV protease and the mixture dialysed overnight against [10 mM Hepes (pH 7.5), 500 mM NaCl, 5% glycerol, 30 mM imidazole, 0.5 mM TCEP]. Undigested protein, his twin strep tagged protein and his tagged TEV was removed by using Ni-Sepharose 6 FF resin (Cytiva) and the unbound fraction containing untagged protein collected. Protein was lastly purified by SEC (Superose 12 pg, Cytiva) in a buffer consisting of 10 mM Hepes (pH 7.5), 500 mM NaCl, 5% glycerol and 0.5 mM TCEP. Proteins were characterised by SDS-polyacrylamide gel electrophoresis and mass spec and then flash- frozen in liquid nitrogen, and stored at -80°C until required.</cite></div><button data-component=toggleme data-target=#togglebox-details-ASAP-WNV-NS2B-NS3-1 data-text="Hide Details">Details</button><div id=togglebox-details-ASAP-WNV-NS2B-NS3-1 class=hide><ul><li>Vector: pNIC-NHStIIT</li><li>Entry clone accession:</li><li>Cell line: E. coli Rosetta 2 strain BL21(DE3)</li><li>Tags and additions: N-terminal, TEV protease cleavable hexahistidine</li><li>Construct protein sequence:
SMSTDMWIERTADISWESDAEITGSSERVDVRLDDDGNFQLMNDPGAPWAGGGGSGGGGGVLWDTPSPKEYKKGDTTTGVYRIMTRGLLGSYQAGAGVMVEGVFHTLWHTTKGAALMSGEGRLDPYWGSVKEDRLCYGGPWKLQHKWNGQDEVQMIVVEPGKNVKNVQTKPGVFKTPEGEIGAVTLDFPTGTSGSPIVDKNGDVIGLYGNGVIMPNGSYISAIVQGERMDEPIPAGFEPEMLRKK</li></ul></div></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href><img title=plasmid width=40 src=../img/icons/plasmid.png></a></div><div id=flawn-NS2B3-protease_plasmid-inactive class=col-11><b>Protein expression construct for inactive WNV NS2B-NS3 protease (construct QQ01WNVNS2B-c002)</b><br>Construct expressing the NS2B cofactor fused to the NS3 protease from West nile virus used as negative control
for FRET assay and in initial crystallography studies. Generated using golden gate cloning and inverse PCR
for construct boundary exploration, based on PDBID: 5GPI [https://www.rcsb.org/structure/5GPI]<br><cite>[ 2022-10-24 ]</cite><br><button data-component=toggleme data-target=#togglebox-protocol-ASAP-WNV-NS2B-NS3-2 data-text="Hide Protocol">Protocol</button><div id=togglebox-protocol-ASAP-WNV-NS2B-NS3-2 class=hide><cite>The construct was transformed into the E. coli strain BL21(DE3)-RR, and cells were grown at 37°C in AIM-TB medium supplemented with kanamycin (50 µg/ml). After reaching an optical density at 600 nm of around 1, the temperature was lowered to 18°C before induction of protein expression overnight. Harvested cells were resuspended in lysis buffer [10 mM Hepes (pH 7.5), 500 mM NaCl, 5% glycerol, 30 mM imidazole, 0.5 mM TCEP, 1% TX-100, 0.5 mg/mL lysozyme, 0.05 mg/mL benzonase]. Proteins were first purified by immobilized metal affinity chromatography (IMAC) using Ni-Sepharose 6 FF resin (Cytiva) the column was washed with binding buffer [10 mM Hepes (pH 7.5), 500 mM NaCl, 5% glycerol, 30 mM imidazole, 0.5 mM TCEP] and target protein eluted using same buffer containing 500 mM imidazole. The his twin strep tag was removed by digestion with TEV protease and the mixture dialysed overnight against [10 mM Hepes (pH 7.5), 500 mM NaCl, 5% glycerol, 30 mM imidazole, 0.5 mM TCEP]. Undigested protein, his twin strep tagged protein and his tagged TEV was removed by using Ni-Sepharose 6 FF resin (Cytiva) and the unbound fraction containing untagged protein collected. Protein was lastly purified by SEC (Superose 12 pg, Cytiva) in a buffer consisting of 10 mM Hepes (pH 7.5), 500 mM NaCl, 5% glycerol and 0.5 mM TCEP. Proteins were characterised by SDS-polyacrylamide gel electrophoresis and mass spec and then flash- frozen in liquid nitrogen, and stored at -80°C until required.</cite></div><button data-component=toggleme data-target=#togglebox-details-ASAP-WNV-NS2B-NS3-2 data-text="Hide Details">Details</button><div id=togglebox-details-ASAP-WNV-NS2B-NS3-2 class=hide><ul><li>Vector: pNIC-NHStIIT</li><li>Entry clone accession:</li><li>Cell line: <em>E. coli</em> Rosetta 2 strain BL21(DE3)</li><li>Tags and additions: N-terminal, TEV protease cleavable hexahistidine</li><li>Construct protein sequence:
SMSTDMWIERTADISWESDAEITGSSERVDVRLDDDGNFQLMNDPGAPWKGGGGSGGGGGVLWDTPSPKEYKKGDTTTGVYRIMTRGLLGSYQAGAGVMVEGVFHTLWATTKGAALMSGEGRLDPYWGSVKEDRLCYGGPWKLQHKWNGQDEVQMIVVEPGKNVKNVQTKPGVFKTPEGEIGAVTLDFPTGTSGSPIVDKNGDVIGLYGNGVIMPNGSYISAIVQGERMDEPIPAGFEPEMLRKK</li></ul></div></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href><img title=xray-fragment-screen width=40 src=../img/icons/xray-fragment-screen.png></a></div><div id=flawn-NS2B3-protease_crystallization-protocol-1 class=col-11><b>Experimental crystallization protocol of inactive WNV NS2B-NS3 protease (construct QQ01WNVNS2B-c002) - not suitable for fragment screening</b><br>The NS2B-NS3 fusion protein with the catalytically important His mutated to Ala crystallized in various coarse screen conditions.
We obtained crystal which diffracted to 1.6 Å, but it is unsuitable for fragment soaking since the active site is occluded.
PDB to be released: PDBID: 8CO8 [https://www.rcsb.org/structure/unreleased/8CO8]<br><cite>[ 2022-11-26 ]</cite><br><button data-component=toggleme data-target=#togglebox-protocol-ASAP-WNV-NS2B-NS3-3 data-text="Hide Protocol">Protocol</button><div id=togglebox-protocol-ASAP-WNV-NS2B-NS3-3 class=hide><cite>Plate format: Swiss CI 3 lens<br>Screen: 0.12M Monosaccharides&ndash;0.1M Buffer System 3 pH8.5&ndash;50% v/v Precipitant Mix 4 (Morpheus)<br>Buffer: 25mM HEPES, pH 7.5, 150mM NaCl, 0.5mM TCEP, 5% Glycerol<br>Concentration: 94 mg/ml<br>Reservoir volume: 30 µl<br>Protein needed: 40 µl per full plate<br>Drop ratio: 1:2 ratio of protein: reservoir solution<br>50nl of NS2B3: 100nl reservoir solution<br>Final volume: 150 nl<br>Incubation temperature: 20°C in Formulatrix<br>Imaging Schedule: first images after 12hrs and then following a Fibonacci sequence of days for further image collections.<br>Crystal typically form within 12hrs, within 24hrs they have reached their maximum size with slight precipitant. Crystals form on their own and have cubic appearance.<br>Size: ~75-100um in length, width and depth</cite></div></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href=https://www.addgene.org/204797/><img title=plasmid width=40 src=../img/icons/plasmid.png></a></div><div id=flawn-NS2B3-protease_plasmid-biochemical class=col-11><b>Additional constructs cloned and availible on addgene</b><br>Construct expressing literature construct PDBID: 2GGV
url: <a href=https://www.rcsb.org/structure/2GGV>https://www.rcsb.org/structure/2GGV</a><br><a href=https://www.addgene.org/204797/>https://www.addgene.org/204797/</a>
<cite>[ 2022-10-24 ]</cite><br></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href=https://www.addgene.org/204799/><img title=plasmid width=40 src=../img/icons/plasmid.png></a></div><div id=flawn-NS2B3-protease_plasmid-biochemical class=col-11><b>Additional constructs cloned and availible on addgene &ndash; continued</b><br>Construct expressing literature construct PDBID: 2IJO
url: <a href=https://www.rcsb.org/structure/2IJO>https://www.rcsb.org/structure/2IJO</a><br><a href=https://www.addgene.org/204799/>https://www.addgene.org/204799/</a>
<cite>[ 2022-10-24 ]</cite><br></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href=https://www.rcsb.org/structure/8CO8><img title=pdb-deposition width=40 src=../img/icons/pdb-deposition.png></a></div><div id=flawn-NS2B3-protease_apo-structure class=col-11><b>Structure of West Nile Virus NS2B-NS3 protease</b><br>Structure of apo WNV NS2B-NS3 protease<br><a href=https://www.rcsb.org/structure/8CO8>https://www.rcsb.org/structure/8CO8</a>
<cite>[ 2023-09-06 ]</cite><br></div></div></div></div><hr style=height:5px;border:none;color:#333;background-color:#333><br><br><br><h3 style=text-align:center id=ASAP-ZIKA-NS2B-NS3>Zika NS2B-NS3 protease</h3><div class="row align-center"><a href="https://www.ncbi.nlm.nih.gov/protein/%25!s%28int=7751225%29"><span class="label success">Gene: 7751225 </span></a>&nbsp;
<a href="https://www.uniprot.org/uniprotkb/%25!s%28int=64320%29"><span class="label error">Uniprot: 64320 </span></a>&nbsp;
<a href="https://www.ebi.ac.uk/intenz/query?cmd=SearchEC&amp;ec=3.4.21.91"><span class="label warning">EC: 3.4.21.91</span></a></div><br><div class="row align-center"><table class=unstyled><tr><td><img title="Crystallography construct" src=/images/icons/tep-stage-completed_hub021b18cb248db28b117f458e39aeb5b_10162_e22862b00861f0d3434857db1ff02de5.png width=300 height=88><br><img title="Assay construct" src=/images/icons/tep-stage-completed_hub021b18cb248db28b117f458e39aeb5b_10162_2851d035e13cbc115749c33624801624.png width=300 height=88></td><td><img title="Biophysical assay" src=/images/icons/tep-stage-in-progress_huc9d943f35f5911a48f54b3ad3b40c43a_10037_80222a2cb4b3b4d0cafa06d6608261b8.png width=300 height=55><br><img title="Robust crystals" src=/images/icons/tep-stage-completed_hub021b18cb248db28b117f458e39aeb5b_10162_e3383ab8d75acf7a535bdc511b331b23.png width=300 height=55><br><img title="Biochemical assay" src=/images/icons/tep-stage-completed_hub021b18cb248db28b117f458e39aeb5b_10162_1b7ab3b95a075266c44f73afb56d1cd3.png width=300 height=55></td><td><img title="X-ray fragment screen and structural analysis" src=/images/icons/tep-stage-completed_hub021b18cb248db28b117f458e39aeb5b_10162_ae1969e18479b9f6892265d5fd14a303.png width=300 height=180></td><td><img title="First-round progression for TEP validation" src=/images/icons/tep-stage-in-progress_huc9d943f35f5911a48f54b3ad3b40c43a_10037_0bf41094ba248abeddae737c4c98b352.png width=300 height=180></td></tr></table></div><b>Contributors:</b>
<span class="label success">Xiaomin Ni </span><span class="label success">Blake Balcomb </span><span class="label success">Andre de Godoy </span><span class="label success">Annette von Delft </span><span class="label success">Beth MacLean </span><span class="label success">Charlie Tomlinson </span><span class="label success">Charline Giroud </span><span class="label success">Daren Fearon </span><span class="label success">Dong Wang </span><span class="label success">Eleanor Williams </span><span class="label success">Jasmin Aschenbrenner </span><span class="label success">Jessica Dixon </span><span class="label success">Korvus Wang </span><span class="label success">Lee Harris </span><span class="label success">Lizbé Koekemoer </span><span class="label success">Lucas Ferreira </span><span class="label success">Michael Fairhead </span><span class="label success">Oleg Fedorov </span><span class="label success">Peter Marples </span><span class="label success">Ryan Lithgo </span><span class="label success">Tamas Szommer </span><span class="label success">Warren Thompson</span><br><br><div class="prettybox warning" data-component=prettybox><b>Target Nominator:</b> Karla Kirkegaard<br><b>Target PI:</b> Frank von Delft<br><b>Therapeutic Area(s):</b> Infectious diseases<br><b>Disease Relevance:</b> NS3 encodes protease that is important for processing the viral polyprotein<br><b>Viral family:</b> flaviviridae<br><b>Viruses:</b> [Zika]<br></div><div class="prettybox success" data-component=prettybox><h4>Summary of Project</h4>The NS3 protease from Zika virus is expressed with the NS2B cofactor needed for activity. This TEP provides early tools to develop NS3 protease inhibitors, including purification protocols of recombinant proteins, and reproducible crystallisation condition suitable for X-ray crystallography fragment screening.</div><div class="prettybox focus" data-component=prettybox><h4>Scientific Background</h4>The Zika virus (ZIKV) has become a global public health threat since it was related to major neurological disorders in both infants and adults [<a href=https://doi.org/10.1016/S0140-6736(17)31450-2>1</a>, <a href=https://doi.org/10.1016/S0140-6736(16)00562-6>2</a>]. TO WRITE</div><div class="prettybox error" data-component=prettybox><h4>Conclusion</h4>Here we have established a purification protocol for recombinant NS2b-NS3 protease from Zika viruses and established a robust crystallisation suitable for fragment screening. We have performed a substantial fragment screening campaign and identified many hits.</div><h4>Resources</h4><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href=N/A><img title=plasmid width=40 src=../img/icons/plasmid.png></a></div><div id=flazi-NS2B3-protease_plasmid-crystallography class=col-11><b>Protein expression construct of ZIKV NS2B-NS3 protease suitable for crystallography</b><br>Bicistronic construct expressing the NS2B cofactor and NS3 protease from Zika virus (64320) used for protein crystallography. Generated using golden gate cloning and inverse PCR for construct boundary exploration, based on PDB 5GPI<br>N/A
<cite>[ 2023-02-07 ]</cite><br><button data-component=toggleme data-target=#togglebox-protocol-ASAP-ZIKA-NS2B-NS3-1 data-text="Hide Protocol">Protocol</button><div id=togglebox-protocol-ASAP-ZIKA-NS2B-NS3-1 class=hide><cite><strong>Transformation</strong> • thaw component cells BL21 (DE3) on ice • add 1 µl of plasmid (100-200 ng/µl ) into 50 µl component cells • incubate on ice for 30 min • 42°C heat shock for 40s, put back on ice for another 5 mins • add 500 µl room temperature SOC medium • incubate at 37°C for 1h no shaking • plate on 90 mm LB-agar kanamycin plate, incubate at 37°C overnight
<strong>Expression</strong> • Use 5-10 colonies to inoculate 100 ml of SOC kanamycin medium in a 250 ml baffled flask and grow overnight 37°C • Use 15 mL to inoculate 1 L of TB medium in a 2.5 L UltraYield Flask • Shake 37°C 180 rpm for 3-4 h (OD600 of 1-2) • Lower temperature of incubator to 18°C and after 1 h induce protein expression with 0.5 mM IPTG and continue shaking overnight • Harvest cells by centrifugation at 4000 g, 20 minutes, 4°C • Freeze cell paste at -80°C
<strong>Purification</strong> (100g cell pellet) Day 1 • Dissolve cell pellet in 500ml Base buffer + 0.5mg/ml Lysozyme + 0.1mg/ml Benzonase + 1 % TX-100 + 30 mM imidazole (Base Buffer:10 mM HEPES, 500 mM NaCl, 5 % glycerol, 0.5 mM TCEP, pH 7.5) • Incubate on ice 1 h • Centrifuge 30,000 g for 1 h, 4°C • Transfer SN to 15 ml Ni-sepharose ff (column pre-equilibrated in Base buffer + 30 mM imidazole) • Wash with 3 x 100ml Base buffer + 30 mM imidazole • Elute protein with Base Buffer + 500 mM imidazole 6 x 10 ml • Pool peak fractions and add TEV protease (1 OD unit TEV for every 10 OD units of target) and dialyze overnight 4°C against 3 L base buffer + 30 mM imidazole
Day2 • Pass dialyzed sample over 15 ml Ni-sepharose ff (column pre-equilibrated in Base buffer + 30 mM imidazole) • Wash with 3 x 10ml Base buffer + 30 mM imidazole • Pool samples and concentrate to around 20 mg/mL run over SEPAX SEC 3000 using mobile phase of 25mM HEPEs pH7.5, 150mM NaCl, 0.5mM TCEP, 5% Glycerol.</cite></div><button data-component=toggleme data-target=#togglebox-details-ASAP-ZIKA-NS2B-NS3-1 data-text="Hide Details">Details</button><div id=togglebox-details-ASAP-ZIKA-NS2B-NS3-1 class=hide><ul><li>Vector: pNIC-6HIS-GST-TEV-GG</li><li>Entry clone accession:</li><li>Cell line: E. coli Rosetta 2 strain BL21(DE3)</li><li>Tags and additions: N-terminal, TEV protease cleavable GST hexahistidine</li><li>Construct protein sequence:
NS2B: SMGKSVDMYIERAGDITWEKDAEVTGNSPRLDVALDESGDFSLVEE
NS3: MKEVKKGETTDGVYRVMTRRLLGSTQVGVGVMQEGVFHTMWHVTKGAALRSGEGRLDPYWGDVKQDLVSYCGPWKLDAAWDGLSEVQLLAVPPGERAKNIQTLPGIFKTKDGDIGAVALDYPAGTSGSPILDKCGRVIGLYGNGVVIKNGSYVSAITQGKREEETPVE</li></ul></div></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href><img title=plasmid width=40 src=../img/icons/plasmid.png></a></div><div id=flazi-NS2B3-protease_plasmid-biochemical class=col-11><b>Protein expression construct of ZIKV NS2B-NS3 suitable for biochemical assay (ZV-NS2BgsgNS3)</b><br>Construct expressing the NS2B cofactor fused to the NS3 protease from Zika virus (64320) used for FRET assay. Generated using golden gate cloning and inverse PCR for construct boundary exploration, based on PDB 5GPI<br><cite>[ 2022-10-24 ]</cite><br><button data-component=toggleme data-target=#togglebox-protocol-ASAP-ZIKA-NS2B-NS3-2 data-text="Hide Protocol">Protocol</button><div id=togglebox-protocol-ASAP-ZIKA-NS2B-NS3-2 class=hide><cite>The construct was transformed into the E. coli strain BL21(DE3)-RR, and cells were grown at 37°C in AIM-TB medium supplemented with kanamycin (50 µg/ml). After reaching an optical density at 600 nm of around 1, the temperature was lowered to 18°C before induction of protein expression overnight. Harvested cells were resuspended in lysis buffer [10 mM Hepes (pH 7.5), 500 mM NaCl, 5% glycerol, 30 mM imidazole, 0.5 mM TCEP, 1% TX-100, 0.5 mg/mL lysozyme, 0.05 mg/mL benzonase]. Proteins were first purified by immobilized metal affinity chromatography (IMAC) using Ni-Sepharose 6 FF resin (Cytiva) the column was washed with binding buffer [10 mM Hepes (pH 7.5), 500 mM NaCl, 5% glycerol, 30 mM imidazole, 0.5 mM TCEP] and target protein eluted using same buffer containing 500 mM imidazole. The his twin strep tag was removed by digestion with TEV protease and the mixture dialysed overnight against [10 mM Hepes (pH 7.5), 500 mM NaCl, 5% glycerol, 30 mM imidazole, 0.5 mM TCEP]. Undigested protein, his twin strep tagged protein and his tagged TEV was removed by using Ni-Sepharose 6 FF resin (Cytiva) and the unbound fraction containing untagged protein collected. Protein was lastly purified by SEC (Superose 12 pg, Cytiva) in a buffer consisting of 10 mM Hepes (pH 7.5), 500 mM NaCl, 5% glycerol and 0.5 mM TCEP. Proteins were characterised by SDS-polyacrylamide gel electrophoresis and mass spec and then flash- frozen in liquid nitrogen, and stored at -80°C until required.</cite></div><button data-component=toggleme data-target=#togglebox-details-ASAP-ZIKA-NS2B-NS3-2 data-text="Hide Details">Details</button><div id=togglebox-details-ASAP-ZIKA-NS2B-NS3-2 class=hide><p><a href="/pdfs/PAGE22-01756 - ASAP; Dengue, Zika and West Nile NS2B3; First round crystallization.pdf" download>Download experimental metadata</a></p><ul><li>Vector: pNIC-NHStIIT</li><li>Entry clone accession:</li><li>Cell line: <em>E. coli</em> Rosetta 2 strain BL21(DE3)</li><li>Tags and additions: N-terminal, TEV protease cleavable hexahistidine</li><li>Construct protein sequence:
NS2B3: SMSGKSVDMYIERAGDITWEKDAEVTGNSPRLDVALDESGDFSLVEDDGPPMREGGGGSGGGGGSGALWDVPAPKEVKKGETTDGVYRVMTRRLLGSTQVGVGVMQEGVFHTMWHVTKGSALRSGEGRLDPYWGDVKQDLVSYCGPWKLDAAWDGHSEVQLLAVPPGERARNIQTLPGIFKTKDGDIGAVALDYPAGTSGSPILDKCGRVIGLYGNGVVIKNGSYVSAITQGRREEETPVE</li></ul></div></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href><img title=plasmid width=40 src=../img/icons/plasmid.png></a></div><div id=flazi-NS2B3-protease_assay-biochemical class=col-11><b>ZIKV NS2B-NS3 protease activity biochemical assay: Inactive assay construct with His to Ala mutation</b><br><cite>[ 2022-10-24 ]</cite><br><button data-component=toggleme data-target=#togglebox-protocol-ASAP-ZIKA-NS2B-NS3-3 data-text="Hide Protocol">Protocol</button><div id=togglebox-protocol-ASAP-ZIKA-NS2B-NS3-3 class=hide><cite>The construct was transformed into the E. coli strain BL21(DE3)-RR, and cells were grown at 37°C in AIM-TB medium supplemented with kanamycin (50 µg/ml). After reaching an optical density at 600 nm of around 1, the temperature was lowered to 18°C before induction of protein expression overnight. Harvested cells were resuspended in lysis buffer [10 mM Hepes (pH 7.5), 500 mM NaCl, 5% glycerol, 30 mM imidazole, 0.5 mM TCEP, 1% TX-100, 0.5 mg/mL lysozyme, 0.05 mg/mL benzonase]. Proteins were first purified by immobilized metal affinity chromatography (IMAC) using Ni-Sepharose 6 FF resin (Cytiva) the column was washed with binding buffer [10 mM Hepes (pH 7.5), 500 mM NaCl, 5% glycerol, 30 mM imidazole, 0.5 mM TCEP] and target protein eluted using same buffer containing 500 mM imidazole. The his twin strep tag was removed by digestion with TEV protease and the mixture dialysed overnight against [10 mM Hepes (pH 7.5), 500 mM NaCl, 5% glycerol, 30 mM imidazole, 0.5 mM TCEP]. Undigested protein, his twin strep tagged protein and his tagged TEV was removed by using Ni-Sepharose 6 FF resin (Cytiva) and the unbound fraction containing untagged protein collected. Protein was lastly purified by SEC (Superose 12 pg, Cytiva) in a buffer consisting of 10 mM Hepes (pH 7.5), 500 mM NaCl, 5% glycerol and 0.5 mM TCEP. Proteins were characterised by SDS-polyacrylamide gel electrophoresis and mass spec and then flash- frozen in liquid nitrogen, and stored at -80°C until required.</cite></div><button data-component=toggleme data-target=#togglebox-details-ASAP-ZIKA-NS2B-NS3-3 data-text="Hide Details">Details</button><div id=togglebox-details-ASAP-ZIKA-NS2B-NS3-3 class=hide><ul><li>Vector: pNIC-NHStIIT</li><li>Entry clone accession:</li><li>Cell line: <em>E. coli</em> Rosetta 2 strain BL21(DE3)</li><li>Tags and additions: N-terminal, TEV protease cleavable hexahistidine</li><li>Construct protein sequence:
NS2B3: MSGKSVDMYIERAGDITWEKDAEVTGNSPRLDVALDESGDFSLVEDDGPPMREGGGGSGGGGGSGALWDVPAPKEVKKGETTDGVYRVMTRRLLGSTQVGVGVMQEGVFHTMWAVTKGSALRSGEGRLDPYWGDVKQDLVSYCGPWKLDAAWDGHSEVQLLAVPPGERARNIQTLPGIFKTKDGDIGAVALDYPAGTSGSPILDKCGRVIGLYGNGVVIKNGSYVSAITQGRREEETPVE</li></ul><p>Enzyme: ZV-NS2BgsgNS3</p><p>Substrate:</p><ul><li>Boc-Gly-Arg-Arg-AMC</li><li>CAS [113866-14-1(free base)]</li><li>Biosynth (FB110553)</li><li>0.9 mg dissolved in 140 µL of DMSO and aliquoted to 50 ul &ndash;> stored at -80C as 10mM</li></ul><p>Buffer: Enzymes storage buffer 10 mM HEPES pH 7.3, 500 mM NaCl, 5% glycerol, 0.5 mM TCEP.</p><p>Assay Buffer (final conc. In plate):</p><ul><li>20 mM Tris, pH 8.5</li><li>10 % glycerol</li><li>0.01 % Triton</li></ul><p>Glycerol and Triton are added to the assay buffer before each run.</p><p>Design:</p><ul><li>10 µL of 10 µM substrate + 10 µL of 0.2 µM enzyme in assay buffer</li><li>(final assay volume: 20 µl).</li><li>Inhibitor compounds were pre-dispensed (from DMSO stock) in duplicates to 384 small volume, black plate (Greiner-784076) and stored in -20 until use (control wells - DMSO only, no compounds).</li><li>Dispensing performed using the GNF dispenser</li></ul><p>Sequence: SMSGKSVDMYIERAGDITWEKDAEVTGNSPRLDVALDESGDFSLVEDDGPPMREGGGGSG
GGGGSGALWDVPAPKEVKKGETTDGVYRVMTRRLLGSTQVGVGVMQEGVFHTMWHVTKGS
ALRSGEGRLDPYWGDVKQDLVSYCGPWKLDAAWDGHSEVQLLAVPPGERARNIQTLPGIF
KTKDGDIGAVALDYPAGTSGSPILDKCGRVIGLYGNGVVIKNGSYVSAITQGRREEETPV
E</p><p>MW: 25556.55, Conc (uM): 225, Extinction Coefficient: 43555</p><p>Example Plate Layout:</p><table><thead><tr><th>Compounds conc. (µM)</th><th></th><th>1</th><th style=text-align:center>2</th><th>3</th><th>….</th><th>22</th><th>23</th><th>24</th></tr></thead><tbody><tr><td>100</td><td>A</td><td>5 µM substrate +buffer</td><td style=text-align:center>5 µM substrate +0.1 µM enzyme</td><td>5 µM substrate +0.1 µM enzyme +compound</td><td>5 µM substrate +0.1 µM enzyme +compound</td><td>5 µM substrate +0.1 µM enzyme +compound</td><td>5 µM substrate +buffer</td><td>5 µM substrate +0.1 µM enzyme</td></tr><tr><td>50</td><td>B</td><td>5 µM substrate +buffer</td><td style=text-align:center>5 µM substrate +0.1 µM enzyme</td><td>5 µM substrate +0.1 µM enzyme +compound</td><td>5 µM substrate +0.1 µM enzyme +compound</td><td>5 µM substrate +0.1 µM enzyme +compound</td><td>5 µM substrate +buffer</td><td>5 µM substrate +0.1 µM enzyme</td></tr><tr><td>25</td><td>C</td><td>5 µM substrate +buffer</td><td style=text-align:center>5 µM substrate +0.1 µM enzyme</td><td>5 µM substrate +0.1 µM enzyme +compound</td><td>5 µM substrate +0.1 µM enzyme +compound</td><td>5 µM substrate +0.1 µM enzyme +compound</td><td>5 µM substrate +buffer</td><td>5 µM substrate +0.1 µM enzyme</td></tr><tr><td></td><td>D</td><td>5 µM substrate +buffer</td><td style=text-align:center>5 µM substrate +0.1 µM enzyme</td><td>5 µM substrate +0.1 µM enzyme +compound</td><td>5 µM substrate +0.1 µM enzyme +compound</td><td>5 µM substrate +0.1 µM enzyme +compound</td><td>5 µM substrate +buffer</td><td>5 µM substrate +0.1 µM enzyme</td></tr><tr><td></td><td>E</td><td>5 µM substrate +buffer</td><td style=text-align:center>5 µM substrate +0.1 µM enzyme</td><td>5 µM substrate +0.1 µM enzyme +compound</td><td>5 µM substrate +0.1 µM enzyme +compound</td><td>5 µM substrate +0.1 µM enzyme +compound</td><td>5 µM substrate +buffer</td><td>5 µM substrate +0.1 µM enzyme</td></tr><tr><td></td><td></td><td>= positive control</td><td style=text-align:center>= negative control</td><td>= contains compound</td><td>= contains compound</td><td>= contains compound</td><td>= positive control</td><td>= negative control</td></tr></tbody></table></div></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href><img title=crystallization-protocol width=40 src=../img/icons/crystallization-protocol.png></a></div><div id=flazi-NS2B3-protease_crystallization-protocol-1 class=col-11><b>Experimental crystallization protocol for ZIKV NS2B-NS3 - unsuitable for fragment soaking</b><br>Initial crystallography with NS2B3 fusion protein expressed from QQ01ZVNS2B-c002 diffracted to 4.36 Å and not suitable for fragment soaking.<br><cite>[ 2022-11-26 ]</cite><br><button data-component=toggleme data-target=#togglebox-protocol-ASAP-ZIKA-NS2B-NS3-4 data-text="Hide Protocol">Protocol</button><div id=togglebox-protocol-ASAP-ZIKA-NS2B-NS3-4 class=hide><cite>ZIKA NS2B3 was concentrated to a final concentration of 33 mg/ml and apo crystals were grown in crystallisation solution containing 30% PEG 1000, 0.1 M SPG, pH 47.0 (Molecular Dimensions) at 20^o^C. One crystal was obtained after 28 days. Data were collected at the I04-1 beamline at 100 K and automatically processed with Diamond Light Source&rsquo;s autoprocessing pipelines using XDS (9) and <a href=https://doi.org/10.1107/s0907444913015308>xia2 3dii</a>.</cite></div><button data-component=toggleme data-target=#togglebox-details-ASAP-ZIKA-NS2B-NS3-4 data-text="Hide Details">Details</button><div id=togglebox-details-ASAP-ZIKA-NS2B-NS3-4 class=hide><a href="/pdfs/PAGE22-01614 - AVIDD ASAP; NS2B-GSG-NS3 fusions for assays.pdf" download>Download experimental metadata</a></div></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href><img title=crystallization-protocol width=40 src=../img/icons/crystallization-protocol.png></a></div><div id=flazi-NS2B3-protease_crystallization-protocol-soaking class=col-11><b>Crystallization protocol for ZIKV NS2B-NS3 protease suitable for fragment soaking</b><br>Crystallography with NS2B and NS3 coexpressed protein from XX01ZVNS2B-c002 diffracted to 1.6 Å and suitable for fragment soaking. PDB code: 8PN6<br><cite>[ 2023-01-31 ]</cite><br><button data-component=toggleme data-target=#togglebox-protocol-ASAP-ZIKA-NS2B-NS3-5 data-text="Hide Protocol">Protocol</button><div id=togglebox-protocol-ASAP-ZIKA-NS2B-NS3-5 class=hide><cite>Plate format: Swiss CI 3 lens Screen: 30% w/v PEG 2000, 0.2M Ammonium sulfate, 0.1M acetate (pH 4.8) Buffer: 25mM HEPES, pH 7.5, 150mM NaCl, 0.5mM TCEP, 5% Glycerol Concentration: 15 mg/ml Reservoir volume: 30 µl Protein needed: 40 µl per full plate Drop ratio: 1:1 ratio of protein: reservoir solution
150nl of NS2B3: 150nl reservoir solution
Final volume: 300 nl Incubation temperature: 20°C in Formulatrix Imaging Schedule: first images after 12hrs and then following a Fibonacci sequence of days for further image collections. Crystal typically form within 12hrs, within 24hrs they have reached their maximum size with slight precipitant. Crystals form on their own and have cubic appearance. Size: ~75-100um in length, width and depth</cite></div></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href=https://www.rcsb.org/structure/8PN6><img title=pdb-deposition width=40 src=../img/icons/pdb-deposition.png></a></div><div id=flazi-NS2B3-protease_apo-structure class=col-11><b>Apo crystal structure of the ZIKV NS2B-NS3 protease</b><br><a href=https://www.rcsb.org/structure/8PN6>https://www.rcsb.org/structure/8PN6</a>
<cite>[ 2023-08-16 ]</cite><br></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href=https://fragalysis.diamond.ac.uk/viewer/react/preview/target/XX01ZVNS2B/tas/lb32627-65><img title=xray-fragment-screen width=40 src=../img/icons/xray-fragment-screen.png></a></div><div id=flazi-NS2B3-protease_fragment-screen class=col-11><b>Fragalysis interactive view of ZIKV NS2B-NS3 protease crystallographic fragment screen</b><br>We performed X-ray crystallography screening at XChem/Diamond Light Source and collected experimental datasets.<br><a href=https://fragalysis.diamond.ac.uk/viewer/react/preview/target/XX01ZVNS2B/tas/lb32627-65>https://fragalysis.diamond.ac.uk/viewer/react/preview/target/XX01ZVNS2B/tas/lb32627-65</a>
<cite>[ 2024-03-20 ]</cite><br><button data-component=toggleme data-target=#togglebox-protocol-ASAP-ZIKA-NS2B-NS3-7 data-text="Hide Protocol">Protocol</button><div id=togglebox-protocol-ASAP-ZIKA-NS2B-NS3-7 class=hide><cite>ZIKV NS2B3 was concentrated to a final concentration of 15 mg/ml and apo crystals were grown in crystallisation solution containing 30% PEG 2000, 0.2 M Ammonium sulphate and 0.1 M acetate buffer, pH 4.8 (Molecular Dimensions). Fragments (libraries: DSIPoised, Probing all fragments, SpotXplorer, CovHetLib) were soaked into crystals as previously described by adding dissolved compounds directly to the crystallisation drops using an Echo liquid handler (final concentration, XX% DMSO); drops were incubated for approximately XX hours before mounting and flash-freezing in liquid nitrogen. Data were collected at the I04-1 beamline at 100 K and automatically processed with Diamond Light Source&rsquo;s autoprocessing pipelines using XDS (9) and either <a href=https://doi.org/10.1107/s0907444913015308>xia2</a> or <a href=https://doi.org/10.1107/s2059798317017235>DIALS</a>. Further analysis was performed with <a href=https://doi.org/10.1107%2FS2059798316020234>XChemExplorer</a>, electron density maps were generated with DIMPLE and ligand-binding events were identified using <a href=https://doi.org/10.1038/ncomms15123>PanDDA</a>. Ligands were modeled into PanDDA-calculated event maps using Coot, restraints were calculated with <a href=http://dx.doi.org/10.1107/S2059798317000067>AceDRG</a>, and structures were refined with <a href=https://doi.org/10.1107/s0907444992010400>BUSTER</a>.</cite></div><button data-component=toggleme data-target=#togglebox-details-ASAP-ZIKA-NS2B-NS3-7 data-text="Hide Details">Details</button><div id=togglebox-details-ASAP-ZIKA-NS2B-NS3-7 class=hide><a href="/pdfs/PAGE23-00345 - AVIDD ASAP; MVMPROA, ZVNS2B & A71EV2A xtal mounting,screening.pdf" download>Download experimental metadata</a></div></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href=https://www.rcsb.org/groups/summary/entry/G_1002289><img title=pdb-group-deposition width=40 src=../img/icons/pdb-group-deposition.png></a></div><div id=flazi-NS2B3-protease_fragment-screen-pdb-group-deposition class=col-11><b>PDB group deposition of ZIKV NS2B-NS3 protease crystallographic fragment screen</b><br>PDB group deposition of X-ray fragment screen<br><a href=https://www.rcsb.org/groups/summary/entry/G_1002289>https://www.rcsb.org/groups/summary/entry/G_1002289</a>
<cite>[ 2024-04-03 ]</cite><br></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href=https://doi.org/10.1101/2024.04.29.591502><img title=preprint width=40 src=../img/icons/preprint.png></a></div><div id=flazi-NS2B3-protease_fragment-screen-preprint class=col-11><b>Crystallographic fragment screening delivers diverse chemical scaffolds for Zika virus NS2B-NS3 protease inhibitor development</b><br>Preprint describing X-ray fragment screen of ZIKV NS2B-NS3 protease, resolving 48 protein:ligand complexes<br><a href=https://doi.org/10.1101/2024.04.29.591502>https://doi.org/10.1101/2024.04.29.591502</a>
<cite>[ 2024-04-29 ]</cite><br></div></div></div></div><hr style=height:5px;border:none;color:#333;background-color:#333><br><br><br><h3 style=text-align:center id=ASAP-ZIKA-NS3-HELICASE>Zika NS3 helicase</h3><div class="row align-center"><a href=https://www.ncbi.nlm.nih.gov/protein/KF383119.1><span class="label success">Gene: KF383119.1 </span></a>&nbsp;
<a href=https://www.uniprot.org/uniprotkb/Q32ZE1><span class="label error">Uniprot: Q32ZE1 </span></a>&nbsp;
<a href="https://www.ebi.ac.uk/intenz/query?cmd=SearchEC&amp;ec=3.6.4.13"><span class="label warning">EC: 3.6.4.13</span></a></div><br><div class="row align-center"><table class=unstyled><tr><td><img title="Crystallography construct" src=/images/icons/tep-stage-completed_hub021b18cb248db28b117f458e39aeb5b_10162_e22862b00861f0d3434857db1ff02de5.png width=300 height=88><br><img title="Assay construct" src=/images/icons/tep-stage-completed_hub021b18cb248db28b117f458e39aeb5b_10162_2851d035e13cbc115749c33624801624.png width=300 height=88></td><td><img title="Biophysical assay" src=/images/icons/tep-stage-not-started_hu0ad6bc3691333ead921c5ce1ee694314_9444_3996d997df11e427d2dc236cf2e2aca0.png width=300 height=55><br><img title="Robust crystals" src=/images/icons/tep-stage-completed_hub021b18cb248db28b117f458e39aeb5b_10162_e3383ab8d75acf7a535bdc511b331b23.png width=300 height=55><br><img title="Biochemical assay" src=/images/icons/tep-stage-in-progress_huc9d943f35f5911a48f54b3ad3b40c43a_10037_cf36f8cbd5eddd1cc66a738f8d6c493c.png width=300 height=55></td><td><img title="X-ray fragment screen and structural analysis" src=/images/icons/tep-stage-completed_hub021b18cb248db28b117f458e39aeb5b_10162_ae1969e18479b9f6892265d5fd14a303.png width=300 height=180></td><td><img title="First-round progression for TEP validation" src=/images/icons/tep-stage-not-started_hu0ad6bc3691333ead921c5ce1ee694314_9444_31ed385b113229ad572b9427160c8c98.png width=300 height=180></td></tr></table></div><b>Contributors:</b>
<span class="label success">Andre Schutzer Godoy </span><span class="label success">Nathalya Cristina de Moraes Roso Mesquita </span><span class="label success">Gabriela Dias Noske </span><span class="label success">Rafaela Sachetto Fernandes </span><span class="label success">Glaucius Oliva </span><span class="label success">Daren Fearon </span><span class="label success">Lizbé Koekemoer </span><span class="label success">Charlie Tomlinson </span><span class="label success">Ryan Lithgo </span><span class="label success">Blake Balcomb </span><span class="label success">Jasmin Aschenbrenner </span><span class="label success">Peter Marples</span><br><br><div class="prettybox warning" data-component=prettybox><b>Target Nominator:</b> Andre Schutzer Godoy<br><b>Target PI:</b> Frank von Delft<br><b>Therapeutic Area(s):</b> Infectious diseases<br><b>Disease Relevance:</b> NS3 encodes a helicase that is important for viral replication<br><b>Viral family:</b> flaviviridae<br><b>Viruses:</b> [Zika]<br></div><div class="prettybox success" data-component=prettybox><h4>Summary of Project</h4>The conserved helicase encoded by the C-terminal portion of the non-structural protein 3 (NS3hel) in flaviviruses is employed by the Zika virus (ZIKV) to unwind double stranded RNA during the replication process. This TEP provides early tools to develop NS3hel inhibitors, including purification protocols of recombinant proteins, and reproducible crystallisation condition suitable for X-ray crystallography fragment screening.</div><div class="prettybox focus" data-component=prettybox><h4>Scientific Background</h4>The Zika virus (ZIKV) has become a global public health threat since it was related to major neurological disorders in both infants and adults [<a href=https://doi.org/10.1016/S0140-6736(17)31450-2>1</a>, <a href=https://doi.org/10.1016/S0140-6736(16)00562-6>2</a>]. ZIKV NS3 helicase (NS3hel) is a multifunctional enzyme involved in double-stranded RNA unwind and formation of viral replication machinery, being therefore critical for viral replication [<a href=https://doi.org/10.1016/j.antiviral.2015.03.011>3</a>]. NS3hel is composed by three domains, with motifs typically associated to RNA binding and ATP hydrolysis [<a href=https://doi.org/10.1038/nsmb.3258>4</a>]. Both these activities seem to be essential for viral replication [<a href=https://doi.org/10.1093/nar/gkz650>5</a>]. Moreover, drug candidates targeting NS3hel interactions with other viral proteins, such as non-structural protein 4B, was proved to be an effective way of blocking all Dengue serotypes, leading to two clinical candidates [<a href=https://doi.org/10.1038/s41586-021-03990-6>6</a>, <a href=https://doi.org/10.1126/scitranslmed.abb2>7</a>]. In this way, ZIKV NS3hel is an attractive drug target for the development of antivirals.</div><div class="prettybox error" data-component=prettybox><h4>Conclusion</h4>Here we have established a purification protocol for recombinant NS3 helicase from Zika viruses and established a robust crystallisation suitable for fragment screening. We have performed a substantial fragment screening campaign and identified many hits.
Fragment screen PDB group deposition: <a href=https://www.rcsb.org/groups/summary/entry/G_1002158>https://www.rcsb.org/groups/summary/entry/G_1002158</a></div><h4>Resources</h4><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href><img title=plasmid width=40 src=../img/icons/plasmid.png></a></div><div id=flazi-NS3-helicase_plasmid-crystallography class=col-11><b>Full-length ZIKV NS3 helicase construct suitable for crystallography</b><br>Full-length ZIKV NS3 helicase construct bearing a N-terminal His-tag followed by a thioredoxin followed by a TEV-cleavage site.
This protein yielded highly reproducible crystals upon microseeding, which typically diffracted to 1.5 Å resolution. This crystal system was DMSO tolerant and therefore suitable for our subsequent fragment soaking.<br><cite>[ 2018-02-01 ]</cite><br><button data-component=toggleme data-target=#togglebox-protocol-ASAP-ZIKA-NS3-HELICASE-1 data-text="Hide Protocol">Protocol</button><div id=togglebox-protocol-ASAP-ZIKA-NS3-HELICASE-1 class=hide><cite>For expression of protein used for crystallisation, the construct was transformed into the E. coli Rosetta 2 strain BL21(DE3), and cells were grown at 37°C in ZYM5052 medium for auto-induction supplemented with kanamycin (50 μg/ml) and chloramphenicol (35 μg/ml). After reaching an optical density at 600 nm of 0.5 to 0.6, the temperature was lowered to 18°C before induction of protein expression overnight. Harvested cells were resuspended in lysis buffer [50 mM Tris (pH 7.0), 500 mM NaCl, 10% glycerol, 40 mM imidazole] and stored at -20°C until purification. For protein purification, pellets were gently thawed in ice and added 0.2 mg/mL lysozyme, 2 mM DTT, 2 µM PMSF, 50 U benzonase, followed by 30min incubation on ice. Lysis was conducted by sonication for 10 min, 45s pulse ON, 60s pulse OFF, 40% amplitude. Lysate was centrifuged for 30min at 15,000g at 4oC for clarification. Proteins were purified by immobilized metal affinity chromatography (IMAC) using HiTrap HP 5.0mL Ni-Sepharose resin (GE Healthcare) equilibrated with buffer A. Then protein was washed with 10 column volumes of Buffer A, than with 10% Buffer B [50 mM Tris (pH 7.0), 500 mM NaCl, 10% glycerol, 500 mM imidazole], followed by elution with 15 mL of 100% Buffer B. Removal of the His6 tag was carried out by addition of recombinant 2 mg of TEV protease and 4 mM DTT during overnight dialysis into 2L of buffer A, followed by purification on a second IMAC column, and lastly by SEC (Superdex 75 16/60, GE Healthcare) in a buffer consisting of 500 mM NaCl, 20 mM Bis-Tris, 10 % Glycerol, pH 7.0. Proteins were characterised by SDS-polyacrylamide gel electrophoresis, then concentrated to 5 mg/mL and flash- frozen in liquid nitrogen and stored at -80°C until required.</cite></div><button data-component=toggleme data-target=#togglebox-details-ASAP-ZIKA-NS3-HELICASE-1 data-text="Hide Details">Details</button><div id=togglebox-details-ASAP-ZIKA-NS3-HELICASE-1 class=hide><ul><li>Vector: pET_trx/LIC-ZIKVNS3hMR766</li><li>Entry clone accession:</li><li>Cell line: E. coli Rosetta 2 strain BL21(DE3)</li><li>Tags and additions: N-terminal, Thioredoxin, TEV protease cleavable hexahistidine</li><li>Construct protein sequence:
<code>MKHHHHHHPMSDKIIHLTDDSFDTDVLKADGAILVDFWAEWCGPCKMIAPILDEIADEYQGKLTVAKLNIDQNPGTAPKYGIRGIPTLLLFKNGEVAATKVGALSKGQLKEFLDANLAGSGSGSENLYFQ*GAMLKKKQLTVLDLHPGAGKTRRVLPEIVREAIKKRLRTVILAPTRVVAAEMEEALRGLPVRYMTTAVNVTHSGTEIVDLMCHATFTSRLLQPIRVPNYNLNIMDEAHFTDPSSIAARGYISTRVEMGEAAAIFMTATPPGTRDAFPDSNSPIMDTEVEVPERAWSSGFDWVTDHSGKTVWFVPSVRNGNEIAACLTKAGKRVIQLSRKTFETEFQKTKNQEWDFVITTDISEMGANFKADRVIDSRRCLKPVILDGERVILAGPMPVTHASAAQRRGRIGRNPNKPGDEYMYGGGCAETDEGHAHWLEARMLLDNIYLQDGLIASLYRPEADKVAAIEGEFKLRTEQRKTFVELMKRGDLPVWLAYQVASAGITYTDRRWCFDGTTNNTIMEDSVPAEVWTKYGEKRVLKPRWMDARVCSDHAALKSFKEFAAGKR</code></li></ul></div></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href=https://www.rcsb.org/structure/5RHY><img title=pdb-deposition width=40 src=../img/icons/pdb-deposition.png></a></div><div id=flazi-NS3-helicase_pdb-structure-apo class=col-11><b>Apo structure of ZIKV NS3 helicase</b><br>Apo structure of ZIKV NS3 helicase at 1.36A resolution in P 1 21 1 space group.<br><a href=https://www.rcsb.org/structure/5RHY>https://www.rcsb.org/structure/5RHY</a>
<cite>[ 2020-06-10 ]</cite><br></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href=https://zenodo.org/record/7864388#.ZEgRzXbMKUk><img title=xray-fragment-screen width=40 src=../img/icons/xray-fragment-screen.png></a></div><div id=flazi-NS3-helicase_fragment-screen-zenodo class=col-11><b>Zenodo deposition of X-ray fragment screen of ZIKV NS3 helicase</b><br>We performed X-ray crystallography screening at XChem/Diamond Light Source with a total of 1200 fragments, which yielded 29 unique ZIKV NS3hel binders.<br><a href=https://zenodo.org/record/7864388#.ZEgRzXbMKUk>https://zenodo.org/record/7864388#.ZEgRzXbMKUk</a>
<cite>[ 2022-06-01 ]</cite><br><button data-component=toggleme data-target=#togglebox-protocol-ASAP-ZIKA-NS3-HELICASE-3 data-text="Hide Protocol">Protocol</button><div id=togglebox-protocol-ASAP-ZIKA-NS3-HELICASE-3 class=hide><cite>ZIKV NS3hel was concentrated to a final concentration of 5 mg/ml and apo crystals were grown in crystallisation solution containing 0.12 M NPS Mix (0.3 M Sodium phosphate dibasic dihydrate, 0.3 M Ammonium sulphate and 0.3 M Sodium nitrate - Molecular Dimensions), 0.1 M MES/Imidazole pH 6.5 (Molecular Dimensions) and 33% Precipitant Mix 4 (11% MPD, 11% PEG 1,000 e 11% PEG 3,350 – Sigma Aldrich). Fragments were soaked into crystals as previously described by adding dissolved compounds directly to the crystallisation drops using an Echo liquid handler (final concentration, 20% DMSO); drops were incubated for approximately 1 to 3 hours before mounting and flash-freezing in liquid nitrogen. Data were collected at the I04-1 beamline at 100 K and automatically processed with Diamond Light Source&rsquo;s autoprocessing pipelines using XDS (9) and either <a href=https://doi.org/10.1107/s0907444913015308>xia2</a> or <a href=https://doi.org/10.1107/s2059798317017235>DIALS</a>. Further analysis was performed with <a href=https://doi.org/10.1107%2FS2059798316020234>XChemExplorer</a>, electron density maps were generated with DIMPLE and ligand-binding events were identified using <a href=https://doi.org/10.1038/ncomms15123>PanDDA</a>. Ligands were modeled into PanDDA-calculated event maps using Coot, restraints were calculated with <a href=http://dx.doi.org/10.1107/S2059798317000067>AceDRG</a>, and structures were refined with <a href=https://doi.org/10.1107/s0907444992010400>BUSTER</a>. PDB codes: 7G9K, 7G9L, 7G9M, 7G9N, 7G9O, 7G9P, 7G9Q, 7G9R, 7G9S, 7G9T, 7G9U, 7G9V, 7G9W, 7G9X, 7G9Y, 7G9Z, 7GA0, 7GA1, 7GA2, 7GA3, 7GA4, 7GA5, 7GA6, 7GA7</cite></div></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href=https://www.rcsb.org/groups/summary/entry/G_1002158><img title=pdb-group-deposition width=40 src=../img/icons/pdb-group-deposition.png></a></div><div id=flazi-NS3-helicase_fragment-screen-pdb-group-deposition class=col-11><b>PDB group deposition of X-ray fragment screen of ZIKV NS3 helicase</b><br>PDB group deposition of the X-ray fragment screen of ZIKV NS3 helicase containing 17 fragment-bound complexes<br><a href=https://www.rcsb.org/groups/summary/entry/G_1002158>https://www.rcsb.org/groups/summary/entry/G_1002158</a>
<cite>[ 2020-06-10 ]</cite><br></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href=https://www.rcsb.org/groups/summary/entry/G_1002270><img title=pdb-group-deposition width=40 src=../img/icons/pdb-group-deposition.png></a></div><div id=flazi-NS3-helicase_fragment-screen-pdb-group-deposition-2 class=col-11><b>PDB group deposition of supplemental X-ray fragment screen of ZIKV NS3 helicase</b><br>PDB group deposition of supplemental X-ray fragment screen of ZIKV NS3 helicase containing 24 fragment-bound complexes<br><a href=https://www.rcsb.org/groups/summary/entry/G_1002270>https://www.rcsb.org/groups/summary/entry/G_1002270</a>
<cite>[ 2023-07-23 ]</cite><br></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href=https://doi.org/10.1101/2024.04.27.591279><img title=preprint width=40 src=../img/icons/preprint.png></a></div><div id=flazi-NS3-helicase_fragment-screen-preprint class=col-11><b>High-throughput crystallographic fragment screening of Zika virus NS3 Helicase</b><br>Preprint describing the X-ray fragment screen of ZIKV NS3 helicase<br><a href=https://doi.org/10.1101/2024.04.27.591279>https://doi.org/10.1101/2024.04.27.591279</a>
<cite>[ 2024-04-29 ]</cite><br></div></div></div></div><hr style=height:5px;border:none;color:#333;background-color:#333><br><br><br><h3 style=text-align:center id=ASAP-ZIKA-NS5-RDRP>Zika NS5 RNA-dependent RNA polymerase</h3><div class="row align-center"><a href=https://www.ncbi.nlm.nih.gov/protein/AY632535.2><span class="label success">Gene: AY632535.2 </span></a>&nbsp;
<a href=https://www.uniprot.org/uniprotkb/Q32ZE1><span class="label error">Uniprot: Q32ZE1 </span></a>&nbsp;
<a href="https://www.ebi.ac.uk/intenz/query?cmd=SearchEC&amp;ec=3.4.21.91"><span class="label warning">EC: 3.4.21.91</span></a></div><br><div class="row align-center"><table class=unstyled><tr><td><img title="Crystallography construct" src=/images/icons/tep-stage-completed_hub021b18cb248db28b117f458e39aeb5b_10162_e22862b00861f0d3434857db1ff02de5.png width=300 height=88><br><img title="Assay construct" src=/images/icons/tep-stage-completed_hub021b18cb248db28b117f458e39aeb5b_10162_2851d035e13cbc115749c33624801624.png width=300 height=88></td><td><img title="Biophysical assay" src=/images/icons/tep-stage-not-started_hu0ad6bc3691333ead921c5ce1ee694314_9444_3996d997df11e427d2dc236cf2e2aca0.png width=300 height=55><br><img title="Robust crystals" src=/images/icons/tep-stage-completed_hub021b18cb248db28b117f458e39aeb5b_10162_e3383ab8d75acf7a535bdc511b331b23.png width=300 height=55><br><img title="Biochemical assay" src=/images/icons/tep-stage-in-progress_huc9d943f35f5911a48f54b3ad3b40c43a_10037_cf36f8cbd5eddd1cc66a738f8d6c493c.png width=300 height=55></td><td><img title="X-ray fragment screen and structural analysis" src=/images/icons/tep-stage-in-progress_huc9d943f35f5911a48f54b3ad3b40c43a_10037_4b49a9d8141ee102ae7f3907ab3f8f53.png width=300 height=180></td><td><img title="First-round progression for TEP validation" src=/images/icons/tep-stage-not-started_hu0ad6bc3691333ead921c5ce1ee694314_9444_31ed385b113229ad572b9427160c8c98.png width=300 height=180></td></tr></table></div><b>Contributors:</b>
<span class="label success">Andre de Godoy </span><span class="label success">Annette von Delft </span><span class="label success">Anu Chandran </span><span class="label success">Blake Balcomb </span><span class="label success">Charlie Tomlinson </span><span class="label success">Charline Giroud </span><span class="label success">Daren Fearon </span><span class="label success">Dong Wang </span><span class="label success">Eleanor Williams </span><span class="label success">Glaucius Oliva </span><span class="label success">Haim Barr </span><span class="label success">Jasmin Aschenbrenner </span><span class="label success">Jessica Dixon </span><span class="label success">Korvus Wang </span><span class="label success">Lee Harris </span><span class="label success">Lizbé Koekemoer </span><span class="label success">Lucas Ferreira </span><span class="label success">Martin Walsch </span><span class="label success">Michael Fairhead </span><span class="label success">Noa Lahav </span><span class="label success">Oleg Fedorov </span><span class="label success">Peter Marples </span><span class="label success">Peter Marples </span><span class="label success">Rafaela Sachetto Fernandes </span><span class="label success">Ryan Lithgo </span><span class="label success">Tamas Szommer </span><span class="label success">Warren Thompson </span><span class="label success">Xiaomin Ni</span><br><br><div class="prettybox warning" data-component=prettybox><b>Target Nominator:</b> Karla Kirkegaard<br><b>Target PI:</b> Frank von Delft<br><b>Therapeutic Area(s):</b> Infectious diseases<br><b>Disease Relevance:</b> NS5 encodes an RNA-dependent RNA polymerase<br><b>Viral family:</b> flaviviridae<br><b>Viruses:</b> [Zika]<br></div><div class="prettybox success" data-component=prettybox><h4>Summary of Project</h4>The RdRp is responsible for synthesizing a complementary negative-sense RNA strand that can then be used as a template to produce extra positive-sense RNA genomes.</div><div class="prettybox focus" data-component=prettybox><h4>Scientific Background</h4>The Zika virus (ZIKV) has become a global public health threat since it was related to major neurological disorders in both infants and adults [<a href=https://doi.org/10.1016/S0140-6736(17)31450-2>1</a>, <a href=https://doi.org/10.1016/S0140-6736(16)00562-6>2</a>]. The non-structural protein 5 (NS5) is the largest non-structural protein from flaviviruses. The N-terminal of NS5 encodes the viral methyltransferase, while the C-terminal encodes an RNA-dependent RNA polymerase (RdRp) [<a href=https://doi.org/10.1038/ncomms14764>3</a>]. The RdRp is responsible for synthesizing a complementary negative-sense RNA strand that can then be used as a template to produce extra positive-sense RNA genomes. Not only critical to viral replication, the flaviviruses RdRp is also a highly conserved multidomain protein, therefore being one of the most valuable targets for the development of new antivirals, including nucleoside and non-nucleoside analogs [<a href=https://doi.org/10.1016/j.sbi.2019.02.014>4</a>].</div><div class="prettybox error" data-component=prettybox><h4>Conclusion</h4>Here we have established a purification protocol for recombinant Truncated ZIKV RdRp from Zika viruses and established a robust crystallisation suitable for fragment screening</div><h4>Resources</h4><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href><img title=plasmid width=40 src=../img/icons/plasmid.png></a></div><div id=flazi-NSF5B-polymerase_plasmid-crystallography class=col-11><b>Protein expression construct of ZIKV NS5B RdRp suitable for crystallography</b><br>Truncated ZIKV RdRp construct bearing a N-terminal His-tag followed by a thioredoxin followed by a TEV-cleavage site.
This protein construct yielded highly reproducible crystals upon microseeding, which typically diffracted to 1.9 Å resolution. This crystal system was DMSO tolerant and therefore suitable for our subsequent fragment soaking.<br><b>SCARAB ID:</b>
<img width=10 src=images/icons/scarab.png>
XXX<br><cite>[ 2022-10-01 ]</cite><br><button data-component=toggleme data-target=#togglebox-protocol-ASAP-ZIKA-NS5-RDRP-1 data-text="Hide Protocol">Protocol</button><div id=togglebox-protocol-ASAP-ZIKA-NS5-RDRP-1 class=hide><cite>A synthetic DNA fragment of the ZIKV NS5 (GenBank: AY632535.2) was codon optimized and synthesized by Genscript. The truncated version for crystallization was amplified using primers 5′-CAGGGCGCCATGTACCATGGGAGCTACGAAGC-3′ and 5′-GACCCGACGCGGTTACAACACTCCGGGTGTGG-3′ and inserted into the expression vector pETTrx-1a/LIC by ligation-independent cloning, as previously described [<a href=https://doi.org/10.1038/ncomms14764>3</a>]. The construct was transformed into the E.coli strain BL21(DE3)-RR and cells were grown at 37˚C in LB media supplemented with kanamycin (50µg/ml) shaking at 250 rpm. After reaching an optical density at 600 nm of 0.6, the temperature was lowered to 18˚C before induction of protein expression overnight with the addition of 0.5 mM IPTG. Shaking was reduced to 160 rpm to slow protein expression. Harvested cells were resuspended in lysis buffer (50 mM HEPES pH 7.5, 500 mM NaCl 25 mM Imidazole, 5 % (v/v) Glycerol, 0.5 mM TCEP-HCl, supplemented with 0.5 mg/mL Lysozyme, 1 X cOmplete™ EDTA free Protease Inhibitor, Benzonase and 2 mM MgCl2). Proteins were first purified by immobilized metal affinity chromatography (IMAC) at 4˚C using Ni-Sepharose 6 FF resin (Cytiva). The sample was incubated with the beads for no more than 15 minutes before the column was washed with binding buffer [10 mM Hepes (pH 7.5), 500 mM NaCl, 5% glycerol, 20 mM imidazole, 1 mM TCEP] and target protein eluted using same buffer containing 500 mM imidazole. The sample was desalted using a HiPrep 26/10 Desalting column (Cytiva) into TEV Cleavage Buffer (50 mM HEPES pH 7.5, 500 mM NaCl, 5 % (v/v) Glycerol 2 mM MgCl2 and 0.5 mM TCEP-HCl). The tag was removed by rapid digestion with Tev protease at a ratio of 1:10 incubating at room temperature while agitated for 1h. Cleaved tag and un-cleaved protein was removed by using Ni-Sepharose 6 FF resin (Cytiva) and the unbound fraction containing untagged protein collected. Note: Sensitivity to Ni resin of cleaved protein necessitates minimal contact with resin at this point, three passes through a gravity column were used. The resulting protein was pooled, concentrated and injected into a HiLoad 16/60 Superdex 200 column (Cytiva) equilibrated with Gel Filtration Buffer (50 mM HEPES pH 7.5, 500 mM NaCl, 5 % (v/v) Glycerol and 0.5 mM TCEP-HCl). Proteins were characterised by SDS-polyacrylamide gel electrophoresis andd mass spec and then flash- frozen in liquid nitrogen, and stored at -80°C until required.</cite></div><button data-component=toggleme data-target=#togglebox-details-ASAP-ZIKA-NS5-RDRP-1 data-text="Hide Details">Details</button><div id=togglebox-details-ASAP-ZIKA-NS5-RDRP-1 class=hide><ul><li>Vector: pETTrx-1a/LIC</li><li>Entry clone accession: AY632535.2</li><li>Cell line: E. coli Rosetta strain E.coli strain BL21(DE3)-RR</li><li>Tags and additions: C-terminal, cleavable hexahistidine</li><li>Construct protein sequence: YHGSYEAPTQGSASSLVNGVVRLLSKPWDVVTGVTGIAMTDTTPYGQQRVFKEKVDTRVPDPQEGTRQVMNIVSSWLWKELGKRKRPRVCTKEEFINKVRSNAALGAIFEEEKEWKTAVEAVNDPRFWALVDREREHHLRGECHSCVYNMMGKREKKQGEFGKAKGSRAIWYMWLGARFLEFEALGFLNEDHWMGRENSGGGVEGLGLQRLGYILEEMNRAPGGKMYADDTAGWDTRISKFDLENEALITNQMEEGHRTLALAVIKYTYQNKVVKVLRPAEGGKTVMDIISRQDQRGSGQVVTYALNTFTNLVVQLIRNMEAEEVLEMQDLWLLRKPEKVTRWLQSNGWDRLKRMAVSGDDCVVKPIDDRFAHALRFLNDMGKVRKDTQEWKPSTGWSNWEEVPFCS
HHFNKLYLKDGRSIVVPCRHQDELIGRARVSPGAGWSIRETACLAKSYAQMWQLLYFHRRDLRLMANAICSAVPVDWVPTGRTTWSIHGKGEWMTTEDMLMVWNRVWIEENDHMEDKTPVTKWTDIPYLGKREDLWCGSLIGHRPRTTWAENIKDTVNMVRRIIGDEEKYMDYLSTQVRYLGEEGSTPGVL</li></ul><p>Purification trials:</p><p><a href="/pdfs/flazi_ns5_rdrp Expression and Purification of Zika NS5 RDRP Trial 3.pdf" download>Download experimental metadata of trial 3</a></p><p><a href="/pdfs/flazi_ns5_rdrp Expression and Purification of Zika NS5 RDRP Trial 4.pdf" download>Download experimental metadata of trial 4</a></p></div></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href><img title=protocol width=40 src=../img/icons/protocol.png></a></div><div id=Crystallization class=col-11><b>Experimental crystallization protocol for Zika NS5 RNA-dependent RNA polymerase; suitable for fragment screening</b><br>We obtained crystal which diffracted to 2.0 Å.
PDB to be released:<br><cite>[ 2022-11-26 ]</cite><br><button data-component=toggleme data-target=#togglebox-protocol-ASAP-ZIKA-NS5-RDRP-2 data-text="Hide Protocol">Protocol</button><div id=togglebox-protocol-ASAP-ZIKA-NS5-RDRP-2 class=hide><cite>(availible at protocols.io)[dx.doi.org/10.17504/protocols.io.n2bvjnxjngk5/v1]</cite></div></div></div></div></div><hr style=height:5px;border:none;color:#333;background-color:#333><br><br><br><h3 style=text-align:center id=MOONSHOT-COV-MPRO>SARS-CoV-2 Mpro</h3><div class="row align-center"><a href="https://www.ncbi.nlm.nih.gov/protein/%25!s%28int=43740578%29"><span class="label success">Gene: 43740578 </span></a>&nbsp;
<a href=https://www.uniprot.org/uniprotkb/P0C6U8><span class="label error">Uniprot: P0C6U8 </span></a>&nbsp;
<a href="https://www.ebi.ac.uk/intenz/query?cmd=SearchEC&amp;ec=3.4.22.69"><span class="label warning">EC: 3.4.22.69</span></a></div><br><div class="row align-center"><table class=unstyled><tr><td><img title="Crystallography construct" src=/images/icons/tep-stage-completed_hub021b18cb248db28b117f458e39aeb5b_10162_e22862b00861f0d3434857db1ff02de5.png width=300 height=88><br><img title="Assay construct" src=/images/icons/tep-stage-completed_hub021b18cb248db28b117f458e39aeb5b_10162_2851d035e13cbc115749c33624801624.png width=300 height=88></td><td><img title="Biophysical assay" src=/images/icons/tep-stage-completed_hub021b18cb248db28b117f458e39aeb5b_10162_05cf9afc49ba5efb1c08e4e2e8fbaf47.png width=300 height=55><br><img title="Robust crystals" src=/images/icons/tep-stage-completed_hub021b18cb248db28b117f458e39aeb5b_10162_e3383ab8d75acf7a535bdc511b331b23.png width=300 height=55><br><img title="Biochemical assay" src=/images/icons/tep-stage-completed_hub021b18cb248db28b117f458e39aeb5b_10162_1b7ab3b95a075266c44f73afb56d1cd3.png width=300 height=55></td><td><img title="X-ray fragment screen and structural analysis" src=/images/icons/tep-stage-completed_hub021b18cb248db28b117f458e39aeb5b_10162_ae1969e18479b9f6892265d5fd14a303.png width=300 height=180></td><td><img title="First-round progression for TEP validation" src=/images/icons/tep-stage-not-started_hu0ad6bc3691333ead921c5ce1ee694314_9444_31ed385b113229ad572b9427160c8c98.png width=300 height=180></td></tr></table></div><b>Contributors:</b>
<span class="label success">Alice Douangamath </span><span class="label success">Daren Fearon </span><span class="label success">Paul Gehrtz </span><span class="label success">Petra Lukacik </span><span class="label success">C. David Owen </span><span class="label success">Efrat Resnick </span><span class="label success">Claire Strain-Damerell </span><span class="label success">Anthony Aimon </span><span class="label success">Péter Ábrányi-Balogh </span><span class="label success">José Brandão-Neto </span><span class="label success">Anna Carbery </span><span class="label success">Gemma Davison </span><span class="label success">Alexandre Dias </span><span class="label success">Thomas D. Downes </span><span class="label success">Louise Dunnett </span><span class="label success">James D. Firth </span><span class="label success">S. Paul Jones </span><span class="label success">Aaron Keeley </span><span class="label success">György M. Keserü </span><span class="label success">Hanna F. Klein </span><span class="label success">Mathew P. Martin </span><span class="label success">Martin E. M. Noble </span><span class="label success">Peter O’Brien </span><span class="label success">Ailsa Powell </span><span class="label success">Rambabu N. Reddi </span><span class="label success">Rachael Skyner </span><span class="label success">Matthew Snee </span><span class="label success">Tika R. Malla </span><span class="label success">Anthony Tumber </span><span class="label success">Lennart Brewitz </span><span class="label success">Michael J. Waring </span><span class="label success">Christopher Schofield </span><span class="label success">John D. Chodera </span><span class="label success">Alpha Lee </span><span class="label success">Nir London </span><span class="label success">Martin A. Walsh</span><br><br><div class="prettybox warning" data-component=prettybox><b>Target Nominator:</b> Martin Walsh<br><b>Target PI:</b> Frank von Delft<br><b>Therapeutic Area(s):</b> Infectious diseases<br><b>Disease Relevance:</b> The SARS-Cov-2 main protease (Mpro) is responsible for viral polyprotein processing and therefore a therapeutic target.<br><b>Viral family:</b> coronaviridae<br><b>Viruses:</b> [SARS-CoV-2]<br></div><div class="prettybox success" data-component=prettybox><h4>Summary of Project</h4>The COVID-19 outbreak of 2020 caused more than one million fatalities so far, it led to a world-wide disruption of public life and triggered economic upheaval. The disease is triggered by SARS-CoV-2, a coronavirus for which there are currently no effective therapeutics available. The SARS-CoV-2 main protease (Mpro) is essential for viral replication and therefore an attractive target to stop the virus. This TEP includes crystal structures of ligand-free Mpro and numerous protein-ligand complexes which are the result of an extensive crystallographic fragment screen. Moreover, it contains information about different enzymatic and viral replication assays for orthogonal validation of small molecule inhibitors in order determine structure-activity relationships. Further hit-to-lead progression of the screening hits was facilitated by the COVID Moonshot consortium and this resulted in several lead series with sub-micromolar potencies. All data for over 1200 synthesised compounds are fully open and constitute a unique resource, not only for this target, but for improving methodology of fragment-to-hit and hit-to-lead progression.</div><div class="prettybox focus" data-component=prettybox><h4>Scientific Background</h4>The emergence of COVID-19 has caused a global pandemic of unexpected scale. The disease is caused by the SARS-Cov-2 virus [<a href=https://doi.org/10.1016%2FS1473-3099(20)30144-4>1</a>, <a href=https://doi.org/10.1038/s41586-020-2008-3>2</a>, <a href=https://doi.org/10.1056/nejmoa2001017>3</a>] for which no approved drugs exist or are in the pipeline. This disease has so far claimed more than 1.2 million lives world-wide (November 2020, <a href=https://covid19.who.int>https://covid19.who.int</a>), despite numerous regional lockdowns in order to stop the spread of the virus. However, despite the fact that SARS-CoV-2 is already the third zoonotic coronavirus outbreak after the emergence of SARS-CoV-1 in 2002 and the Middle East Respiratory Syndrome (MERS-CoV) in 2012 [<a href=https://pubmed.ncbi.nlm.nih.gov/23078800/>4</a>, <a href=https://doi.org/10.1016/s0140-6736(03)13967-0>5</a>, <a href=https://doi.org/10.1056/nejmoa1211721>6</a>], there is a notable lack of antiviral treatment options. SARS-CoV-2 is a large enveloped, positive-sense, single-stranded RNA Betacoronavirus from the order of nidoviruses. Its RNA encodes two open reading frames that, through ribosome frame-shifting, generates two polyproteins pp1a and pp1ab [<a href=https://doi.org/10.1093%2Fnar%2F18.7.1825>7</a>]. The genome contains a large replicase gene encompassing nonstructural proteins (Nsps), followed by structural and accessory genes. The polyproteins are processed by two viral cysteine proteases: a papain-like protease (PLpro) which cleaves three sites, releasing non-structural proteins nsp1-3 and a 3C-like protease, also referred to as the main protease (Mpro), that cleaves at 11 sites to release non-structural proteins (nsp4-16). These non-structural proteins form the replicase complex responsible for replication and transcription of the viral genome and have led to Mpro and PLPro being the primary targets for antiviral drug development [<a href=https://doi.org/10.1111/febs.12936>8</a>].</div><div class="prettybox error" data-component=prettybox><h4>Conclusion</h4><p>This TEP provides a comprehensive sampling of the binding properties of the active site of SARS-Cov-2 main protease by small molecule fragment molecules. Together with the established enzymatic assays, it forms the bases for rational, structure-based drug development. The usefulness of the data and protocols presented in this TEP have been demonstrated by the <a href=https://covid.postera.ai/covid>COVID Moonshot consortium</a> successfully developing sub-micromolar binders, that are now in lead optimisation.</p><ul><li>Fragalysis interactive structure viewer (and downloads of all structures): <a href=https://fragalysis.diamond.ac.uk/viewer/react/preview/target/Mpro>https://fragalysis.diamond.ac.uk/viewer/react/preview/target/Mpro</a></li><li>Fragment screen PDB group deposition (currently contains only a subset of structures): <a href=https://www.rcsb.org/groups/summary/entry/G_1002153>https://www.rcsb.org/groups/summary/entry/G_1002153</a></li></ul></div><h4>Resources</h4><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href=https://www.diamond.ac.uk/dam/jcr:feefeac2-8f38-4dba-84e4-139f83e8a375/ProteaseMandM_v4.docx><img title=plasmid width=40 src=../img/icons/plasmid.png></a></div><div id=moonshot-mpro_plasmid-crystallography class=col-11><b>Protein expression construct of SARS-CoV-2 Mpro suitable for crystallographyic fragment screens</b><br>Mpro from SARS-CoV-2 was cloned into pGEX-6P-1 for heterologous protein expression in Escherichia coli. The protein was purified by Ni-affinity chromatography, followed by HRV 3C protease treatment which yielded native N- and C-termini. HRV 3C protease was removed by reverse Ni-affinity chromatography, followed by a gel filtration step. The purified protein appears as a dimer in size-exclusion chromatography.<br><a href=https://www.diamond.ac.uk/dam/jcr:feefeac2-8f38-4dba-84e4-139f83e8a375/ProteaseMandM_v4.docx>https://www.diamond.ac.uk/dam/jcr:feefeac2-8f38-4dba-84e4-139f83e8a375/ProteaseMandM_v4.docx</a>
<cite>[ 2020-02-14 ]</cite><br><button data-component=toggleme data-target=#togglebox-protocol-MOONSHOT-COV-MPRO-1 data-text="Hide Protocol">Protocol</button><div id=togglebox-protocol-MOONSHOT-COV-MPRO-1 class=hide><cite>The expression clone of Mpro (Accession: YP_009725301, Protein Region: S1-Q306) consists of a synthetic gene in pGEX-6P-1 expression plasmid containing N-terminal GST and C-terminal 3C PROTEASE and 6xHIS tags. After transformation into BL21(DE3)-R3-pRARE2, expression was performed in Auto Induction Medium supplemented with glycerol and 100 μg/ml Carbenicillin. The cultures were grown at 37 °C, 200 rpm for 5 h before switching to 18 °C, 200 rpm for a further 10 h. The cells were harvested by centrifugation and stored at -80 °C. Cell pellets were resuspended in buffer containing 50 mM Tris pH 8, 300 mM NaCl, 10 mM Imidazole and 0.03 μg/ml Benzonase before disrupting with a highpressure homogeniser (3 passes, 30 kpsi, 4 °C). The lysate was clarified by centrifugation at 50,000 × g and the supernatant applied to a Nickel-NTA gravity column and washed with buffer containing 50 mM Tris pH 8, 300 mM NaCl and 25 mM imidazole pH 8. The protein was eluted using a gradient of 25–500 mM imidazole. N-terminal His-tagged HRV 3 C Protease was added to the eluted protein at 1:10 w/w ratio. The mixture was dialysed overnight at 4 °C against 50 mM Tris pH 8, 300 mM NaCl, 1 mM TCEP, after which the HRV 3 C protease and other impurities were removed from the cleaved target protein by reverse Nickel-NTA. The relevant fractions were concentrated and applied to an S200 16/60 gel filtration column equilibrated in 20 mM HEPES pH 7.5, 50 mM NaCl buffer. The protein was concentrated to 30 mg/ml using a 10 kDa MWCO centrifugal filter device.</cite></div></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href=https://fragalysis.diamond.ac.uk/viewer/react/preview/target/Mpro><img title=xray-fragment-screen width=40 src=../img/icons/xray-fragment-screen.png></a></div><div id=moonshot-mpro-fragment-screen-fragalysis class=col-11><b>Fragalysis interactive exploration of X-ray fragment screen of SARS-CoV-2 Mpro</b><br>We performed an exhaustive crystallographic fragment screen to probe the Mpro active site, and to identify opportunities for fragment merging or growing. Additionally, a library of mild electrophilic fragments was screened by mass spectrometry for probing the binding properties around the active site cysteine. Overall, a total of 1742 soaking and 1139 co-crystallization experiments resulted in 1877 mounted crystals. This resulted in determination of 91 protein-fragment structures.<br><a href=https://fragalysis.diamond.ac.uk/viewer/react/preview/target/Mpro>https://fragalysis.diamond.ac.uk/viewer/react/preview/target/Mpro</a>
<cite>[ 2020-03-18 ]</cite><br><button data-component=toggleme data-target=#togglebox-protocol-MOONSHOT-COV-MPRO-2 data-text="Hide Protocol">Protocol</button><div id=togglebox-protocol-MOONSHOT-COV-MPRO-2 class=hide><cite><p><strong>Crystallization and Structure determination</strong><br>Protein was thawed and diluted to 5 mg/ml using 20 mM Hepes pH 7.5, 50 mM NaCl. The sample was centrifuged at 100 000 g for 15 minutes to remove aggregates. MPRO crystals were obtained by mixing 150nL of protein in with 300 nL of reservoir solution containing 0.1 M MES pH 6.5, 15% PEG6K, 5% DMSO and 50nL seed stock. Crystals were grown using the sitting drop vapor diffusion method at 20°C and appeared within 24 hours.
<strong>Crystallographic fragment screening</strong><br>Data were collected at the beamline I04-1 at 100 K and processed with the fully automated pipelines at Diamond Light Source [<a href=https://doi.org/10.1016/j.ymeth.2011.06.010>11</a>], which variously combine XDS [<a href=https://doi.org/10.1107/s0907444909047374>12</a>], xia2 [<a href=https://doi.org/10.1107/s0907444913015308>13</a>], autoPROC [<a href=https://journals.iucr.org/https://dx.doi.org/10.1107/S0907444911007773>14</a>] and DIALS [<a href=https://doi.org/10.1107%2FS2059798317017235>15</a>], and select resolution limits algorithmically; no manual curation of processing parameters was applied. Further analysis was performed through XChemExplorer [<a href=https://doi.org/10.1107%2FS2059798316020234>16</a>]: for each dataset, the version of processed data was selected by the default XChemExplorer score, and electron density maps were generated with Dimple. Ligand-binding events were identified using PanDDA [<a href=https://doi.org/10.1038/ncomms15123>18</a>] (both the released version 0.2 and a pre-release development version (<a href=https://github.com/ConorFWild/pandda)>https://github.com/ConorFWild/pandda)</a>, and ligands were modelled into PanDDAcalculated event maps using Coot [<a href=https://doi.org/10.1107/s0907444910007493>19</a>]. Restraints were calculated with ACEDRG [<a href=https://doi.org/10.1107%2FS2059798317000067>20</a>] or GRADE, structures were refined with Refmac [<a href=https://doi.org/10.1107/s0907444911001314>22</a>] and Buster.</p><p>Fragments were soaked into crystals by adding dissolved compound directly to the crystallisation drops. The following libraries were screened: the DSipoised library (Enamine), a version of the poised library [<a href=https://doi.org/10.1039/C5SC03115J>24</a>]; a version of the MiniFrags library [<a href=https://doi.org/10.1016/j.drudis.2019.03.009>25</a>] assembled in-house; the FragLites library; a library of shape-diverse 3D fragments (“York3D”) [<a href=https://doi.org/10.1002/chem.202001123>27</a>]; heterocyclic electrophiles [<a href="http://xlink.rsc.org/?DOI=c8md00327k">28</a>]; and the SpotFinder library. All fragments were in 100% DMSO at varying stock concentrations, detailed at <a href=https://www.diamond.ac.uk/Instruments/Mx/>https://www.diamond.ac.uk/Instruments/Mx/</a> Fragment-Screening/Fragment-Libraries.html). In brief, 55 nl of fragment stock solutions in DMSO (DSIpoised, FragLite, PepLites, York 3D, Covalent Heterocylces and SpotFinder all at 500 mM, MiniFrags at 1 M and Cysteine covalent library at 20 mM) were transferred directly to 500 nl crystallisation drops using an ECHO liquid handler giving a final compound concentration of 2–100 mM and DMSO concentration of 10%. Drops were incubated at room temperature for ~1 h prior to mounting and flash cooling in liquid nitrogen without the addition of further cryoprotectant.
<strong>FRET-based proteolytic assay</strong><br>Assays are performed in Greiner 384 low volume plates into which the compounds are dispensed using an Echo 555 acoustic dispenser keeping the DMSO concentration &lt; 1%. Mpro is pre-incubated for 15 minutes at room temperature with the inhibitor compounds before the substrate is added to initiate the reaction. Protease reaction was measured continuously in a BMG Pherastar FS with a 480/520 ex/em filter set. The final reaction concentrations are 20 mM HEPES pH 7.3, 1 mM TCEP, 50 mM NaCl, 0.01% Tween-20, 10% glycerol, 5 nM Mpro, 375 nM fluorogenic peptide substrate ([5-FAM]-AVLQSGFR-[Lys(Dabcyl)]-K-amide) in 20 μL. All data is directly uploaded to the Moonshot vault on the Collaborative Drug Discovery Inc. (CDD) website for data analysis. The readouts are converted to % inhibition and the Levenberg–Marquardt algorithm is used to fit a Hill equation to dose-response data constrained with minimum value = 0.0 and maximum value = 100 to calculate IC50 values.
<strong>RapidFire Mass Spectrometry assay</strong><br>Assays are performed in Greiner 384 low volume plates into which the compounds are dispensed using an Echo 555 acoustic dispenser keeping the DMSO concentration &lt; 1%. A 15 μM enzyme stock solution is prepared in 20 mM HEPES, pH 7.5 and 300 mM NaCl, and subsequently diluted to a working solution of 300 nM Mpro in assay buffer (20 mM HEPES, pH 7.5 and 50 mM NaCl). Mpro is pre-incubated for 15 minutes at room temperature with the inhibitor compounds before the addition of substrate to initiate the reaction. The final reaction concentrations are 20 mM HEPES pH 7.5, 50 mM NaCl, 150 nM Mpro and 2 μM substrate (TSAVLQSGFRK-NH2) in 50 μL. The reaction is incubated for 10 min at room temperature before quenching with 10% formic acid. The reactions are analysed with MS using RapidFire (RF) 365 high-throughput sampling robot (Agilent) connected to an iFunnel Agilent 6550 accurate mass quadrupole time-of-flight (QTOF) mass spectrometer using electrospray. RapidFire integrator software (Agilent) was used to extract the m/z (+1) charge states of both the substrate (1191.67 Da) and cleaved N-terminal product TSAVLQ (617.34 Da) from the total ion chromatogram data which is used to calculate the percentage inhibition. All data is directly uploaded to the Moonshot vault for data analysis. The readouts are converted to % inhibition and the Levenberg–Marquardt algorithm is used to fit a Hill equation to dose-response data constrained with minimum value = 0.0 and maximum value = 100 to calculate IC50 values.
Protocols, data tables, and hits can be downloaded <a href=https://www.diamond.ac.uk/covid-19/for-scientists/Main-protease-structure-and-XChem/Downloads.html>here</a>.</p></cite></div></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href=https://www.rcsb.org/groups/summary/entry/G_1002153><img title=pdb-group-deposition width=40 src=../img/icons/pdb-group-deposition.png></a></div><div id=moonshot-mpro-fragment-screen-pdb-group-deposition class=col-11><b>PDB group deposition of crystallographic fragment screen of SARS-CoV-2 Mpro</b><br>SARS-CoV-2 main protease from Coronaviridae sp. screened against DSI poised (Enamine), Fraglites and Peplites (Newcastle university), Mini Frags (Astex), York 3D (York university), electrophile cysteine covalent (Weizman institute) fragment libraries by X-ray Crystallography at the XChem facility of Diamond Light Source beamline I04-1
Selected structures were deposited in the PDB via a group deposition.<br><a href=https://www.rcsb.org/groups/summary/entry/G_1002153>https://www.rcsb.org/groups/summary/entry/G_1002153</a>
<cite>[ 2020-04-07 ]</cite><br></div></div></div></div><div class=prettybox data-component=prettybox><div class=container><div class=row><div class=col-1 style=display:flex;align-items:center><a href=https://www.rcsb.org/groups/summary/entry/G_1002272><img title=pdb-group-deposition width=40 src=../img/icons/pdb-group-deposition.png></a></div><div id=moonshot-mpro-inhibitors-pdb-group-deposition class=col-11><b>PDB group deposition of SARS-CoV-2 main protease in compelex with inhibitors from the COVID Moonshot</b><br>SARS-CoV-2 main protease screened against COVID Moonshot compounds by X-ray Crystallography at the XChem facility of Diamond Light Source beamline I04-1<br><a href=https://www.rcsb.org/groups/summary/entry/G_1002272>https://www.rcsb.org/groups/summary/entry/G_1002272</a>
<cite>[ 2023-11-29 ]</cite><br></div></div></div></div><hr style=height:5px;border:none;color:#333;background-color:#333><br><br><br></div></div></main><footer></footer><script src=/js/kube.js type=text/javascript></script><script src=/js/kube.legenda.js type=text/javascript></script><script src=/js/master.js type=text/javascript></script></body></html>