---
description: "Developmental Research Project and Mentored Project Awards"
title: "projects"
draft: false
layout: faq
---

# Developmental Research Projects

## Defining mutational constraints of ASAP-prioritized SARS-CoV-2 target proteins via multiplexed, yeast-based cDNA

The COVID-19 pandemic has illuminated that RNA viruses continue to plague the human population and that our ability to anticipate viral outbreaks is limited. To begin to address this challenge, the ASAP team is using a structure-based approach to identify druggable regions of enzymes and structural proteins from positive-strand RNA viruses. Thus, the ASAP virology team needs efficient mutational methods to explore sequence variation within viral genomes, allowing them to: 1) validate druggable targets, including protein domains that lack known enzymatic activities; and 2) identify potential mechanisms of, and hopefully ameliorate, resistance to ASAP-developed antiviral compounds. Our labs have developed powerful multi-site techniques – eukaryotic multiplex genome engineering (eMAGE) -  to edit the ~30-kb SARS-CoV-2 cDNA stably cloned within the stable genetic environment of the yeast artificial chromosome (YAC). In this context, multiplexed clusters of mutations can be created by hijacking the mechanisms of DNA replication, repair, and recombination in yeast. Then, individual replicon or virus genomes can be recovered and tested either individually or in pools to identify mutant genomes whose growth is favored or disfavored under different selection pressures.  Seeking to direct this powerful system toward the implementation of ASAP goals, we will pursue three aims:  (1) Enumerate sequence constraints in the viral Nucleocapsid (Nc) on the formation of non-infectious virus-like particles; (2) Screen candidate Mpro-targeting antiviral compounds for those that are resistance-robust; (3) Enumerate sequence constraints in the nsp13 helicase zinc-binding and other domains on viral viability. Impact: Our work will develop an advanced SARS-CoV-2 genomics toolkit enabling a unique ability to dissect complex polygenic phenotypes for further functional analysis, to reconstruct variants with uncharacterized genotypes, and to provide valuable resources for other research and therapeutic efforts. More broadly, our approach establishes a new paradigm for the rapid reconstruction of viral genome variants, which will expedite research and therapeutic development in future viral outbreaks.

### Project Leader: Brett Lindenbach, Yale University, New Haven, CT 

---

## DNA-encoded libraries targeting viral proteases

In the context of antiviral drug-discovery, the DEL approach is particularly attractive. Having a set of focussed DELs enriched for antiviral activity would allow rapid identification of potent hit molecules in response to emerging threats – in contrast to all other hit finding methods, focussed DELs can reveal compounds that are already significantly optimised for their target directly from the screen. In this proposed project, we will exploit a unique opportunity to use our DEL chemistry to provide targeted libraries focussed on viral proteases
selected by the ASAP consortium. These will include covalent DELs containing specific active site warheads and side pocket groups established to have viral protease activity that are selected in concert with the ASAP medicinal chemistry teams resulting in libraries with the highest chances of delivering potent leads for future programmes. Whilst it is possible to access DELs through commercial suppliers, such as Hitgen and WuXi, commercial collaborations in this area can be costly, or have significant tie-ins for downstream exploitation. Moreover, they rely on existing libraries and do not allow for the synthesis of tailored-libraries. This project will deliver bespoke DELs uniquely poised for viral protease targets with the highest fidelity. Moreover, by working alongside discovery projects within the consortium, libraries will be designed with cutting-edge SAR knowledge for directly relevant targets. Within the project, we will also develop second generation libraries based on hits from initial selections to demonstrate the use of on-DNA library maturation to carry out the early stages of optimisation, exploiting the
advantages of speed and throughput of on-DNA compound synthesis and testing to accelerate the hit-to-lead phase of drug discovery. This has the potential to significantly accelerate the drug-discovery processes, which is critical for rapid response to emerging threats.

### Project Leader: Michael Waring, Newcastle University, Newcastle upon Tyne, United Kingdom

---

## Development of iPSC-derived primary-like systems for medium-throughput antiviral drug discovery

Antiviral drug development is confounded by the use of cell lines that do not faithfully represent the host and organ systems from which they are derived. These cell lines are heavily relied upon in automated pre-clinical development pipelines because there are few other cost-effective options that are amenable to high throughput drug development. Here we propose to improve and characterize the iPSC-derived type II pneumocytes developed by the Zwaka and White laboratories for SARS-CoV-2 antiviral drug discovery. Furthermore, we will develop iPSC-derived primary-like models of neurons, cardiomyocytes, gastrointestinal epithelium cells,macrophages, and natural killer cells, which are common sites of infection for coronaviruses, flaviviruses,and enteroviruses. These new primary-like models will be characterized for cell type specific expression patterns and relevant receptors and the ability to support viral replication of the three target viral families. Finally, these models will be automated in 96- to 384-well format for medium-throughput antiviral drug discovery available to the ASAP consortium. Beyond the ASAP-targeted viruses discussed in this proposal, these models will also be of great use for screening drugs against the many other viral pathogens of pandemic potential which infect these critical organ systems. We expect it to take 2 years to establish, optimize and standardize all of the iPSC-derived systems proposed here. Although we plan to have a functional and standardized type II pneumocyte antiviral assay available immediately, with several more cell types coming online in the first year.

### Project Leader: Thomas Zwaka, Icahn School of Medicine at Mount Sinai, New York, NY

---

# Mentored Research Projects

---

## Bespoke Molecular Models for the AI-Driven Structure-Enabled Antiviral Platform

The AI-driven Structure-Enabled Antiviral Platform (ASAP) employs state-of-the-art massively distributed computational free energy calculations to predict binding affinities between a lead compound and its therapeutic target in order to make efficient use of synthetic resources. Yet
the predictive ability is critically reliant on the accuracy of the simplified computational models (or force fields) that underlie the atomistic simulations. I propose to leverage breakthroughs in my work with the Open Force Field Initiative, to build bespoke force fields for the accurate description of the dynamics and interactions of compound series relevant to the ASAP goals.

### Project Leader: Joshua Horton, Newcastle University, Newcastle upon Tyne, United Kingdom

---

## Rapid, expansive and out-of-catalogue exploration of fragment merges via synthesis on low-cost robotics
Aim 1: 10 - 100x sampling of fragment merges within catalogue-scale budget. This project aims to make
multi-step synthesis of antiviral compounds cheap (<5k) and rapid (<4 weeks) for the exploration of the merge-design space, a product of generative design algorithms, that uses the power of multiple fragments, to maximise the number of protein interactions through combining experimentally determined fragment structural-poses (3D) and electron density distribution (functional groups). [1-3]
Aim 2: Escape catalogues by multi-step reactions within realistic budget. To fully exploit the antiviral
merging design space required to achieve Aim 1, this project aims to digitise multi-step chemistry as required, by reaction pathways suggested my Manifold, for the synthesis of merging designs. Reaction protocols, CAR’s digitised reaction repertoire, will be generalised for automated chemistry suitability, as required by merging designs.
Aim 3: Mitigate synthetic attrition by executing multiple reaction routes per compound. Synthetic protocol
development requires exploration of multiple condition parameters and should not be limited to traditional bench- based investigations. This project aims to expand CAR’s ability to execute multiple protocols per reaction type, using the power of automation, to rapidly explore multiple reaction conditions. This extends to utilising multiple protocols for the synthesis of antivirals requiring chemistry, example Buchwald-Hartwig, sensitive to parameter choice.

### Project Leader: Warren Thompson, Diamond Light Source, Oxford, United Kingdom
